<!DOCTYPE html>
<html lang="be">
<head>
  <title>Люстэрка RiboGrove</title>
  <link rel="stylesheet" href="css/style.css">
  <link rel="stylesheet" href="css/release-page.css">
</head>
<body>
<div class="container">
<div class="lang-section">
<p>&#127760;&nbsp;<a href="index.html">English</a> / <span class="active-lang">Беларуская</span> / <a href="latest_ribogrove_release_ua.html">Українська</a> / <a href="latest_ribogrove_release_ru.html">Русский</a></p>
</div>
<p><div class="warn-section"><a href="https://mbio.bas-net.by/cager/en/ribogrove">Асноўны сайт</a>, дзе размяшчаецца база дадзеных RiboGrove, можа быць недасяжным па-за межамі Беларусі праз тэхнічныя прычыны ды бяду наогул. Таму існуе гэтае люстэрка, а файлы RiboGrove дасяжныя праз Дропбокс (спасылкі <a href="#downloads">ніжэй</a>).<br><br>
На жаль, новыя выпускі RiboGrove будуць выходзіць з затрымкай пасля выпуску 13.219 да жніўня 2024 г.,<br>бо галоўнага куратара базы дадзеных прызвалі ў войска.</div></p><br>
<h1 style="display:none">RiboGrove</h1>
<img src="img/RiboGrove_logo.png"><br><br>
<h2>Змест</h2>
<ul>
<li><a href="#overview">Што такое RiboGrove</a>
<ul class="ribogrove-nested-list">
<li><a href="#other-databases">RiboGrove і іншыя базы дадзеных 16S рРНК</a></li>
<li><a href="#genome-categories">Катэгорыі геномаў</a></li>
</ul>
</li>
<li><a href="#downloads">Файлы</a>
<ul class="ribogrove-nested-list">
<li><a href="#current-release">Актуальны выпуск RiboGrove — 13.219</li>
<li><a href="#release-archive">Архіў выпускаў RiboGrove</li>
<li><a href="#release-notes">Каментары да выпуску</li>
</ul>
</li>
<li><a href="#summary">Статыстычнае апісанне</a>
<ul class="ribogrove-nested-list">
<li><a href="#db-size">Памер RiboGrove</a></li>
<li><a href="#gene-lens">Даўжыня генаў 16S рРНК</a></li>
<li><a href="#copy-number">Колькасть копій генаў 16S рРНК</a></li>
<li><a href="#top-longest">Топ-10 найдаўжэйшых генаў 16S рРНК</a></li>
<li><a href="#top-shortest">Топ-10 найкарацейшых генаў 16S рРНК</a></li>
<li><a href="#top-copy-num">Топ-10 геномаў з найбольшай колькасцю копій генаў 16S рРНК</a></li>
<li><a href="#top-var">Топ-10 геномаў з найбольшай зменлівасцю генаў 16S рРНК</a></li>
<li><a href="#primers-coverages">Спектр дзеяння пар праймераў да розных V-регіёнаў бактэрыяльных генаў 16S рРНК</a></li>
</ul>
</li>
<li><a href="#searching-data">Пошук дадзеных у RiboGrove</a>
<ul class="ribogrove-nested-list">
<li><a href="#header-format">Фармат загалоўка</a></li>
<li><a href="#subset-by-header">Адбор паслядоўнасцяў паводле загалоўкаў</a></li>
<li><a href="#subset-by-len">Адбор паслядоўнасцяў паводле даўжыні</a></li>
<li><a href="#select-head">Адбор дадзеных загалоўкаў</a></li>
</ul>
</li>
<li><a href="#contacts">Кантакты</a></li>
<li><a href="#citing-ribogrove">Цытаванне RiboGrove</a></li>
<li><a href="#faq">Пытанні, якія ў людзей узнікаюць пра RiboGrove</a></li>
</ul>
<hr>
<div id="overview" class="pad-anchor"></div>
<h2>Што такое RiboGrove</h2>
<p>RiboGrove — гэта база дадзеных паслядоўнасцей генаў 16S рРНК бактэрый і архей.</p>
<p>RiboGrove асноўваецца на базе дадзеных <a href="https://ncbi.nlm.nih.gov/refseq/">RefSeq</a>. RiboGrove змяшчае толькі поўнапамерныя паслядоўнасці генаў 16S рРНК, а самі паслядоўнасці паходзяць з цалкам сабраных геномаў прокарыёт, дэпанаваных у RefSeq. Таму мы пастулюем высокую надзейнасць паслядоўнасцей, размешчаных у RiboGrove.</p>
<div id="other-databases" class="pad-anchor"></div>
<p>Табліца ніжэй паказвае, чым (якасна) RiboGrove адрозніваецца ад падобных баз дадзеных паслядоўнасцей РНК, а менавіта <a href="https://rrndb.umms.med.umich.edu/"><i>rrn</i>DB</a>, <a href="https://www.arb-silva.de/">Silva</a>, <a href="http://rdp.cme.msu.edu/">RDP</a> і <a href="https://greengenes.secondgenome.com/">Greengenes</a>. Кажучы каротка, RiboGrove саступае аналагам у колькасці паслядоўнасцей і іх разнастайнасці, але пераўзыходзіць у надзейнасці паслядоўнасцей.</p>
<table class="sum-table">
<tbody>
  <tr class="sumtab-row"><th></th><th class="alcentr">RiboGrove</th><th class="alcentr"><i>rrn</i>DB</th><th class="alcentr">Silva</th><th class="alcentr">RDP</th><th class="alcentr">Greengenes</th></tr>
  <tr class="sumtab-row"><td class="feature-col">Прадстаўленыя арганізмы</td><td class="alcentr">Бактэрыі<br>Археі</td><td class="alcentr">Бактэрыі<br>Археі</td><td class="alcentr">Бактэрыі<br>Археі<br>Эўкарыёты</td><td class="alcentr">Бактэрыі<br>Археі<br>Эўкарыёты</td><td class="alcentr">Бактэрыі<br>Археі</td></tr>
  <tr class="sumtab-row"><td class="feature-col">Прадстаўленыя субадзінкі рыбасом</td><td class="alcentr">Малая</td><td class="alcentr">Малая</td><td class="alcentr">Вялікая<br>Малая</td><td class="alcentr">Вялікая<br>Малая</td><td class="alcentr">Малая</td></tr>
  <tr class="sumtab-row"><td class="feature-col">Змяшчае паслядоўнасці сабраных геномаў</td><td class="alcentr">Так</td><td class="alcentr">Так</td><td class="alcentr">Так</td><td class="alcentr">Так</td><td class="alcentr">Так</td></tr>
  <tr class="sumtab-row"><td class="feature-col">Змяшчае паслядоўнасці ампліконаў</td><td class="alcentr">Не</td><td class="alcentr">Не</td><td class="alcentr">Так</td><td class="alcentr">Так</td><td class="alcentr">Так</td></tr>
  <tr class="sumtab-row"><td class="feature-col">Змяшчае няпоўныя паслядоўнасці генаў</td><td class="alcentr">Не</td><td class="alcentr">Так</td><td class="alcentr">Так</td><td class="alcentr">Так</td><td class="alcentr">Так</td></tr>
  <tr class="sumtab-row"><td class="feature-col">Адрознівае катэгорыі геномаў</td><td class="alcentr">Так</td><td class="alcentr">Не</td><td class="alcentr"><i>Крытэр<br>непрымянімы</i></td><td class="alcentr"><i>Крытэр<br>непрымянімы</i></td><td class="alcentr"><i>Крытэр<br>непрымянімы</i></td></tr>
</tbody>
</table>
<div id="genome-categories" class="pad-anchor"></div>
<h3>Катэгорыі геномаў</h3>
<p>Усе геномы, з якіх браліся дадзеныя для стварэння RiboGrove, былі падзеленыя на тры катэгорыі паводле іхняй надзейнасці:</p>
<ol style="list-style:none;">
  <li><strong>Катэгорыя 1</strong> (найбольшая надзейнасць). Геномы, якія не дэманструюць прыкмет нізкаякаснай зборкі і якія былі секвенаваныя з дапамогай тэхналогіі PacBio альбо камбінацыі (Oxford&nbsp;Nanopore&nbsp;+&nbsp;Illumina).</li>
  <li><strong>Катэгорыя 2.</strong> Геномы, якія не дэманструюць прыкмет нізкаякаснай зборкі, секвенаваныя з дапамогай любой іншай тэхналогіі (або тыя, для якіх тэхналогія секвенавання не была ўказаная).</li>
  <li><strong>Катэгорыя 3</strong> (найменшая надзейнасць). Геномы, якія дэманструюць хаця б адну прыкмету нізкаякаснай зборкі.</li>
</ol>
<p>Прыкметы нізкаякаснай зборкі:</p>
<ul>
  <li>Геном утрымлівае хаця б адну выраджаную аснову у паслядоўнасцях яго генаў 16S рРНК.</li>
  <li>Зборка геному утрымлівае хаця б адзін запіс базы дадзеных RefSeq, у назве якой ёсць фраза «map unlocalized», і гэты запіс утрымлівае паслядоўнасць гена 16S рРНК або яго частку.</li>
</ul>
<br>
<p>Праграмы, з дапамогай якіх была створана RiboGrove, знаходзяцца ў наступным ГітХаб-рэпазіторыі: <a href="https://github.com/masikol/ribogrove-tools">ribogrove-tools</a>.</p>
<hr>
<div id="downloads" class="pad-anchor"></div>
<h2>Файлы</h2>
<div id="current-release" class="pad-anchor"></div>
<h3>Актуальны выпуск RiboGrove — 13.219 (19.07.2023)</h3>
<p>Выпуск заснованы на базе дадзеных RefSeq 219.</p>
<ul>
<li>Fasta-файл з поўнапамернымі паслядоўнасцямі генаў 16S рРНК. <a href="https://www.dropbox.com/scl/fi/qw50iuifa2yysevcy58ht/ribogrove_13.219_sequences.fasta.gz?rlkey=uwbrhxyxcou82rie20lmzmtt9&dl=0">Спампаваць („гзіпʼнуты“ fasta-файл, 7,49 Мб)</a></li>
<li>Метададзеныя. <a href="https://www.dropbox.com/scl/fi/b89nnjelmerg05clwnk8n/metadata_ribogrove_13.219.zip?rlkey=j4hrnbo40dls46uykkpy8rgkl&dl=0">Спампаваць (zip-архіў 5,50 Мб)</a><br>
<details>
  <summary><small>Якая менавіта інфармацыя утрымліваецца ў метададзеных?</small></summary>
  <small>
    <p>Метададзеныя складаюцца з наступных файлаў:</p>
    <ol>
    <li>
      <span class="samp-highl">discarded_sequences.fasta.gz</span><br>
      Гэта fasta-файл, які змяшчае паслядоўнасці, якія прысутнічалі ў зыходных геномах, былі пры гэтым анатаваныя як гены 16S рРНК, але якія былі „забракаваныя“ (discarded) праз іхнюю непаўнату, прысутнасць унутрыгенных паўтораў і г.д. Такім чынам, гэтыя паслядоўнасці не ўключаныя ў RiboGrove.
    </li>
    <li>
      <span class="samp-highl">source_RefSeq_genomes.tsv</span><br>
      Гэта TSV-файл, які змяшчае інфармацыю пра тое, якія геномы выкарыстоўваліся для стварэння RiboGrove.
    </li>
    <li>
      <span class="samp-highl">gene_seqs_statistics.tsv</span>, <span class="samp-highl">discarded_gene_seqs_statistics.tsv</span><br>
      Гэта TSV-файлы, якія змяшчаюць інфармацыю пра нуклеатыдны склад, памер, геномную і таксанамічную прыналежнасць паслядоўнасцей генаў. Першы файл з гэтых двух апісвае фінальныя паслядоўнасці, уключаныя ў RiboGrove, а другі — „забракаваныя“ паслядоўнасці.
    </li>
    <li>
      <span class="samp-highl">categories.tsv</span><br>
      Гэта TSV-файл, які змяшчае інфармацыю пра катэгорыю кожнага геному, які быў выкарыстаны для стварэння RiboGrove. Таксама у гэтым файле ёсць інфамацыя пра тэхналогіі, якія выкарыстоўваліся для секвенавання гэтых геномаў.
    </li>
    <li>
      <span class="samp-highl">taxonomy.tsv</span><br>
      Гэта TSV-файл, які змяшчае таксанамічную прыналежнасць кожнага геному і гена.
    </li>
    <li>
      <span class="samp-highl">intragenic_repeats.tsv</span><br>
      Гэта TSV-файл, які змяшчае інфармацыю пра унутрыгенныя паўторы ў паслядоўнасцях генаў, знойдзеныя з дапамогай праграмы <a href="https://github.com/deprekate/RepeatFinder">RepeatFinder</a>.
    </li>
    <li>
      <span class="samp-highl">entropy_summary.tsv</span><br>
      Гэта TSV-файл, які змяшчае інфармацыю пра унутрыгеномную зменлівасць генаў 16S рРНК. Унутрыгеномную зменлівасць вылічалі толькі для геномаў катэгорыі&nbsp;1, у якіх ёсць хаця б два гены 16S рРНК. Для ацэнкі зменлівасці выкарыстоўвалі энтрапію Шэнана: выраўноўвалі паслядоўнасці генаў кожнага геному з дапамогай праграмы <a href="https://drive5.com/muscle5/">MUSCLE</a>, і пасля вылічалі энтрапію для кожнай калонкі множнага выраўноўвання.
    </li>
    <li>
      <span class="samp-highl">primer_pair_genomic_coverage.tsv</span><br>
      Гэта TSV-файл, які змяшчае геномнае пакрыццё пар <span title="Палімэразная ланцуговая рэакцыя" style="cursor:help">ПЛР</span>-праймераў да розных V-участкаў генаў 16S рРНК. Напрыклад, для сямейства <i>Enterobacteriaceae</i> геномнае пакрыццё пары праймераў — гэта адсотак геномаў <i>Enterobacteriaceae</i>, якія змяшчаюць прынамсі адзін ген 16S рРНК, на матрыцы якога (тэарытычна) можа сфармавацца ПЛР-прадукт з дапамогай гэтай пары праймераў.
    </li>
  </ol></small>
</details>
</li>
</ul>
<p>Файл fasta сціснуты з дапамогаю праграмы gzip, а файлы з метададзенымі сціснутыя ў zip-архіў. Каб распакаваць іх, карыстальнікі Linux і Mac OS могуць скарастыцца якраз праграмамі gzip і zip; гэтыя праграмы звычайна ўсталяваныя разам з аперыцыйнай сістэмай. Карыстальнікам Windows можа прыдацца бясплатная праграма-архіватар <a href="https://www.7-zip.org/">7-Zip</a>.</p>
<div id="release-archive" class="pad-anchor"></div>
<h3>Архіў выпускаў RiboGrove</h3>
<p>Усе папярэднія выпускі можна знайсці ў <a href="archive/ribogrove_archive_en.html"> архіве выпускаў RiboGrove</a>. На жаль, ён вядзецца толькі па-англійску.</p>
<div id="release-notes" class="pad-anchor"></div>
<h3>Каментары да выпуску</h3>
<p>Пачынаючы з RiboGrove 13.219, метададзеныя змяшчаюць файл <span class="samp-highl">primer_pair_genomic_coverage.tsv</span>. У гэтым файле можна знайсці значэнні <a href="#primers-coverages">пакрыцця</a> розных пар <span title="Палімеразная ланцуговая рэакцыя">ПЛР</span>-праймераў да розных V-участкаў для бактэрый кожнага тыпу, класу, парадку, сямейства, роду і віду.</p>
<p>Каментары да ўсіх выпускаў RiboGrove можна знайсці на <a href="ribogrove_release_notes_en.html">старонцы каментароў да выпускаў</a>.</p>
<hr>
<div id="summary" class="pad-anchor"></div>
<h2>Статыстычнае апісанне</h2>
<div id="db-size" class="pad-anchor"></div>
<table class="sum-table">
<caption>Памер RiboGrove</caption>
<tbody>
<tr><th></th><th class="numcol">Бактэрыі</th><th class="numcol">Археі</th><th class="numcol">Усяго</th></tr>
<tr class="sumtab-row">
<td>Колькасць паслядоўнасцяў генаў</td>
<td class="numcol">184&nbsp;732</td>
<td class="numcol">902</td>
<td class="numcol">185&nbsp;654</td>
</tr>
<tr class="sumtab-row">
<td>Колькасць унікальных паслядоўнасцяў</td>
<td class="numcol">48&nbsp;358</td>
<td class="numcol">649</td>
<td class="numcol">49&nbsp;019</td>
</tr>
<tr class="sumtab-row">
<td>Колькасць відаў</td>
<td class="numcol">8&nbsp;664</td>
<td class="numcol">419</td>
<td class="numcol">9&nbsp;086</td>
</tr>
<tr class="sumtab-row">
<td>Колькасць геномаў</td>
<td class="numcol">34&nbsp;159</td>
<td class="numcol">531</td>
<td class="numcol">34&nbsp;693</td>
</tr>
<tr class="sumtab-row">
<td>Колькасць геномаў 1-й катэгорыі</td>
<td class="numcol">22&nbsp;986</td>
<td class="numcol">201</td>
<td class="numcol">23&nbsp;190</td>
</tr>
<tr class="sumtab-row">
<td>Колькасць геномаў 2-й катэгорыі</td>
<td class="numcol">11&nbsp;026</td>
<td class="numcol">330</td>
<td class="numcol">11&nbsp;356</td>
</tr>
<tr class="sumtab-row">
<td>Колькасць геномаў 3-й катэгорыі</td>
<td class="numcol">147</td>
<td class="numcol">0</td>
<td class="numcol">147</td>
</tr>
</tbody>
</table>
<div id="gene-lens" class="pad-anchor"></div>
<table class="sum-table">
<caption>Даўжыня генаў 16S рРНК</caption>
<tbody>
<tr><th></th><th class="numcol">Бактэрыі</th><th class="numcol">Археі</th></tr>
<tr class="sumtab-row">
<td>Мінімальная даўжыня, п.а.</td>
<td class="numcol">1&nbsp;401,00</td>
<td class="numcol">1&nbsp;439,00</td>
</tr>
<tr class="sumtab-row">
<td>25-ы працэнтыль, п.а. <sup>*</sup></td>
<td class="numcol">1&nbsp;517,00</td>
<td class="numcol">1&nbsp;471,00</td>
</tr>
<tr class="sumtab-row">
<td>Медыяна, п.а. <sup>*</sup></td>
<td class="numcol">1&nbsp;531,00</td>
<td class="numcol">1&nbsp;473,00</td>
</tr>
<tr class="sumtab-row">
<td>75-ы працэнтыль, п.а. <sup>*</sup></td>
<td class="numcol">1&nbsp;542,00</td>
<td class="numcol">1&nbsp;486,00</td>
</tr>
<tr class="sumtab-row">
<td>Сярэдняя даўжыня, п.а. <sup>*</sup></td>
<td class="numcol">1&nbsp;527,07</td>
<td class="numcol">1&nbsp;494,07</td>
</tr>
<tr class="sumtab-row">
<td>Мода, п.а. <sup>*</sup></td>
<td class="numcol">1&nbsp;537,00</td>
<td class="numcol">1&nbsp;472,00</td>
</tr>
<tr class="sumtab-row">
<td>Максімальная даўжыня, п.а.</td>
<td class="numcol">2&nbsp;438,00</td>
<td class="numcol">3&nbsp;604,00</td>
</tr>
<tr class="sumtab-row">
<td>Сярэднеквадратычнае адхіленне, п.а.<sup>*</sup></td>
<td class="numcol">25,49</td>
<td class="numcol">131,48</td>
</tr>
</tbody>
</table>
<p><sup>*</sup> Метрыкі, пазначаныя зорачкай, вылічаныя з папярэдняй нармалізацыяй: вылічэннем <em>медыян</em> даўжынь генаў у межах віду.</p>
<div id="copy-number" class="pad-anchor"></div>
<table class="sum-table">
<caption>Колькасць копій генаў 16S рРНК</caption>
<tbody>
<tr><th class="numcol">Колькасць копій <sup>*</sup></th><th class="alncenter" colspan="2">Бактэрыі</th><th class="alncenter" colspan="2">Археі</th></tr>
<tr><td></td><td>Колькасць відаў</td><td>Адсотак відаў, %</td><td>Колькасць відаў</td><td>Адсотак відаў, %</td></tr>
<tr class="sumtab-row">
<td class="numcol">1</td>
<td class="numcol">1&nbsp;050</td>
<td class="numcol">12,12</td>
<td class="numcol">220</td>
<td class="numcol">52,51</td>
</tr>
<tr class="sumtab-row">
<td class="numcol">2</td>
<td class="numcol">1&nbsp;558</td>
<td class="numcol">17,98</td>
<td class="numcol">127</td>
<td class="numcol">30,31</td>
</tr>
<tr class="sumtab-row">
<td class="numcol">3</td>
<td class="numcol">1&nbsp;310</td>
<td class="numcol">15,12</td>
<td class="numcol">56</td>
<td class="numcol">13,37</td>
</tr>
<tr class="sumtab-row">
<td class="numcol">4</td>
<td class="numcol">1&nbsp;136</td>
<td class="numcol">13,11</td>
<td class="numcol">11</td>
<td class="numcol">2,63</td>
</tr>
<tr class="sumtab-row">
<td class="numcol">5</td>
<td class="numcol">717</td>
<td class="numcol">8,28</td>
<td class="numcol">5</td>
<td class="numcol">1,19</td>
</tr>
<tr class="sumtab-row">
<td class="numcol">6</td>
<td class="numcol">941</td>
<td class="numcol">10,86</td>
<td class="numcol">0</td>
<td class="numcol">0,00</td>
</tr>
<tr class="sumtab-row">
<td class="numcol">7</td>
<td class="numcol">759</td>
<td class="numcol">8,76</td>
<td class="numcol">0</td>
<td class="numcol">0,00</td>
</tr>
<tr class="sumtab-row">
<td class="numcol">8</td>
<td class="numcol">426</td>
<td class="numcol">4,92</td>
<td class="numcol">0</td>
<td class="numcol">0,00</td>
</tr>
<tr class="sumtab-row">
<td class="numcol">9</td>
<td class="numcol">245</td>
<td class="numcol">2,83</td>
<td class="numcol">0</td>
<td class="numcol">0,00</td>
</tr>
<tr class="sumtab-row">
<td class="numcol">10</td>
<td class="numcol">190</td>
<td class="numcol">2,19</td>
<td class="numcol">0</td>
<td class="numcol">0,00</td>
</tr>
<tr class="sumtab-row">
<td class="numcol">11</td>
<td class="numcol">103</td>
<td class="numcol">1,19</td>
<td class="numcol">0</td>
<td class="numcol">0,00</td>
</tr>
<tr class="sumtab-row">
<td class="numcol">12</td>
<td class="numcol">81</td>
<td class="numcol">0,93</td>
<td class="numcol">0</td>
<td class="numcol">0,00</td>
</tr>
<tr class="sumtab-row">
<td class="numcol">13</td>
<td class="numcol">36</td>
<td class="numcol">0,42</td>
<td class="numcol">0</td>
<td class="numcol">0,00</td>
</tr>
<tr class="sumtab-row">
<td class="numcol">14</td>
<td class="numcol">60</td>
<td class="numcol">0,69</td>
<td class="numcol">0</td>
<td class="numcol">0,00</td>
</tr>
<tr class="sumtab-row">
<td class="numcol">15</td>
<td class="numcol">20</td>
<td class="numcol">0,23</td>
<td class="numcol">0</td>
<td class="numcol">0,00</td>
</tr>
<tr class="sumtab-row">
<td class="numcol">16</td>
<td class="numcol">5</td>
<td class="numcol">0,06</td>
<td class="numcol">0</td>
<td class="numcol">0,00</td>
</tr>
<tr class="sumtab-row">
<td class="numcol">17</td>
<td class="numcol">8</td>
<td class="numcol">0,09</td>
<td class="numcol">0</td>
<td class="numcol">0,00</td>
</tr>
<tr class="sumtab-row">
<td class="numcol">18</td>
<td class="numcol">7</td>
<td class="numcol">0,08</td>
<td class="numcol">0</td>
<td class="numcol">0,00</td>
</tr>
<tr class="sumtab-row">
<td class="numcol">19</td>
<td class="numcol">2</td>
<td class="numcol">0,02</td>
<td class="numcol">0</td>
<td class="numcol">0,00</td>
</tr>
<tr class="sumtab-row">
<td class="numcol">20</td>
<td class="numcol">6</td>
<td class="numcol">0,07</td>
<td class="numcol">0</td>
<td class="numcol">0,00</td>
</tr>
<tr class="sumtab-row">
<td class="numcol">21</td>
<td class="numcol">1</td>
<td class="numcol">0,01</td>
<td class="numcol">0</td>
<td class="numcol">0,00</td>
</tr>
<tr class="sumtab-row">
<td class="numcol">24</td>
<td class="numcol">1</td>
<td class="numcol">0,01</td>
<td class="numcol">0</td>
<td class="numcol">0,00</td>
</tr>
<tr class="sumtab-row">
<td class="numcol">27</td>
<td class="numcol">1</td>
<td class="numcol">0,01</td>
<td class="numcol">0</td>
<td class="numcol">0,00</td>
</tr>
<tr class="sumtab-row">
<td class="numcol">37</td>
<td class="numcol">1</td>
<td class="numcol">0,01</td>
<td class="numcol">0</td>
<td class="numcol">0,00</td>
</tr>
</tbody>
</table>
<p><sup>*</sup> Медыянная колькасць копій у межах віду.</p>
<div id="top-longest" class="pad-anchor"></div>
<table class="sum-table">
<caption>Топ-10 найдаўжэйшых генаў 16S рРНК</caption>
<tbody>
<tr><th class="alnleft">Арганізм</th><th class="numcol">Даўжыня, п.а.</th><th class="alnleft">SeqID <sup>*</sup></th><th class="alnleft">Код доступу<br>геномнай зборкі</th></tr>
<tr>
<td colspan="4" class="subhead">Бактэрыі</td>
</tr>
<tr class="sumtab-row">
<td><i>Thermus</i> <i>thermophilus</i> AA2-2</td>
<td class="numcol">2&nbsp;438</td>
<td class="seqid-td">GCF_019974355.1:NZ_AP024929.1:249100-251537:minus</td>
<td><a href="https://ncbi.nlm.nih.gov/assembly/GCF_019974355.1">GCF_019974355.1</a></td>
</tr>
<tr class="sumtab-row">
<td><i>Ca. Annandia</i> <i>pinicola</i> Ad13-065</td>
<td class="numcol">1&nbsp;887</td>
<td class="seqid-td">GCF_020541245.1:NZ_CP045876.1:290071-291957:minus</td>
<td><a href="https://ncbi.nlm.nih.gov/assembly/GCF_020541245.1">GCF_020541245.1</a></td>
</tr>
<tr class="sumtab-row">
<td><i>Nitrosophilus</i> <i>labii</i> HRV44</td>
<td class="numcol">1&nbsp;806</td>
<td class="seqid-td">GCF_014466985.1:NZ_AP022826.1:1258017-1259822:minus<br>GCF_014466985.1:NZ_AP022826.1:1532588-1534393:minus<br>GCF_014466985.1:NZ_AP022826.1:1939914-1941719:minus</td>
<td><a href="https://ncbi.nlm.nih.gov/assembly/GCF_014466985.1">GCF_014466985.1</a></td>
</tr>
<tr class="sumtab-row">
<td><i>Gelria</i> sp. Kuro-4</td>
<td class="numcol">1&nbsp;788</td>
<td class="seqid-td">GCF_019668485.1:NZ_AP024619.1:2016182-2017969:minus</td>
<td><a href="https://ncbi.nlm.nih.gov/assembly/GCF_019668485.1">GCF_019668485.1</a></td>
</tr>
<tr class="sumtab-row">
<td><i>Thermoanaerobacter</i> <i>brockii</i> Ako-1</td>
<td class="numcol">1&nbsp;781</td>
<td class="seqid-td">GCF_000175295.2:NC_014964.1:2252888-2254668:minus</td>
<td><a href="https://ncbi.nlm.nih.gov/assembly/GCF_000175295.2">GCF_000175295.2</a></td>
</tr>
<tr class="sumtab-row">
<td><i>Thermoanaerobacter</i><br><i>pseudethanolicus</i> ATCC 33223</td>
<td class="numcol">1&nbsp;781</td>
<td class="seqid-td">GCF_000019085.1:NC_010321.1:2265744-2267524:minus</td>
<td><a href="https://ncbi.nlm.nih.gov/assembly/GCF_000019085.1">GCF_000019085.1</a></td>
</tr>
<tr class="sumtab-row">
<td><i>Thermoanaerobacter</i> sp. RKWS2</td>
<td class="numcol">1&nbsp;754</td>
<td class="seqid-td">GCF_026240795.1:NZ_CP110888.1:94012-95765:plus</td>
<td><a href="https://ncbi.nlm.nih.gov/assembly/GCF_026240795.1">GCF_026240795.1</a></td>
</tr>
<tr class="sumtab-row">
<td><i>Campylobacter</i> <i>sputorum</i> LMG 7795</td>
<td class="numcol">1&nbsp;744</td>
<td class="seqid-td">GCF_008245005.1:NZ_CP043427.1:609141-610884:plus<br>GCF_008245005.1:NZ_CP043427.1:930699-932442:minus<br>GCF_008245005.1:NZ_CP043427.1:1503078-1504821:minus</td>
<td><a href="https://ncbi.nlm.nih.gov/assembly/GCF_008245005.1">GCF_008245005.1</a></td>
</tr>
<tr class="sumtab-row">
<td><i>Campylobacter</i> <i>sputorum</i> RM3237</td>
<td class="numcol">1&nbsp;744</td>
<td class="seqid-td">GCF_002220795.1:NZ_CP019682.1:607981-609724:plus<br>GCF_002220795.1:NZ_CP019682.1:929565-931308:minus<br>GCF_002220795.1:NZ_CP019682.1:1501945-1503688:minus</td>
<td><a href="https://ncbi.nlm.nih.gov/assembly/GCF_002220795.1">GCF_002220795.1</a></td>
</tr>
<tr class="sumtab-row">
<td><i>Campylobacter</i> <i>sputorum</i> CCUG 20703</td>
<td class="numcol">1&nbsp;743</td>
<td class="seqid-td">GCF_002220735.1:NZ_CP019683.1:606847-608589:plus<br>GCF_002220735.1:NZ_CP019683.1:935163-936905:minus<br>GCF_002220735.1:NZ_CP019683.1:1558189-1559931:minus</td>
<td><a href="https://ncbi.nlm.nih.gov/assembly/GCF_002220735.1">GCF_002220735.1</a></td>
</tr>
<tr>
<td colspan="4" class="subhead">Археі</td>
</tr>
<tr class="sumtab-row">
<td><i>Pyrobaculum</i> <i>ferrireducens</i> 1860</td>
<td class="numcol">3&nbsp;604</td>
<td class="seqid-td">GCF_000234805.1:NC_016645.1:127214-130817:plus</td>
<td><a href="https://ncbi.nlm.nih.gov/assembly/GCF_000234805.1">GCF_000234805.1</a></td>
</tr>
<tr class="sumtab-row">
<td><i>Pyrobaculum</i> <i>aerophilum</i> IM2</td>
<td class="numcol">2&nbsp;213</td>
<td class="seqid-td">GCF_000007225.1:NC_003364.1:1089640-1091852:plus</td>
<td><a href="https://ncbi.nlm.nih.gov/assembly/GCF_000007225.1">GCF_000007225.1</a></td>
</tr>
<tr class="sumtab-row">
<td><i>Pyrobaculum</i> <i>arsenaticum</i> DSM 13514</td>
<td class="numcol">2&nbsp;212</td>
<td class="seqid-td">GCF_000016385.1:NC_009376.1:623323-625534:minus</td>
<td><a href="https://ncbi.nlm.nih.gov/assembly/GCF_000016385.1">GCF_000016385.1</a></td>
</tr>
<tr class="sumtab-row">
<td><i>Aeropyrum</i> <i>pernix</i> K1</td>
<td class="numcol">2&nbsp;202</td>
<td class="seqid-td">GCF_000011125.1:NC_000854.2:1218712-1220913:minus</td>
<td><a href="https://ncbi.nlm.nih.gov/assembly/GCF_000011125.1">GCF_000011125.1</a></td>
</tr>
<tr class="sumtab-row">
<td><i>Pyrobaculum</i> <i>neutrophilum</i> V24Sta</td>
<td class="numcol">2&nbsp;197</td>
<td class="seqid-td">GCF_000019805.1:NC_010525.1:690419-692615:plus</td>
<td><a href="https://ncbi.nlm.nih.gov/assembly/GCF_000019805.1">GCF_000019805.1</a></td>
</tr>
<tr class="sumtab-row">
<td><i>Ca. Mancarchaeum</i> <i>acidiphilum</i> Mia14</td>
<td class="numcol">2&nbsp;008</td>
<td class="seqid-td">GCF_002214165.1:NZ_CP019964.1:751297-753304:minus</td>
<td><a href="https://ncbi.nlm.nih.gov/assembly/GCF_002214165.1">GCF_002214165.1</a></td>
</tr>
<tr class="sumtab-row">
<td><i>Ca. Micrarchaeum</i> sp. A_DKE</td>
<td class="numcol">2&nbsp;003</td>
<td class="seqid-td">GCF_016806735.1:NZ_CP060530.1:203642-205644:minus</td>
<td><a href="https://ncbi.nlm.nih.gov/assembly/GCF_016806735.1">GCF_016806735.1</a></td>
</tr>
<tr class="sumtab-row">
<td><i>Caldivirga</i> <i>maquilingensis</i> IC-167</td>
<td class="numcol">1&nbsp;679</td>
<td class="seqid-td">GCF_000018305.1:NC_009954.1:129150-130828:minus</td>
<td><a href="https://ncbi.nlm.nih.gov/assembly/GCF_000018305.1">GCF_000018305.1</a></td>
</tr>
<tr class="sumtab-row">
<td><i>Aeropyrum</i> <i>camini</i> SY1</td>
<td class="numcol">1&nbsp;650</td>
<td class="seqid-td">GCF_000591035.1:NC_022521.1:1165168-1166817:minus</td>
<td><a href="https://ncbi.nlm.nih.gov/assembly/GCF_000591035.1">GCF_000591035.1</a></td>
</tr>
<tr class="sumtab-row">
<td><i>Pyrolobus</i> <i>fumarii</i> 1A</td>
<td class="numcol">1&nbsp;576</td>
<td class="seqid-td">GCF_000223395.1:NC_015931.1:84671-86246:minus</td>
<td><a href="https://ncbi.nlm.nih.gov/assembly/GCF_000223395.1">GCF_000223395.1</a></td>
</tr>
</tbody>
</table>
<p><sup>*</sup> SeqID — гэта ідэнтыфікатар паслядоўнасці RiboGrove.</p>
<div id="top-shortest" class="pad-anchor"></div>
<table class="sum-table">
<caption>Топ-10 найкарацейшых генаў 16S рРНК</caption>
<tbody>
<tr><th class="alnleft">Арганізм</th><th class="numcol">Даўжыня, п.а.</th><th class="alnleft">SeqID <sup>*</sup></th><th class="alnleft">Код доступу<br>геномнай зборкі</th></tr>
<tr>
<td colspan="4" class="subhead">Бактэрыі</td>
</tr>
<tr class="sumtab-row">
<td><i>Anabaena</i> sp. YBS01</td>
<td class="numcol">1&nbsp;401</td>
<td class="seqid-td">GCF_009498015.1:NZ_CP034058.1:6920299-6921699:minus</td>
<td><a href="https://ncbi.nlm.nih.gov/assembly/GCF_009498015.1">GCF_009498015.1</a></td>
</tr>
<tr class="sumtab-row">
<td><i>Clostridioides</i> <i>difficile</i> TW11</td>
<td class="numcol">1&nbsp;426</td>
<td class="seqid-td">GCF_009362915.1:NZ_CP045224.1:4068440-4069865:minus</td>
<td><a href="https://ncbi.nlm.nih.gov/assembly/GCF_009362915.1">GCF_009362915.1</a></td>
</tr>
<tr class="sumtab-row">
<td><i>Roseicitreum</i> <i>antarcticum</i> ZS2-28</td>
<td class="numcol">1&nbsp;447</td>
<td class="seqid-td">GCF_014681765.1:NZ_CP061498.1:3436150-3437596:plus</td>
<td><a href="https://ncbi.nlm.nih.gov/assembly/GCF_014681765.1">GCF_014681765.1</a></td>
</tr>
<tr class="sumtab-row">
<td><i>Hirschia</i> <i>baltica</i> ATCC 49814</td>
<td class="numcol">1&nbsp;448</td>
<td class="seqid-td">GCF_000023785.1:NC_012982.1:2336679-2338126:minus</td>
<td><a href="https://ncbi.nlm.nih.gov/assembly/GCF_000023785.1">GCF_000023785.1</a></td>
</tr>
<tr class="sumtab-row">
<td><i>Sagittula</i> sp. MA-2</td>
<td class="numcol">1&nbsp;449</td>
<td class="seqid-td">GCF_030126985.1:NZ_CP126145.1:439-1887:plus<br>GCF_030126985.1:NZ_CP126145.1:2907211-2908659:minus</td>
<td><a href="https://ncbi.nlm.nih.gov/assembly/GCF_030126985.1">GCF_030126985.1</a></td>
</tr>
<tr class="sumtab-row">
<td><i>Mameliella</i> <i>alba</i> KU6B</td>
<td class="numcol">1&nbsp;449</td>
<td class="seqid-td">GCF_011405015.1:NZ_AP022337.1:267140-268588:plus<br>GCF_011405015.1:NZ_AP022337.1:1420943-1422391:plus<br>GCF_011405015.1:NZ_AP022337.1:3191212-3192660:minus</td>
<td><a href="https://ncbi.nlm.nih.gov/assembly/GCF_011405015.1">GCF_011405015.1</a></td>
</tr>
<tr class="sumtab-row">
<td><i>Clostridioides</i> <i>difficile</i> Cd18</td>
<td class="numcol">1&nbsp;450</td>
<td class="seqid-td">GCF_018884705.1:NZ_CP037806.1:136016-137465:plus</td>
<td><a href="https://ncbi.nlm.nih.gov/assembly/GCF_018884705.1">GCF_018884705.1</a></td>
</tr>
<tr class="sumtab-row">
<td><i>Hyphomonas</i> sp. Mor2</td>
<td class="numcol">1&nbsp;451</td>
<td class="seqid-td">GCF_001854405.1:NZ_CP017718.1:2304269-2305719:minus</td>
<td><a href="https://ncbi.nlm.nih.gov/assembly/GCF_001854405.1">GCF_001854405.1</a></td>
</tr>
<tr class="sumtab-row">
<td><i>Listeria</i> <i>monocytogenes</i> 3BS28</td>
<td class="numcol">1&nbsp;452</td>
<td class="seqid-td">GCF_018604265.1:NZ_CP075876.1:2326384-2327835:minus</td>
<td><a href="https://ncbi.nlm.nih.gov/assembly/GCF_018604265.1">GCF_018604265.1</a></td>
</tr>
<tr class="sumtab-row">
<td><i>Antarctobacter</i> <i>heliothermus</i> SMS3</td>
<td class="numcol">1&nbsp;453</td>
<td class="seqid-td">GCF_002237555.1:NZ_CP022540.1:1369380-1370832:plus<br>GCF_002237555.1:NZ_CP022540.1:2482480-2483932:plus</td>
<td><a href="https://ncbi.nlm.nih.gov/assembly/GCF_002237555.1">GCF_002237555.1</a></td>
</tr>
<tr>
<td colspan="4" class="subhead">Археі</td>
</tr>
<tr class="sumtab-row">
<td><i>Ignicoccus</i> <i>hospitalis</i> KIN4/I</td>
<td class="numcol">1&nbsp;439</td>
<td class="seqid-td">GCF_000017945.1:NC_009776.1:728362-729800:plus</td>
<td><a href="https://ncbi.nlm.nih.gov/assembly/GCF_000017945.1">GCF_000017945.1</a></td>
</tr>
<tr class="sumtab-row">
<td><i>Methanocaldococcus</i> <i>lauensis</i> SG7</td>
<td class="numcol">1&nbsp;457</td>
<td class="seqid-td">GCF_902827225.1:NZ_LR792632.1:542755-544211:plus</td>
<td><a href="https://ncbi.nlm.nih.gov/assembly/GCF_902827225.1">GCF_902827225.1</a></td>
</tr>
<tr class="sumtab-row">
<td><i>Halorubrum</i> sp. BOL3-1</td>
<td class="numcol">1&nbsp;463</td>
<td class="seqid-td">GCF_004114375.1:NZ_CP034692.1:397753-399215:minus</td>
<td><a href="https://ncbi.nlm.nih.gov/assembly/GCF_004114375.1">GCF_004114375.1</a></td>
</tr>
<tr class="sumtab-row">
<td><i>Natronomonas</i> <i>gomsonensis</i> KCTC 4088</td>
<td class="numcol">1&nbsp;466</td>
<td class="seqid-td">GCF_024300825.1:NZ_CP101323.1:2500564-2502029:plus</td>
<td><a href="https://ncbi.nlm.nih.gov/assembly/GCF_024300825.1">GCF_024300825.1</a></td>
</tr>
<tr class="sumtab-row">
<td><i>Ca. Methanomethylophilus</i> <i>alvus</i> Mx-05</td>
<td class="numcol">1&nbsp;466</td>
<td class="seqid-td">GCF_003711245.1:NZ_CP017686.1:283608-285073:plus</td>
<td><a href="https://ncbi.nlm.nih.gov/assembly/GCF_003711245.1">GCF_003711245.1</a></td>
</tr>
<tr class="sumtab-row">
<td><i>Salinirubellus</i> <i>salinus</i> ZS-35-S2</td>
<td class="numcol">1&nbsp;466</td>
<td class="seqid-td">GCF_025231485.1:NZ_CP104003.1:3070232-3071697:plus</td>
<td><a href="https://ncbi.nlm.nih.gov/assembly/GCF_025231485.1">GCF_025231485.1</a></td>
</tr>
<tr class="sumtab-row">
<td><i>Ca. Methanomethylophilus</i> <i>alvus</i> Mx1201</td>
<td class="numcol">1&nbsp;466</td>
<td class="seqid-td">GCF_000300255.2:NC_020913.1:283607-285072:plus</td>
<td><a href="https://ncbi.nlm.nih.gov/assembly/GCF_000300255.2">GCF_000300255.2</a></td>
</tr>
<tr class="sumtab-row">
<td><i>Methanospirillum</i> <i>hungatei</i> GP1</td>
<td class="numcol">1&nbsp;466</td>
<td class="seqid-td">GCF_019263745.1:NZ_CP077107.1:4649-6114:plus<br>GCF_019263745.1:NZ_CP077107.1:1359562-1361027:minus<br>GCF_019263745.1:NZ_CP077107.1:1365502-1366967:minus<br>GCF_019263745.1:NZ_CP077107.1:1986020-1987485:minus</td>
<td><a href="https://ncbi.nlm.nih.gov/assembly/GCF_019263745.1">GCF_019263745.1</a></td>
</tr>
<tr class="sumtab-row">
<td><i>Natronomonas</i> <i>marina</i> ZY43</td>
<td class="numcol">1&nbsp;466</td>
<td class="seqid-td">GCF_024298905.1:NZ_CP101154.1:18680-20145:plus</td>
<td><a href="https://ncbi.nlm.nih.gov/assembly/GCF_024298905.1">GCF_024298905.1</a></td>
</tr>
<tr class="sumtab-row">
<td><i>Methanospirillum</i> <i>hungatei</i> JF-1</td>
<td class="numcol">1&nbsp;466</td>
<td class="seqid-td">GCF_000013445.1:NC_007796.1:39814-41279:plus<br>GCF_000013445.1:NC_007796.1:1301079-1302544:minus<br>GCF_000013445.1:NC_007796.1:3501525-3502990:minus<br>GCF_000013445.1:NC_007796.1:3507609-3509074:minus</td>
<td><a href="https://ncbi.nlm.nih.gov/assembly/GCF_000013445.1">GCF_000013445.1</a></td>
</tr>
<tr class="sumtab-row">
<td><i>Ca. Methanomethylophilus</i><br><i>alvus</i> MGYG-HGUT-02456</td>
<td class="numcol">1&nbsp;466</td>
<td class="seqid-td">GCF_902387285.1:NZ_LR699000.1:283607-285072:plus</td>
<td><a href="https://ncbi.nlm.nih.gov/assembly/GCF_902387285.1">GCF_902387285.1</a></td>
</tr>
<tr class="sumtab-row">
<td><i>Natronomonas</i> <i>halophila</i> C90</td>
<td class="numcol">1&nbsp;466</td>
<td class="seqid-td">GCF_013391085.1:NZ_CP058334.1:1530622-1532087:minus</td>
<td><a href="https://ncbi.nlm.nih.gov/assembly/GCF_013391085.1">GCF_013391085.1</a></td>
</tr>
<tr class="sumtab-row">
<td><i>Methanospirillum</i> sp. J.3.6.1-F.2.7.3</td>
<td class="numcol">1&nbsp;466</td>
<td class="seqid-td">GCF_018502485.1:NZ_CP075546.1:133354-134819:plus<br>GCF_018502485.1:NZ_CP075546.1:825954-827419:plus<br>GCF_018502485.1:NZ_CP075546.1:872641-874106:plus<br>GCF_018502485.1:NZ_CP075546.1:1727419-1728884:plus</td>
<td><a href="https://ncbi.nlm.nih.gov/assembly/GCF_018502485.1">GCF_018502485.1</a></td>
</tr>
</tbody>
</table>
<p><sup>*</sup> SeqID — гэта ідэнтыфікатар паслядоўнасці RiboGrove.</p>
<div id="top-copy-num" class="pad-anchor"></div>
<table class="sum-table">
<caption>Топ-10 геномаў з найбольшай колькасцю копій генаў 16S рРНК</caption>
<tbody>
<tr><th class="alnleft">Арганізм</th><th class="numcol">Колькасць копій</th><th class="alnleft">Код доступу<br>геномнай зборкі</th></tr>
<tr>
<td colspan="4" class="subhead">Бактэрыі</td>
</tr>
<tr class="sumtab-row">
<td><i>Tumebacillus</i> <i>avium</i> AR23208</td>
<td class="numcol">37</td>
<td><a href="https://ncbi.nlm.nih.gov/assembly/GCF_002162355.1">GCF_002162355.1</a></td>
</tr>
<tr class="sumtab-row">
<td><i>Tumebacillus</i> <i>algifaecis</i> THMBR28</td>
<td class="numcol">27</td>
<td><a href="https://ncbi.nlm.nih.gov/assembly/GCF_002243515.1">GCF_002243515.1</a></td>
</tr>
<tr class="sumtab-row">
<td><i>Photobacterium</i> <i>phosphoreum</i> MIP2473</td>
<td class="numcol">24</td>
<td><a href="https://ncbi.nlm.nih.gov/assembly/GCF_949787665.1">GCF_949787665.1</a></td>
</tr>
<tr class="sumtab-row">
<td><i>Peribacillus</i> <i>asahii</i> KF4</td>
<td class="numcol">21</td>
<td><a href="https://ncbi.nlm.nih.gov/assembly/GCF_023823975.1">GCF_023823975.1</a></td>
</tr>
<tr class="sumtab-row">
<td><i>Aneurinibacillus</i> sp. Ricciae_BoGa-3</td>
<td class="numcol">21</td>
<td><a href="https://ncbi.nlm.nih.gov/assembly/GCF_028421645.1">GCF_028421645.1</a></td>
</tr>
<tr class="sumtab-row">
<td><i>Priestia</i> <i>megaterium</i> S2</td>
<td class="numcol">21</td>
<td><a href="https://ncbi.nlm.nih.gov/assembly/GCF_012275205.1">GCF_012275205.1</a></td>
</tr>
<tr class="sumtab-row">
<td><i>Photobacterium</i> <i>damselae</i> Phdp Wu-1</td>
<td class="numcol">21</td>
<td><a href="https://ncbi.nlm.nih.gov/assembly/GCF_003130755.1">GCF_003130755.1</a></td>
</tr>
<tr class="sumtab-row">
<td><i>Photobacterium</i> <i>toruni</i> WD2103</td>
<td class="numcol">20</td>
<td><a href="https://ncbi.nlm.nih.gov/assembly/GCF_024494545.1">GCF_024494545.1</a></td>
</tr>
<tr class="sumtab-row">
<td><i>Neobacillus</i> <i>drentensis</i> JC05</td>
<td class="numcol">20</td>
<td><a href="https://ncbi.nlm.nih.gov/assembly/GCF_021560175.1">GCF_021560175.1</a></td>
</tr>
<tr class="sumtab-row">
<td><i>Photobacterium</i> <i>damselae</i> CSP DAM1</td>
<td class="numcol">20</td>
<td><a href="https://ncbi.nlm.nih.gov/assembly/GCF_021766015.1">GCF_021766015.1</a></td>
</tr>
<tr class="sumtab-row">
<td><i>Photobacterium</i> <i>damselae</i> AS-15-3942-9</td>
<td class="numcol">20</td>
<td><a href="https://ncbi.nlm.nih.gov/assembly/GCF_021768365.1">GCF_021768365.1</a></td>
</tr>
<tr class="sumtab-row">
<td><i>Photobacterium</i> <i>damselae</i> AS-16-0963-1</td>
<td class="numcol">20</td>
<td><a href="https://ncbi.nlm.nih.gov/assembly/GCF_021768345.1">GCF_021768345.1</a></td>
</tr>
<tr class="sumtab-row">
<td><i>Neobacillus</i> sp. SuZ13</td>
<td class="numcol">20</td>
<td><a href="https://ncbi.nlm.nih.gov/assembly/GCF_030123365.1">GCF_030123365.1</a></td>
</tr>
<tr class="sumtab-row">
<td><i>Photobacterium</i> <i>damselae</i> RM-71</td>
<td class="numcol">20</td>
<td><a href="https://ncbi.nlm.nih.gov/assembly/GCF_001708035.2">GCF_001708035.2</a></td>
</tr>
<tr class="sumtab-row">
<td><i>Photobacterium</i> <i>damselae</i> AS-15-3942-7</td>
<td class="numcol">20</td>
<td><a href="https://ncbi.nlm.nih.gov/assembly/GCF_021768405.1">GCF_021768405.1</a></td>
</tr>
<tr class="sumtab-row">
<td><i>Photobacterium</i> <i>damselae</i> 04Ya311</td>
<td class="numcol">20</td>
<td><a href="https://ncbi.nlm.nih.gov/assembly/GCF_026001825.1">GCF_026001825.1</a></td>
</tr>
<tr class="sumtab-row">
<td><i>Photobacterium</i> <i>damselae</i> XP-11</td>
<td class="numcol">20</td>
<td><a href="https://ncbi.nlm.nih.gov/assembly/GCF_023973125.1">GCF_023973125.1</a></td>
</tr>
<tr class="sumtab-row">
<td><i>Photobacterium</i> <i>damselae</i> AS-15-0759-2</td>
<td class="numcol">20</td>
<td><a href="https://ncbi.nlm.nih.gov/assembly/GCF_021768425.1">GCF_021768425.1</a></td>
</tr>
<tr class="sumtab-row">
<td><i>Photobacterium</i> <i>damselae</i> 9046-81</td>
<td class="numcol">20</td>
<td><a href="https://ncbi.nlm.nih.gov/assembly/GCF_009763125.1">GCF_009763125.1</a></td>
</tr>
<tr class="sumtab-row">
<td><i>Photobacterium</i> <i>damselae</i> CSP DAM2</td>
<td class="numcol">20</td>
<td><a href="https://ncbi.nlm.nih.gov/assembly/GCF_021765875.1">GCF_021765875.1</a></td>
</tr>
<tr class="sumtab-row">
<td><i>Moritella</i> sp. 5</td>
<td class="numcol">20</td>
<td><a href="https://ncbi.nlm.nih.gov/assembly/GCF_018219455.1">GCF_018219455.1</a></td>
</tr>
<tr class="sumtab-row">
<td><i>Moritella</i> sp. 28</td>
<td class="numcol">20</td>
<td><a href="https://ncbi.nlm.nih.gov/assembly/GCF_018219435.1">GCF_018219435.1</a></td>
</tr>
<tr class="sumtab-row">
<td><i>Moritella</i> sp. 36</td>
<td class="numcol">20</td>
<td><a href="https://ncbi.nlm.nih.gov/assembly/GCF_018219415.1">GCF_018219415.1</a></td>
</tr>
<tr>
<td colspan="4" class="subhead">Археі</td>
</tr>
<tr class="sumtab-row">
<td><i>Natronorubrum</i> <i>bangense</i> JCM 10635</td>
<td class="numcol">5</td>
<td><a href="https://ncbi.nlm.nih.gov/assembly/GCF_004799645.1">GCF_004799645.1</a></td>
</tr>
<tr class="sumtab-row">
<td><i>Natrinema</i> sp. SYSU A 869</td>
<td class="numcol">5</td>
<td><a href="https://ncbi.nlm.nih.gov/assembly/GCF_019879105.1">GCF_019879105.1</a></td>
</tr>
<tr class="sumtab-row">
<td><i>Methanococcoides</i> <i>orientis</i> LMO-1</td>
<td class="numcol">5</td>
<td><a href="https://ncbi.nlm.nih.gov/assembly/GCF_021184045.1">GCF_021184045.1</a></td>
</tr>
<tr class="sumtab-row">
<td><i>Methanoplanus</i> <i>endosymbiosus</i> DSM 3599</td>
<td class="numcol">5</td>
<td><a href="https://ncbi.nlm.nih.gov/assembly/GCF_024662215.1">GCF_024662215.1</a></td>
</tr>
<tr class="sumtab-row">
<td><i>Natronorubrum</i> <i>aibiense</i> 7-3</td>
<td class="numcol">5</td>
<td><a href="https://ncbi.nlm.nih.gov/assembly/GCF_009392895.1">GCF_009392895.1</a></td>
</tr>
<tr class="sumtab-row">
<td><i>Natronococcus</i> <i>occultus</i> SP4</td>
<td class="numcol">4</td>
<td><a href="https://ncbi.nlm.nih.gov/assembly/GCF_000328685.1">GCF_000328685.1</a></td>
</tr>
<tr class="sumtab-row">
<td><i>Methanosphaera</i> <i>stadtmanae</i> DSM 3091</td>
<td class="numcol">4</td>
<td><a href="https://ncbi.nlm.nih.gov/assembly/GCF_000012545.1">GCF_000012545.1</a></td>
</tr>
<tr class="sumtab-row">
<td><i>Methanococcus</i> <i>vannielii</i> SB</td>
<td class="numcol">4</td>
<td><a href="https://ncbi.nlm.nih.gov/assembly/GCF_000017165.1">GCF_000017165.1</a></td>
</tr>
<tr class="sumtab-row">
<td><i>Methanospirillum</i> <i>hungatei</i> JF-1</td>
<td class="numcol">4</td>
<td><a href="https://ncbi.nlm.nih.gov/assembly/GCF_000013445.1">GCF_000013445.1</a></td>
</tr>
<tr class="sumtab-row">
<td><i>Haloterrigena</i> <i>salifodinae</i> BOL5-1</td>
<td class="numcol">4</td>
<td><a href="https://ncbi.nlm.nih.gov/assembly/GCF_016906025.1">GCF_016906025.1</a></td>
</tr>
<tr class="sumtab-row">
<td><i>Methanosphaera</i> <i>stadtmanae</i><br>MGYG-HGUT-02164</td>
<td class="numcol">4</td>
<td><a href="https://ncbi.nlm.nih.gov/assembly/GCF_902384015.1">GCF_902384015.1</a></td>
</tr>
<tr class="sumtab-row">
<td><i>Haloarcula</i> <i>sinaiiensis</i> ATCC 33800</td>
<td class="numcol">4</td>
<td><a href="https://ncbi.nlm.nih.gov/assembly/GCF_018200015.1">GCF_018200015.1</a></td>
</tr>
<tr class="sumtab-row">
<td><i>Methanogenium</i> <i>organophilum</i> DSM 3596</td>
<td class="numcol">4</td>
<td><a href="https://ncbi.nlm.nih.gov/assembly/GCF_026684035.1">GCF_026684035.1</a></td>
</tr>
<tr class="sumtab-row">
<td><i>Halomicrobium</i> <i>salinisoli</i> TH30</td>
<td class="numcol">4</td>
<td><a href="https://ncbi.nlm.nih.gov/assembly/GCF_020405245.1">GCF_020405245.1</a></td>
</tr>
<tr class="sumtab-row">
<td><i>Methanogenium</i> sp. S4BF</td>
<td class="numcol">4</td>
<td><a href="https://ncbi.nlm.nih.gov/assembly/GCF_029633965.1">GCF_029633965.1</a></td>
</tr>
<tr class="sumtab-row">
<td><i>Methanospirillum</i> sp. J.3.6.1-F.2.7.3</td>
<td class="numcol">4</td>
<td><a href="https://ncbi.nlm.nih.gov/assembly/GCF_018502485.1">GCF_018502485.1</a></td>
</tr>
<tr class="sumtab-row">
<td><i>Halosiccatus</i> <i>urmianus</i> IBRC-M: 10911</td>
<td class="numcol">4</td>
<td><a href="https://ncbi.nlm.nih.gov/assembly/GCF_020217425.1">GCF_020217425.1</a></td>
</tr>
<tr class="sumtab-row">
<td><i>Halomicrobium</i> <i>salinisoli</i> LT50</td>
<td class="numcol">4</td>
<td><a href="https://ncbi.nlm.nih.gov/assembly/GCF_020405185.1">GCF_020405185.1</a></td>
</tr>
<tr class="sumtab-row">
<td><i>Methanospirillum</i> <i>hungatei</i> GP1</td>
<td class="numcol">4</td>
<td><a href="https://ncbi.nlm.nih.gov/assembly/GCF_019263745.1">GCF_019263745.1</a></td>
</tr>
</tbody>
</table>
<div id="top-var" class="pad-anchor"></div>
<table class="sum-table">
<caption>Топ-10 геномаў з найбольшай зменлівасцю генаў 16S рРНК</caption>
<tbody>
<tr><th class="alnleft">Арганізм</th><th class="numcol">Сума энтрапіі<sup> *</sup>, біты</th><th class="numcol">Сярэдняя энтрапія<sup> *</sup>, біты</th><th class="numcol">Колькасць зменлівых пазіцый</th><th class="numcol">Колькасць копій генаў</th><th class="alnleft">Код доступу<br>геномнай зборкі</th></tr>
<tr>
<td colspan="6" class="subhead">Бактэрыі</td>
</tr>
<tr class="sumtab-row">
<td><i>Escherichia</i> <i>coli</i> P276M</td>
<td class="numcol">433,81</td>
<td class="numcol">0,26</td>
<td class="numcol">569</td>
<td class="numcol">6</td>
<td><a href="https://www.ncbi.nlm.nih.gov/assembly/GCF_009762385.1">GCF_009762385.1</a></td>
</tr>
<tr class="sumtab-row">
<td><i>Listeria</i> <i>monocytogenes</i><br>10-092876-1155 LM6</td>
<td class="numcol">357,10</td>
<td class="numcol">0,20</td>
<td class="numcol">370</td>
<td class="numcol">3</td>
<td><a href="https://www.ncbi.nlm.nih.gov/assembly/GCF_001999045.1">GCF_001999045.1</a></td>
</tr>
<tr class="sumtab-row">
<td><i>Klebsiella</i> <i>pneumoniae</i> GZ-1</td>
<td class="numcol">304,27</td>
<td class="numcol">0,18</td>
<td class="numcol">464</td>
<td class="numcol">8</td>
<td><a href="https://www.ncbi.nlm.nih.gov/assembly/GCF_014854815.1">GCF_014854815.1</a></td>
</tr>
<tr class="sumtab-row">
<td><i>Streptococcus</i> <i>infantis</i> SO</td>
<td class="numcol">291,50</td>
<td class="numcol">0,18</td>
<td class="numcol">308</td>
<td class="numcol">3</td>
<td><a href="https://www.ncbi.nlm.nih.gov/assembly/GCF_021497965.1">GCF_021497965.1</a></td>
</tr>
<tr class="sumtab-row">
<td><i>Bacillus</i> <i>pumilus</i> EB130</td>
<td class="numcol">272,85</td>
<td class="numcol">0,16</td>
<td class="numcol">427</td>
<td class="numcol">8</td>
<td><a href="https://www.ncbi.nlm.nih.gov/assembly/GCF_019710455.1">GCF_019710455.1</a></td>
</tr>
<tr class="sumtab-row">
<td><i>Synechococcus</i> sp. NB0720_010</td>
<td class="numcol">243,35</td>
<td class="numcol">0,16</td>
<td class="numcol">265</td>
<td class="numcol">3</td>
<td><a href="https://www.ncbi.nlm.nih.gov/assembly/GCF_023078835.1">GCF_023078835.1</a></td>
</tr>
<tr class="sumtab-row">
<td><i>Caminibacter</i> <i>mediatlanticus</i> TB-2</td>
<td class="numcol">228,78</td>
<td class="numcol">0,15</td>
<td class="numcol">282</td>
<td class="numcol">4</td>
<td><a href="https://www.ncbi.nlm.nih.gov/assembly/GCF_005843985.1">GCF_005843985.1</a></td>
</tr>
<tr class="sumtab-row">
<td><i>Xanthomonas</i> <i>oryzae</i> YNCX</td>
<td class="numcol">227,74</td>
<td class="numcol">0,15</td>
<td class="numcol">248</td>
<td class="numcol">3</td>
<td><a href="https://www.ncbi.nlm.nih.gov/assembly/GCF_024499285.1">GCF_024499285.1</a></td>
</tr>
<tr class="sumtab-row">
<td><i>Sporomusa</i> <i>termitida</i> DSM 4440</td>
<td class="numcol">226,25</td>
<td class="numcol">0,13</td>
<td class="numcol">247</td>
<td class="numcol">12</td>
<td><a href="https://www.ncbi.nlm.nih.gov/assembly/GCF_007641255.1">GCF_007641255.1</a></td>
</tr>
<tr class="sumtab-row">
<td><i>Campylobacter</i> <i>hyointestinalis</i> CHY5</td>
<td class="numcol">217,64</td>
<td class="numcol">0,13</td>
<td class="numcol">237</td>
<td class="numcol">3</td>
<td><a href="https://www.ncbi.nlm.nih.gov/assembly/GCF_013372165.1">GCF_013372165.1</a></td>
</tr>
<tr>
<td colspan="6" class="subhead">Археі</td>
</tr>
<tr class="sumtab-row">
<td><i>Halomicrobium</i> sp. ZPS1 <sup>**</sup></td>
<td class="numcol">137,00</td>
<td class="numcol">0,09</td>
<td class="numcol">137</td>
<td class="numcol">2</td>
<td><a href="https://www.ncbi.nlm.nih.gov/assembly/GCF_009217585.1">GCF_009217585.1</a></td>
</tr>
<tr class="sumtab-row">
<td><i>Halosiccatus</i> <i>urmianus</i> IBRC-M: 10911</td>
<td class="numcol">131,55</td>
<td class="numcol">0,09</td>
<td class="numcol">146</td>
<td class="numcol">4</td>
<td><a href="https://www.ncbi.nlm.nih.gov/assembly/GCF_020217425.1">GCF_020217425.1</a></td>
</tr>
<tr class="sumtab-row">
<td><i>Halapricum</i> <i>desulfuricans</i> HSR12-2</td>
<td class="numcol">128,00</td>
<td class="numcol">0,09</td>
<td class="numcol">128</td>
<td class="numcol">2</td>
<td><a href="https://www.ncbi.nlm.nih.gov/assembly/GCF_017094525.1">GCF_017094525.1</a></td>
</tr>
<tr class="sumtab-row">
<td><i>Halomicrobium</i> <i>salinisoli</i> TH30</td>
<td class="numcol">127,74</td>
<td class="numcol">0,09</td>
<td class="numcol">145</td>
<td class="numcol">4</td>
<td><a href="https://www.ncbi.nlm.nih.gov/assembly/GCF_020405245.1">GCF_020405245.1</a></td>
</tr>
<tr class="sumtab-row">
<td><i>Halapricum</i> <i>desulfuricans</i> HSR-Bgl</td>
<td class="numcol">127,00</td>
<td class="numcol">0,09</td>
<td class="numcol">127</td>
<td class="numcol">2</td>
<td><a href="https://www.ncbi.nlm.nih.gov/assembly/GCF_017094445.1">GCF_017094445.1</a></td>
</tr>
<tr class="sumtab-row">
<td><i>Halomicrobium</i> <i>mukohataei</i> JP60</td>
<td class="numcol">125,81</td>
<td class="numcol">0,09</td>
<td class="numcol">137</td>
<td class="numcol">3</td>
<td><a href="https://www.ncbi.nlm.nih.gov/assembly/GCF_004803735.1">GCF_004803735.1</a></td>
</tr>
<tr class="sumtab-row">
<td><i>Halomicrobium</i> <i>salinisoli</i> LT50</td>
<td class="numcol">123,31</td>
<td class="numcol">0,08</td>
<td class="numcol">140</td>
<td class="numcol">4</td>
<td><a href="https://www.ncbi.nlm.nih.gov/assembly/GCF_020405185.1">GCF_020405185.1</a></td>
</tr>
<tr class="sumtab-row">
<td><i>Halapricum</i> <i>desulfuricans</i> HSR-Est</td>
<td class="numcol">111,00</td>
<td class="numcol">0,08</td>
<td class="numcol">111</td>
<td class="numcol">2</td>
<td><a href="https://www.ncbi.nlm.nih.gov/assembly/GCF_017094465.1">GCF_017094465.1</a></td>
</tr>
<tr class="sumtab-row">
<td><i>Halapricum</i> <i>desulfuricans</i> HSR12-1</td>
<td class="numcol">109,00</td>
<td class="numcol">0,07</td>
<td class="numcol">109</td>
<td class="numcol">2</td>
<td><a href="https://www.ncbi.nlm.nih.gov/assembly/GCF_017094505.1">GCF_017094505.1</a></td>
</tr>
<tr class="sumtab-row">
<td><i>Halorussus</i> sp. XZYJT49</td>
<td class="numcol">105,10</td>
<td class="numcol">0,07</td>
<td class="numcol">113</td>
<td class="numcol">3</td>
<td><a href="https://www.ncbi.nlm.nih.gov/assembly/GCF_023238205.1">GCF_023238205.1</a></td>
</tr>
</tbody>
</table>
<p><sup>*</sup> Энтрапія ў гэтым выпадку — гэта <a href="https://ru.wikipedia.org/wiki/Информационная_энтропия">энтрапія Шэнана</a>, якую вылічалі для кожнай калонкі множнага выраўноўвання поўнапамерных паслядоўнасцяў генаў 16S рРНК кожнага геному. Пасля гэтага падсумоўвалі энтрапію (калонка «Сума энтрапіі») і усярэднівалі (калонка «Сярэдняя энтрапія»).</p>
<p><sup>**</sup> <i>Halomicrobium</i> sp. ZPS1 — паказальны выпадак. У гэтым геноме два гены 16S рРНК, таму энтрапія роўная колькасці нуклеатыдаў, якія не супадаюць у іхніх паслядоўнасцях. Вылічыўшы адсотак ідэнтычнасці (скарочана АІ) паслядоўнасцяў гэтых двух генаў, атрымаем 90,70%! Нагадаем, што за парог АІ паслядоўнасцяў генаў 16S рРНК для аднясення арганізмаў да адрозных <em>родаў</em> звычайна бярэцца (зрэшты, адвольна) значэнне 95%.</p>
<div id="primers-coverages" class="pad-anchor"></div>
<table class="sum-table"><caption>Спектр дзеяння<sup>*</sup> пар праймераў да розных V-регіёнаў бактэрыяльных генаў 16S рРНК</caption>
<tbody class="primer-cov-tbody">
<tr>
  <th class="alnleft" rowspan="2">Тып<br />бактэрый</th>
  <th class="numcol" rowspan="2">Колькасць<br />геномаў</th>
  <th class="numcol">Поўны<br />ген</th>
  <th class="numcol">V1–V2</th>
  <th class="numcol">V1–V3</th>
  <th class="numcol">V3–V4</th>
  <th class="numcol">V3–V5</th>
  <th class="numcol">V4</th>
  <th class="numcol">V4–V5</th>
  <th class="numcol">V4–V6</th>
  <th class="numcol">V5–V6</th>
  <th class="numcol">V5–V7</th>
  <th class="numcol">V6–V7</th>
  <th class="numcol">V6–V8</th>
</tr>
<tr>
  <th class="numcol">27F–1492R<br />(%)</th>
  <th class="numcol">27F–338R<br />(%)</th>
  <th class="numcol">27F–534R<br />(%)</th>
  <th class="numcol">341F–785R<br />(%)</th>
  <th class="numcol">341F–944R<br />(%)</th>
  <th class="numcol">515F–806R<br />(%)</th>
  <th class="numcol">515F–944R<br />(%)</th>
  <th class="numcol">515F–1100R<br />(%)</th>
  <th class="numcol">784F–1100R<br />(%)</th>
  <th class="numcol">784F–1193R<br />(%)</th>
  <th class="numcol">939F–1193R<br />(%)</th>
  <th class="numcol">939F–1378R<br />(%)</th>
</tr>
<tr class="sumtab-row">
  <td><i>Pseudomonadota</i></td>
  <td class="numcol">18&nbsp;897</td>
  <td class="numcol">99,67</td>
  <td class="numcol">99,48</td>
  <td class="numcol">99,64</td>
  <td class="numcol">99,94</td>
  <td class="numcol">82,00</td>
  <td class="numcol">99,86</td>
  <td class="numcol">82,05</td>
  <td class="numcol">89,45</td>
  <td class="numcol">89,17</td>
  <td class="numcol">93,47</td>
  <td class="numcol">92,37</td>
  <td class="numcol">96,64</td>
</tr>
<tr class="sumtab-row">
  <td><i>Bacillota</i></td>
  <td class="numcol">7&nbsp;667</td>
  <td class="numcol">99,84</td>
  <td class="numcol">99,73</td>
  <td class="numcol">99,80</td>
  <td class="numcol">99,95</td>
  <td class="numcol">95,90</td>
  <td class="numcol">99,99</td>
  <td class="numcol">95,83</td>
  <td class="numcol">99,53</td>
  <td class="numcol">98,07</td>
  <td class="numcol">97,43</td>
  <td class="numcol">98,63</td>
  <td class="numcol">99,36</td>
</tr>
<tr class="sumtab-row">
  <td><i>Actinomycetota</i></td>
  <td class="numcol">3&nbsp;253</td>
  <td class="numcol">99,82</td>
  <td class="numcol">98,86</td>
  <td class="numcol">99,60</td>
  <td class="numcol">93,48</td>
  <td class="numcol">62,62</td>
  <td class="numcol">93,24</td>
  <td class="numcol">62,40</td>
  <td class="numcol">95,88</td>
  <td class="numcol">99,66</td>
  <td class="numcol">99,75</td>
  <td class="numcol">99,78</td>
  <td class="numcol">96,93</td>
</tr>
<tr class="sumtab-row">
  <td><i>Bacteroidota</i></td>
  <td class="numcol">1&nbsp;270</td>
  <td class="numcol">96,06</td>
  <td class="numcol">95,43</td>
  <td class="numcol">95,83</td>
  <td class="numcol">99,69</td>
  <td class="numcol">61,73</td>
  <td class="numcol">99,13</td>
  <td class="numcol">61,34</td>
  <td class="numcol">38,43</td>
  <td class="numcol">38,66</td>
  <td class="numcol">94,65</td>
  <td class="numcol">92,28</td>
  <td class="numcol">94,72</td>
</tr>
<tr class="sumtab-row">
  <td><i>Campylobacterota</i></td>
  <td class="numcol">984</td>
  <td class="numcol">100,00</td>
  <td class="numcol">100,00</td>
  <td class="numcol">100,00</td>
  <td class="numcol">100,00</td>
  <td class="numcol">100,00</td>
  <td class="numcol">99,90</td>
  <td class="numcol">99,90</td>
  <td class="numcol">99,90</td>
  <td class="numcol">99,59</td>
  <td class="numcol">99,59</td>
  <td class="numcol">99,59</td>
  <td class="numcol">99,39</td>
</tr>
<tr class="sumtab-row">
  <td><i>Mycoplasmatota</i></td>
  <td class="numcol">533</td>
  <td class="numcol">97,75</td>
  <td class="numcol">94,93</td>
  <td class="numcol">75,98</td>
  <td class="numcol">98,12</td>
  <td class="numcol">90,62</td>
  <td class="numcol">98,31</td>
  <td class="numcol">91,18</td>
  <td class="numcol">70,54</td>
  <td class="numcol">41,84</td>
  <td class="numcol">42,59</td>
  <td class="numcol">79,55</td>
  <td class="numcol">0,38</td>
</tr>
<tr class="sumtab-row">
  <td><i>Spirochaetota</i></td>
  <td class="numcol">368</td>
  <td class="numcol">50,54</td>
  <td class="numcol">51,09</td>
  <td class="numcol">51,09</td>
  <td class="numcol">93,75</td>
  <td class="numcol">99,73</td>
  <td class="numcol">93,75</td>
  <td class="numcol">99,73</td>
  <td class="numcol">100,00</td>
  <td class="numcol">79,08</td>
  <td class="numcol">79,08</td>
  <td class="numcol">91,58</td>
  <td class="numcol">39,13</td>
</tr>
<tr class="sumtab-row">
  <td><i>Cyanobacteriota</i></td>
  <td class="numcol">234</td>
  <td class="numcol">99,57</td>
  <td class="numcol">100,00</td>
  <td class="numcol">99,57</td>
  <td class="numcol">100,00</td>
  <td class="numcol">5,56</td>
  <td class="numcol">100,00</td>
  <td class="numcol">5,56</td>
  <td class="numcol">100,00</td>
  <td class="numcol">1,71</td>
  <td class="numcol">1,71</td>
  <td class="numcol">100,00</td>
  <td class="numcol">99,57</td>
</tr>
<tr class="sumtab-row">
  <td><i>Chlamydiota</i></td>
  <td class="numcol">190</td>
  <td class="numcol">0,00</td>
  <td class="numcol">0,00</td>
  <td class="numcol">0,00</td>
  <td class="numcol">100,00</td>
  <td class="numcol">100,00</td>
  <td class="numcol">0,00</td>
  <td class="numcol">0,00</td>
  <td class="numcol">0,00</td>
  <td class="numcol">100,00</td>
  <td class="numcol">100,00</td>
  <td class="numcol">100,00</td>
  <td class="numcol">94,21</td>
</tr>
<tr class="sumtab-row">
  <td><i>Verrucomicrobiota</i></td>
  <td class="numcol">119</td>
  <td class="numcol">99,16</td>
  <td class="numcol">0,00</td>
  <td class="numcol">99,16</td>
  <td class="numcol">100,00</td>
  <td class="numcol">10,08</td>
  <td class="numcol">100,00</td>
  <td class="numcol">10,08</td>
  <td class="numcol">100,00</td>
  <td class="numcol">1,68</td>
  <td class="numcol">1,68</td>
  <td class="numcol">98,32</td>
  <td class="numcol">98,32</td>
</tr>
<tr class="sumtab-row">
  <td><i>Thermodesulfobacteriota</i></td>
  <td class="numcol">119</td>
  <td class="numcol">100,00</td>
  <td class="numcol">99,16</td>
  <td class="numcol">100,00</td>
  <td class="numcol">100,00</td>
  <td class="numcol">45,38</td>
  <td class="numcol">100,00</td>
  <td class="numcol">45,38</td>
  <td class="numcol">100,00</td>
  <td class="numcol">94,12</td>
  <td class="numcol">89,92</td>
  <td class="numcol">95,80</td>
  <td class="numcol">99,16</td>
</tr>
<tr class="sumtab-row">
  <td><i>Fusobacteriota</i></td>
  <td class="numcol">87</td>
  <td class="numcol">100,00</td>
  <td class="numcol">96,55</td>
  <td class="numcol">100,00</td>
  <td class="numcol">100,00</td>
  <td class="numcol">100,00</td>
  <td class="numcol">100,00</td>
  <td class="numcol">100,00</td>
  <td class="numcol">100,00</td>
  <td class="numcol">98,85</td>
  <td class="numcol">98,85</td>
  <td class="numcol">100,00</td>
  <td class="numcol">0,00</td>
</tr>
<tr class="sumtab-row">
  <td><i>Deinococcota</i></td>
  <td class="numcol">80</td>
  <td class="numcol">100,00</td>
  <td class="numcol">100,00</td>
  <td class="numcol">100,00</td>
  <td class="numcol">100,00</td>
  <td class="numcol">100,00</td>
  <td class="numcol">100,00</td>
  <td class="numcol">100,00</td>
  <td class="numcol">100,00</td>
  <td class="numcol">0,00</td>
  <td class="numcol">0,00</td>
  <td class="numcol">51,25</td>
  <td class="numcol">100,00</td>
</tr>
<tr class="sumtab-row">
  <td><i>Planctomycetota</i></td>
  <td class="numcol">59</td>
  <td class="numcol">100,00</td>
  <td class="numcol">18,64</td>
  <td class="numcol">100,00</td>
  <td class="numcol">100,00</td>
  <td class="numcol">62,71</td>
  <td class="numcol">100,00</td>
  <td class="numcol">62,71</td>
  <td class="numcol">0,00</td>
  <td class="numcol">0,00</td>
  <td class="numcol">0,00</td>
  <td class="numcol">3,39</td>
  <td class="numcol">0,00</td>
</tr>
<tr class="sumtab-row">
  <td><i>Myxococcota</i></td>
  <td class="numcol">48</td>
  <td class="numcol">100,00</td>
  <td class="numcol">100,00</td>
  <td class="numcol">100,00</td>
  <td class="numcol">100,00</td>
  <td class="numcol">100,00</td>
  <td class="numcol">100,00</td>
  <td class="numcol">100,00</td>
  <td class="numcol">100,00</td>
  <td class="numcol">100,00</td>
  <td class="numcol">100,00</td>
  <td class="numcol">100,00</td>
  <td class="numcol">100,00</td>
</tr>
<tr class="sumtab-row">
  <td><i>Thermotogota</i></td>
  <td class="numcol">43</td>
  <td class="numcol">100,00</td>
  <td class="numcol">97,67</td>
  <td class="numcol">100,00</td>
  <td class="numcol">100,00</td>
  <td class="numcol">9,30</td>
  <td class="numcol">100,00</td>
  <td class="numcol">9,30</td>
  <td class="numcol">100,00</td>
  <td class="numcol">0,00</td>
  <td class="numcol">0,00</td>
  <td class="numcol">58,14</td>
  <td class="numcol">97,67</td>
</tr>
<tr class="sumtab-row">
  <td><i>Chloroflexota</i></td>
  <td class="numcol">41</td>
  <td class="numcol">100,00</td>
  <td class="numcol">90,24</td>
  <td class="numcol">100,00</td>
  <td class="numcol">36,59</td>
  <td class="numcol">0,00</td>
  <td class="numcol">92,68</td>
  <td class="numcol">0,00</td>
  <td class="numcol">87,80</td>
  <td class="numcol">4,88</td>
  <td class="numcol">4,88</td>
  <td class="numcol">92,68</td>
  <td class="numcol">26,83</td>
</tr>
<tr class="sumtab-row">
  <td><i>Acidobacteriota</i></td>
  <td class="numcol">32</td>
  <td class="numcol">96,88</td>
  <td class="numcol">96,88</td>
  <td class="numcol">96,88</td>
  <td class="numcol">100,00</td>
  <td class="numcol">100,00</td>
  <td class="numcol">100,00</td>
  <td class="numcol">100,00</td>
  <td class="numcol">100,00</td>
  <td class="numcol">62,50</td>
  <td class="numcol">46,88</td>
  <td class="numcol">84,38</td>
  <td class="numcol">100,00</td>
</tr>
<tr class="sumtab-row">
  <td><i>Bdellovibrionota</i></td>
  <td class="numcol">21</td>
  <td class="numcol">100,00</td>
  <td class="numcol">100,00</td>
  <td class="numcol">100,00</td>
  <td class="numcol">100,00</td>
  <td class="numcol">66,67</td>
  <td class="numcol">100,00</td>
  <td class="numcol">66,67</td>
  <td class="numcol">100,00</td>
  <td class="numcol">100,00</td>
  <td class="numcol">100,00</td>
  <td class="numcol">100,00</td>
  <td class="numcol">100,00</td>
</tr>
<tr class="sumtab-row">
  <td><i>Chlorobiota</i></td>
  <td class="numcol">15</td>
  <td class="numcol">100,00</td>
  <td class="numcol">100,00</td>
  <td class="numcol">100,00</td>
  <td class="numcol">100,00</td>
  <td class="numcol">0,00</td>
  <td class="numcol">0,00</td>
  <td class="numcol">0,00</td>
  <td class="numcol">0,00</td>
  <td class="numcol">100,00</td>
  <td class="numcol">93,33</td>
  <td class="numcol">86,67</td>
  <td class="numcol">6,67</td>
</tr>
<tr class="sumtab-row">
  <td><i>Aquificota</i></td>
  <td class="numcol">15</td>
  <td class="numcol">100,00</td>
  <td class="numcol">20,00</td>
  <td class="numcol">100,00</td>
  <td class="numcol">100,00</td>
  <td class="numcol">20,00</td>
  <td class="numcol">100,00</td>
  <td class="numcol">20,00</td>
  <td class="numcol">100,00</td>
  <td class="numcol">0,00</td>
  <td class="numcol">0,00</td>
  <td class="numcol">6,67</td>
  <td class="numcol">20,00</td>
</tr>
<tr class="sumtab-row">
  <td><i>Rhodothermota</i></td>
  <td class="numcol">12</td>
  <td class="numcol">33,33</td>
  <td class="numcol">33,33</td>
  <td class="numcol">33,33</td>
  <td class="numcol">100,00</td>
  <td class="numcol">100,00</td>
  <td class="numcol">100,00</td>
  <td class="numcol">100,00</td>
  <td class="numcol">100,00</td>
  <td class="numcol">100,00</td>
  <td class="numcol">100,00</td>
  <td class="numcol">100,00</td>
  <td class="numcol">100,00</td>
</tr>
<tr class="sumtab-row">
  <td><i>Nitrospirota</i></td>
  <td class="numcol">11</td>
  <td class="numcol">100,00</td>
  <td class="numcol">100,00</td>
  <td class="numcol">100,00</td>
  <td class="numcol">100,00</td>
  <td class="numcol">63,64</td>
  <td class="numcol">100,00</td>
  <td class="numcol">63,64</td>
  <td class="numcol">100,00</td>
  <td class="numcol">100,00</td>
  <td class="numcol">63,64</td>
  <td class="numcol">63,64</td>
  <td class="numcol">100,00</td>
</tr>
<tr class="sumtab-row">
  <td><i>Ca. Saccharibacteria</i></td>
  <td class="numcol">7</td>
  <td class="numcol">100,00</td>
  <td class="numcol">100,00</td>
  <td class="numcol">100,00</td>
  <td class="numcol">100,00</td>
  <td class="numcol">14,29</td>
  <td class="numcol">14,29</td>
  <td class="numcol">14,29</td>
  <td class="numcol">14,29</td>
  <td class="numcol">0,00</td>
  <td class="numcol">0,00</td>
  <td class="numcol">100,00</td>
  <td class="numcol">100,00</td>
</tr>
<tr class="sumtab-row">
  <td><i>Synergistota</i></td>
  <td class="numcol">7</td>
  <td class="numcol">100,00</td>
  <td class="numcol">100,00</td>
  <td class="numcol">100,00</td>
  <td class="numcol">100,00</td>
  <td class="numcol">0,00</td>
  <td class="numcol">100,00</td>
  <td class="numcol">0,00</td>
  <td class="numcol">100,00</td>
  <td class="numcol">0,00</td>
  <td class="numcol">0,00</td>
  <td class="numcol">100,00</td>
  <td class="numcol">100,00</td>
</tr>
<tr class="sumtab-row">
  <td><i>Deferribacterota</i></td>
  <td class="numcol">6</td>
  <td class="numcol">100,00</td>
  <td class="numcol">100,00</td>
  <td class="numcol">100,00</td>
  <td class="numcol">100,00</td>
  <td class="numcol">0,00</td>
  <td class="numcol">100,00</td>
  <td class="numcol">0,00</td>
  <td class="numcol">100,00</td>
  <td class="numcol">100,00</td>
  <td class="numcol">100,00</td>
  <td class="numcol">100,00</td>
  <td class="numcol">100,00</td>
</tr>
<tr class="sumtab-row">
  <td><i>Elusimicrobiota</i></td>
  <td class="numcol">4</td>
  <td class="numcol">100,00</td>
  <td class="numcol">50,00</td>
  <td class="numcol">100,00</td>
  <td class="numcol">100,00</td>
  <td class="numcol">0,00</td>
  <td class="numcol">100,00</td>
  <td class="numcol">0,00</td>
  <td class="numcol">100,00</td>
  <td class="numcol">75,00</td>
  <td class="numcol">75,00</td>
  <td class="numcol">100,00</td>
  <td class="numcol">100,00</td>
</tr>
<tr class="sumtab-row">
  <td><i>Gemmatimonadota</i></td>
  <td class="numcol">4</td>
  <td class="numcol">100,00</td>
  <td class="numcol">100,00</td>
  <td class="numcol">100,00</td>
  <td class="numcol">100,00</td>
  <td class="numcol">100,00</td>
  <td class="numcol">100,00</td>
  <td class="numcol">100,00</td>
  <td class="numcol">100,00</td>
  <td class="numcol">100,00</td>
  <td class="numcol">100,00</td>
  <td class="numcol">100,00</td>
  <td class="numcol">100,00</td>
</tr>
<tr class="sumtab-row">
  <td><i>Chrysiogenota</i></td>
  <td class="numcol">2</td>
  <td class="numcol">100,00</td>
  <td class="numcol">100,00</td>
  <td class="numcol">100,00</td>
  <td class="numcol">100,00</td>
  <td class="numcol">100,00</td>
  <td class="numcol">100,00</td>
  <td class="numcol">100,00</td>
  <td class="numcol">100,00</td>
  <td class="numcol">100,00</td>
  <td class="numcol">100,00</td>
  <td class="numcol">100,00</td>
  <td class="numcol">100,00</td>
</tr>
<tr class="sumtab-row">
  <td><i>Ignavibacteriota</i></td>
  <td class="numcol">2</td>
  <td class="numcol">100,00</td>
  <td class="numcol">100,00</td>
  <td class="numcol">100,00</td>
  <td class="numcol">100,00</td>
  <td class="numcol">100,00</td>
  <td class="numcol">100,00</td>
  <td class="numcol">100,00</td>
  <td class="numcol">100,00</td>
  <td class="numcol">100,00</td>
  <td class="numcol">100,00</td>
  <td class="numcol">100,00</td>
  <td class="numcol">100,00</td>
</tr>
<tr class="sumtab-row">
  <td><i>Kiritimatiellota</i></td>
  <td class="numcol">2</td>
  <td class="numcol">100,00</td>
  <td class="numcol">100,00</td>
  <td class="numcol">100,00</td>
  <td class="numcol">100,00</td>
  <td class="numcol">100,00</td>
  <td class="numcol">100,00</td>
  <td class="numcol">100,00</td>
  <td class="numcol">100,00</td>
  <td class="numcol">0,00</td>
  <td class="numcol">0,00</td>
  <td class="numcol">100,00</td>
  <td class="numcol">100,00</td>
</tr>
<tr class="sumtab-row">
  <td><i>Dictyoglomota</i></td>
  <td class="numcol">2</td>
  <td class="numcol">100,00</td>
  <td class="numcol">100,00</td>
  <td class="numcol">100,00</td>
  <td class="numcol">100,00</td>
  <td class="numcol">0,00</td>
  <td class="numcol">100,00</td>
  <td class="numcol">0,00</td>
  <td class="numcol">100,00</td>
  <td class="numcol">0,00</td>
  <td class="numcol">0,00</td>
  <td class="numcol">100,00</td>
  <td class="numcol">0,00</td>
</tr>
<tr class="sumtab-row">
  <td><i>Thermomicrobiota</i></td>
  <td class="numcol">2</td>
  <td class="numcol">100,00</td>
  <td class="numcol">100,00</td>
  <td class="numcol">100,00</td>
  <td class="numcol">100,00</td>
  <td class="numcol">0,00</td>
  <td class="numcol">100,00</td>
  <td class="numcol">0,00</td>
  <td class="numcol">100,00</td>
  <td class="numcol">0,00</td>
  <td class="numcol">0,00</td>
  <td class="numcol">50,00</td>
  <td class="numcol">50,00</td>
</tr>
<tr class="sumtab-row">
  <td><i>Fibrobacterota</i></td>
  <td class="numcol">2</td>
  <td class="numcol">100,00</td>
  <td class="numcol">100,00</td>
  <td class="numcol">100,00</td>
  <td class="numcol">100,00</td>
  <td class="numcol">100,00</td>
  <td class="numcol">100,00</td>
  <td class="numcol">100,00</td>
  <td class="numcol">100,00</td>
  <td class="numcol">100,00</td>
  <td class="numcol">100,00</td>
  <td class="numcol">100,00</td>
  <td class="numcol">100,00</td>
</tr>
<tr class="sumtab-row">
  <td><i>Ca. Omnitrophota</i></td>
  <td class="numcol">1</td>
  <td class="numcol">100,00</td>
  <td class="numcol">100,00</td>
  <td class="numcol">100,00</td>
  <td class="numcol">100,00</td>
  <td class="numcol">0,00</td>
  <td class="numcol">100,00</td>
  <td class="numcol">0,00</td>
  <td class="numcol">100,00</td>
  <td class="numcol">100,00</td>
  <td class="numcol">100,00</td>
  <td class="numcol">100,00</td>
  <td class="numcol">100,00</td>
</tr>
<tr class="sumtab-row">
  <td><i>Armatimonadota</i></td>
  <td class="numcol">1</td>
  <td class="numcol">100,00</td>
  <td class="numcol">0,00</td>
  <td class="numcol">100,00</td>
  <td class="numcol">100,00</td>
  <td class="numcol">100,00</td>
  <td class="numcol">100,00</td>
  <td class="numcol">100,00</td>
  <td class="numcol">100,00</td>
  <td class="numcol">100,00</td>
  <td class="numcol">100,00</td>
  <td class="numcol">100,00</td>
  <td class="numcol">100,00</td>
</tr>
<tr class="sumtab-row">
  <td><i>Coprothermobacterota</i></td>
  <td class="numcol">1</td>
  <td class="numcol">0,00</td>
  <td class="numcol">0,00</td>
  <td class="numcol">0,00</td>
  <td class="numcol">100,00</td>
  <td class="numcol">100,00</td>
  <td class="numcol">100,00</td>
  <td class="numcol">100,00</td>
  <td class="numcol">0,00</td>
  <td class="numcol">0,00</td>
  <td class="numcol">0,00</td>
  <td class="numcol">100,00</td>
  <td class="numcol">0,00</td>
</tr>
<tr class="sumtab-row">
  <td><i>Atribacterota</i></td>
  <td class="numcol">1</td>
  <td class="numcol">100,00</td>
  <td class="numcol">100,00</td>
  <td class="numcol">100,00</td>
  <td class="numcol">100,00</td>
  <td class="numcol">100,00</td>
  <td class="numcol">100,00</td>
  <td class="numcol">100,00</td>
  <td class="numcol">100,00</td>
  <td class="numcol">0,00</td>
  <td class="numcol">0,00</td>
  <td class="numcol">100,00</td>
  <td class="numcol">100,00</td>
</tr>
<tr class="sumtab-row">
  <td><i>Ca. Bipolaricaulota</i></td>
  <td class="numcol">1</td>
  <td class="numcol">0,00</td>
  <td class="numcol">0,00</td>
  <td class="numcol">0,00</td>
  <td class="numcol">100,00</td>
  <td class="numcol">100,00</td>
  <td class="numcol">100,00</td>
  <td class="numcol">100,00</td>
  <td class="numcol">0,00</td>
  <td class="numcol">0,00</td>
  <td class="numcol">0,00</td>
  <td class="numcol">0,00</td>
  <td class="numcol">0,00</td>
</tr>
<tr class="sumtab-row">
  <td><i>Balneolota</i></td>
  <td class="numcol">1</td>
  <td class="numcol">100,00</td>
  <td class="numcol">100,00</td>
  <td class="numcol">100,00</td>
  <td class="numcol">100,00</td>
  <td class="numcol">100,00</td>
  <td class="numcol">100,00</td>
  <td class="numcol">100,00</td>
  <td class="numcol">100,00</td>
  <td class="numcol">100,00</td>
  <td class="numcol">100,00</td>
  <td class="numcol">100,00</td>
  <td class="numcol">100,00</td>
</tr>
<tr class="sumtab-row">
  <td><i>Ca. Cloacimonadota</i></td>
  <td class="numcol">1</td>
  <td class="numcol">100,00</td>
  <td class="numcol">100,00</td>
  <td class="numcol">100,00</td>
  <td class="numcol">100,00</td>
  <td class="numcol">100,00</td>
  <td class="numcol">100,00</td>
  <td class="numcol">100,00</td>
  <td class="numcol">100,00</td>
  <td class="numcol">100,00</td>
  <td class="numcol">100,00</td>
  <td class="numcol">100,00</td>
  <td class="numcol">100,00</td>
</tr>
<tr class="sumtab-row">
  <td><i>Nitrospinota</i></td>
  <td class="numcol">1</td>
  <td class="numcol">100,00</td>
  <td class="numcol">100,00</td>
  <td class="numcol">100,00</td>
  <td class="numcol">100,00</td>
  <td class="numcol">100,00</td>
  <td class="numcol">100,00</td>
  <td class="numcol">100,00</td>
  <td class="numcol">100,00</td>
  <td class="numcol">0,00</td>
  <td class="numcol">0,00</td>
  <td class="numcol">100,00</td>
  <td class="numcol">100,00</td>
</tr>
<tr class="sumtab-row">
  <td><i>Caldisericota</i></td>
  <td class="numcol">1</td>
  <td class="numcol">100,00</td>
  <td class="numcol">100,00</td>
  <td class="numcol">100,00</td>
  <td class="numcol">100,00</td>
  <td class="numcol">0,00</td>
  <td class="numcol">0,00</td>
  <td class="numcol">0,00</td>
  <td class="numcol">0,00</td>
  <td class="numcol">0,00</td>
  <td class="numcol">100,00</td>
  <td class="numcol">100,00</td>
  <td class="numcol">100,00</td>
</tr>
<tr class="sumtab-row">
  <td><i>Calditrichota</i></td>
  <td class="numcol">1</td>
  <td class="numcol">100,00</td>
  <td class="numcol">100,00</td>
  <td class="numcol">100,00</td>
  <td class="numcol">100,00</td>
  <td class="numcol">100,00</td>
  <td class="numcol">100,00</td>
  <td class="numcol">100,00</td>
  <td class="numcol">100,00</td>
  <td class="numcol">100,00</td>
  <td class="numcol">100,00</td>
  <td class="numcol">100,00</td>
  <td class="numcol">100,00</td>
</tr>
<tr class="sumtab-row">
  <td><i>Ca. Absconditabacteria</i></td>
  <td class="numcol">1</td>
  <td class="numcol">100,00</td>
  <td class="numcol">0,00</td>
  <td class="numcol">100,00</td>
  <td class="numcol">100,00</td>
  <td class="numcol">0,00</td>
  <td class="numcol">100,00</td>
  <td class="numcol">0,00</td>
  <td class="numcol">0,00</td>
  <td class="numcol">0,00</td>
  <td class="numcol">100,00</td>
  <td class="numcol">0,00</td>
  <td class="numcol">0,00</td>
</tr>
<tr class="sumtab-row">
  <td><i>Lentisphaerota</i></td>
  <td class="numcol">1</td>
  <td class="numcol">100,00</td>
  <td class="numcol">0,00</td>
  <td class="numcol">100,00</td>
  <td class="numcol">100,00</td>
  <td class="numcol">100,00</td>
  <td class="numcol">100,00</td>
  <td class="numcol">100,00</td>
  <td class="numcol">100,00</td>
  <td class="numcol">0,00</td>
  <td class="numcol">0,00</td>
  <td class="numcol">100,00</td>
  <td class="numcol">100,00</td>
</tr>
</tbody>
</table>
<p><sup>*</sup> Спектр дзеяння пары праймераў — гэта адсотак геномаў, якія маюць прынамсі адзін ген 16S рРНК, які магчыма ампліфікаваць з дапамогай ПЦР і з выкарыстаннем гэтай пары праймераў. Падрабязнасці апісаныя ў нашым <a href="https://masikol.github.io/latest_ribogrove_release_be.html#citing-ribogrove">артыкуле пра RiboGrove</a>.</p>
<p>Ёсць больш падрабязная табліца — у файле <span class="samp">primer_pair_genomic_coverage.tsv</span> у метададзеных <a href="#downloads">metadata</a>. Тая табліца змяшчае пакрыцце не толькі для тыпаў бактэрый, але і для кожнага класу, парадку, сямейства, роду ды віду. Апроч таго, тая табліца змяшчае пакрыццё для пары праймераў 1115F–1492R (участак V7–V9). У гэтай табліцы няма гэтай пары, каб табліца не была занадта вялікай.</p>
<table class="sum-table"><caption>Праймеры, спектр дзеяння якіх ацэньваўся</caption>
<tbody>
  <tr><th>Назва праймера</th><th>Паслядоўнасць</th><th>Спасылка</th></tr>
  <tr class="sumtab-row"><td>27F</td><td>AGAGTTTGATYMTGGCTCAG</td><td><a href="https://doi.org/10.1128/AEM.02272-07">Frank et al., 2008</a></td></tr>
  <tr class="sumtab-row"><td>338R</td><td>GCTGCCTCCCGTAGGAGT</td><td><a href="https://doi.org/10.1128/aem.62.2.625-630.1996">Suzuki et al., 1996</a></td></tr>
  <tr class="sumtab-row"><td>341F<sup> *</sup></td><td>CCTACGGGNGGCWGCAG</td><td><a href="https://doi.org/10.1093/nar/gks808">Klindworth et al., 2013</a></td></tr>
  <tr class="sumtab-row"><td>515F</td><td>GTGCCAGCMGCCGCGGTAA</td><td><a href="https://doi.org/10.1111/j.1550-7408.1999.tb04612.x">Turner et al., 1999</a></td></tr>
  <tr class="sumtab-row"><td>534R</td><td>ATTACCGCGGCTGCTGG</td><td><a href="https://doi.org/10.1186/s40168-015-0087-4">Walker et al., 2015</a></td></tr>
  <tr class="sumtab-row"><td>784F</td><td>AGGATTAGATACCCTGGTA</td><td><a href="https://doi.org/10.1371/journal.pone.0002836">Andersson et al., 2008</a></td></tr>
  <tr class="sumtab-row"><td>785R<sup> *</sup></td><td>GACTACHVGGGTATCTAATCC</td><td><a href="https://doi.org/10.1093/nar/gks808">Klindworth et al., 2013</a></td></tr>
  <tr class="sumtab-row"><td>806R</td><td>GGACTACHVGGGTWTCTAAT</td><td><a href="https://doi.org/10.1073/pnas.1000080107">Caporaso et al., 2010</a></td></tr>
  <tr class="sumtab-row"><td>939F</td><td>GAATTGACGGGGGCCCGCACAAG</td><td><a href="https://doi.org/10.1016/j.anaerobe.2014.04.006">Lebuhn et al., 2014</a></td></tr>
  <tr class="sumtab-row"><td>944R</td><td>GAATTAAACCACATGCTC</td><td><a href="https://doi.org/10.1186/s40168-017-0396-x">Fuks et al., 2018</a></td></tr>
  <tr class="sumtab-row"><td>1100R</td><td>AGGGTTGCGCTCGTTG</td><td><a href="https://doi.org/10.1111/j.1550-7408.1999.tb04612.x">Turner et al., 1999</a></td></tr>
  <tr class="sumtab-row"><td>1193R</td><td>ACGTCATCCCCACCTTCC</td><td><a href="https://doi.org/10.1371/journal.pone.0056329">Bodenhausen et al, 2013</a></td></tr>
  <tr class="sumtab-row"><td>1378R</td><td>CGGTGTGTACAAGGCCCGGGAACG</td><td><a href="https://doi.org/10.1016/j.anaerobe.2014.04.006">Lebuhn et al., 2014</a></td></tr>
  <tr class="sumtab-row"><td>1492R</td><td>TACCTTGTTACGACTT</td><td><a href="https://doi.org/10.1128/AEM.02272-07">Frank et al., 2008</a></td></tr>
</tbody>
</table>
<p><sup>*</sup> Праймеры 341F і 785R выкарыстоўваюцца ў <a href="https://support.illumina.com/downloads/16s_metagenomic_sequencing_library_preparation.html">пратаколе</a> падрыхтоўкі бібліятэкі для секвенавання ампліконаў регіёнаў V3–V4 генаў 16S рРНК на апараце Illumina MiSeq.</p>
<hr>
<div id="searching-data" class="pad-anchor"></div>
<h2>Пошук дадзеных у RiboGrove</h2>
<p>RiboGrove — вельмі мінімалістычная база дадзеных: яна складаецца з двух файлаў fasta з дадатковымі метададзенымі. Таму пашыраных сродкаў пошуку няма. Мы прызнае&#769;м гэтую праблему і пакідаем ніжэй некалькі парад наконт пошуку дадзеных у RiboGrove.</p>
<div id="header-format" class="pad-anchor"></div>
<h3>Фармат загалоўка</h3>
<p>Запісы паслядоўнасцяў RiboGrove у фармаце fasta маюць наступны фармат загалоўка:</p>
<p class="samp-highl samp-vwide" style="font-size:14px">&gt;GCF_000978375.1:NZ_CP009686.1:8908-10459:plus ;d__Bacteria;p__Firmicutes;c__Bacilli;o__Bacillales;f__Bacillaceae;g__Bacillus;s__cereus; category:1</p>
<p>Асноўныя часткі загалоўка адасобленыя адна ад адной прабеламі. У загалоўку павінна быць тры такія часткі:</p>
<ol>
<li>Ідэнтыфікатар паслядоўнасці (SeqID): <span class="samp-highl">GCF_000978375.1:NZ_CP009686.1:8908-10459:plus</span>. SeqID, у сваю чаргу, складаецца з чатырох частак, адасобленых двукропʼямі (<span class="samp-highl">:</span>):
<ol>
<li>Код доступу зборкі геному, з якога паходзіць паслядоўнасць гена: <span class="samp-highl">GCF_000978375.1</span>.</li>
<li>Код доступу паслядоўнасці RefSeq, з якой паходзіць паслядоўнасць гена: <span class="samp-highl">NZ_CP009686.1</span>.</li>
<li>Каардынаты гена у гэтай геномнай паслядоўнасці: <span class="samp-highl">8908-10459</span> (нумерацыя ідзе з 1-кі, інтэрвалы уключаюць левыя і правыя межы).</li>
<li>Ланцуг геномнай паслядоўнасці, на якой ген знаходзіцца: <span class="samp-highl">plus</span> (альбо <span class="samp-highl">minus</span>).</li>
</ol>
</li>
<li>Радок таксаноміі, а менавіта назвы дамену (<span class="samp-highl">Bacteria</span>), тыпу (<span class="samp-highl">Firmicutes</span>), класу (<span class="samp-highl">Bacilli</span>), парадку (<span class="samp-highl">Bacillales</span>), сямейства (<span class="samp-highl">Bacillaceae</span>), роду (<span class="samp-highl">Bacillus</span>), а таксама відавы эпітэт (<span class="samp-highl">cereus</span>).<br>
Перад кожнай назвай пішацца прэфікс, які пазначае ранг: <span class="samp-highl">d__</span> для дамену, <span class="samp-highl">p__</span> для тыпу, <span class="samp-highl">c__</span> для класу, <span class="samp-highl">o__</span> для парадку, <span class="samp-highl">f__</span> для сямейства, <span class="samp-highl">g__</span> для роду і <span class="samp-highl">s__</span> для відавага эпітэту. У прэфіксах пасля літары пішуцца <strong>два</strong> знакі ніжняга падкрэслівання.<br>
Таксанамічныя назвы адасобленыя і аточаныя кропкамі з коскай (<span class="samp-highl">;</span>).</li>
<li>Катэгорыя геному, з якога паслядоўнасць гена паходзіць: (<span class="samp-highl">category:1</span>).</li>
</ol>
<h3>Адбор паслядоўнасцяў</h3>
<p>Адбіраць тыя ці іншыя паслядоўнасці з файлаў fasta можна з дапамогай праграмы <strong>Seqkit</strong> (<a href="https://github.com/shenwei356/seqkit">ГітХаб-рэпазіторый</a>, <a href="https://bioinf.shenwei.me/seqkit/">дакументацыя</a>). Гэта бясплатная, кросплатформавая, шматфункцыянальная і даволі шустрая праграма, якая можа апрацоўваць як „гзіпʼнутыя“, так і простыя нясціснутыя файлы fasta. Для адбору паслядоўнасцяў зручна выкарыстоўваць падпраграмы <span class="samp">seqkit grep</span> і <span class="samp">seqkit seq</span> .</p>
<div id="subset-by-header" class="pad-anchor"></div>
<h4>Адбор паслядоўнасцяў паводле загалоўкаў</h4>
<p>Няхай мы робім выбарку з файла <span class="samp">ribogrove_13.219_sequences.fasta.gz</span>. Карыснымі могуць быць наступныя прыклады каманд карыстання праграмай <span class="samp">seqkit grep</span>:</p>
<p><strong>Прыклад 1</strong>. Выбраць адну паслядоўнасць паводле SeqID.</p>
<p class="samp-highl samp-vwide">seqkit grep -p "GCF_000978375.1:NZ_CP009686.1:8908-10459:plus" ribogrove_13.219_sequences.fasta.gz</p>
<p class="samp-comment">Опцыя <span class="samp">-p</span> задае радок, які праграма будзе шукаць у fasta-загалоўках (на самой справе, толькі ў той частцы загалоўкаў, дзе запісаны SeqID: да першага прабела).</p>
<p><strong>Прыклад 2</strong>. Выбраць усе паслядоўнасці генаў адной геномнай паслядоўнасці RefSeq паводле кода доступу <span class="samp">NZ_CP009686.1</span>.</p>
<p class="samp-highl samp-vwide">seqkit grep -nrp ":NZ_CP009686.1:" ribogrove_13.219_sequences.fasta.gz</p>
<p class="samp-comment">Тут спатрэбяцца яшчэ дзве опцыі: <span class="samp">-n</span> і <span class="samp">-r</span>. Першая кажа праграме шукаць падрадкі&#769; у загалоўках цалкам, а не толькі ў ідэнтыфікатарах (SeqID). Другая опцыя кажа праграме, што ёй трэба шукаць не толькі тыя загалоўкі, якія цалкам супадаюць з шуканым радком, а таксама і тыя, якія змяшчаюць шуканы радок як сваю частку.</p>
<p class="samp-comment">Каб забяспечыць спецыфічнасць пошуку, атачайце код доступу двукропʼямі (<span class="samp-highl">:</span>).</p>
<p><strong>Прыклад 3</strong>. Выбраць усе паслядоўнасці генаў аднаго геному (код доступу геномнай зборкі <a href="https://ncbi.nlm.nih.gov/assembly/GCF_019357495.1">GCF_019357495.1</a>).</p>
<p class="samp-highl samp-vwide">seqkit grep -nrp "GCF_019357495.1:" ribogrove_13.219_sequences.fasta.gz</p>
<p class="samp-comment">Каб забяспечыць спецыфічнасць пошуку, пішыце двукропʼе (<span class="samp-highl">:</span>) пасля коду доступу зборкі.</p>
<p><strong>Прыклад 4</strong>. Выбраць усе паслядоўнасці актынабактэрый.</p>
<p class="samp-highl samp-vwide">seqkit grep -nrp ";p__Actinobacteria;" ribogrove_13.219_sequences.fasta.gz</p>
<p class="samp-comment">Каб забяспечыць спецыфічнасць пошуку, атачайце таксаномію кропкамі з коскай (<span class="samp-highl">;</span>).</p>
<p><strong>Прыклад 5</strong>. Выбраць усе паслядоўнасці з геномаў катэгорыі 1.</p>
<p class="samp-highl samp-vwide">seqkit grep -nrp "category:1" ribogrove_13.219_sequences.fasta.gz</p>
<p><strong>Прыклад 6</strong>. Выбраць усе паслядоўнасці апроч тых, якія належаць да тыпу <i>Firmicutes</i>.</p>
<p class="samp-highl samp-vwide">seqkit grep -nvrp ";p__Firmicutes;" ribogrove_13.219_sequences.fasta.gz</p>
<p class="samp-comment">Звярніце ўвагу на опцыю <span class="samp">-v</span> у шэгару опцый <span class="samp">-nvrp</span>. Гэтая опцыя інвертуе паводзіны праграмы, і цяпер яна знойдзе ўсе паслядоўнасці, у загалоўках якіх <em>няма</em> радку «<span class="samp">;p__Firmicutes;</span>».</p>
<div id="subset-by-len" class="pad-anchor"></div>
<h4>Адбор паслядоўнасцяў паводле даўжыні</h4>
<p>Адбіраць паслядоўнасці паводле даўжыні можна з дапамогай праграмы <span class="samp">seqkit seq</span>.</p>
<p><strong>Прыклад 1</strong>. Выбраць усе паслядоўнасці даўжэй за 1600 п.а.</p>
<p class="samp-highl samp-vwide">seqkit seq -m 1601 ribogrove_13.219_sequences.fasta.gz</p>
<p class="samp-comment">Опцыя <span class="samp">-m</span> задае мінімальную даўжыню паслядоўнасцяў, якія праграма падасць на выхад.</p>
<p><strong>Прыклад 2</strong>. Выбраць усе паслядоўнасці карацей за 1500 п.а.</p>
<p class="samp-highl samp-vwide">seqkit seq -M 1499 ribogrove_13.219_sequences.fasta.gz</p>
<p class="samp-comment">Опцыя <span class="samp">-M</span> задае максімальную даўжыню паслядоўнасці, якія праграма падасць на выхад.</p>
<p><strong>Прыклад 3</strong>. Выбраць усе паслядоўнасці, даўжыня якіх знаходзіцца ў прамежку [1500, 1600] п.а.</p>
<p class="samp-highl samp-vwide">seqkit seq -m 1500 -M 1600 ribogrove_13.219_sequences.fasta.gz</p>
<div id="select-head" class="pad-anchor"></div>
<h4>Адбор дадзеных загалоўкаў</h4>
<p>Часам бывае карысна атрымаць толькі дадзеныя загалоўкаў з файла fasta. Гэта можна зрабіць з дапамогай праграмы <span class="samp">seqkit seq</span>.</p>
<p><strong>Прыклад 1</strong>. Выбраць проста ўсе загалоўкі.</p>
<p class="samp-highl samp-vwide">seqkit seq -n ribogrove_13.219_sequences.fasta.gz</p>
<p class="samp-comment">Опцыя <span class="samp">-n</span> кажа праграме, каб тая падавала на выхад толькі загалоўкі.</p>
<p><strong>Прыклад 2</strong>. Выбраць усе ідэнтыфікатары паслядоўнасцяў (то бок часткі загалоўкаў да першага прабела).</p>
<p class="samp-highl samp-vwide">seqkit seq -ni ribogrove_13.219_sequences.fasta.gz</p>
<p class="samp-comment">Опцыя <span class="samp">-i</span> кажа праграме, каб тая падавала на выхад толькі ідэнтыфікатары.</p>
<p><strong>Прыклад 3</strong>. Выбраць усе коды доступу геномных паслядоўнасцяў з БД RefSeq.</p>
<p class="samp-highl samp-vwide">seqkit seq -ni ribogrove_13.219_sequences.fasta.gz | cut -f2 -d':' | sort | uniq</p>
<p class="samp-comment">Такая команда спрацуе, толькі калі на кампʼютары ўсталяваныя ўтыліты <span class="samp">cut</span>, <span class="samp">sort</span>, і <span class="samp">uniq</span> (на Linux і Mac OS гэтыя праграмы звычайна ўсталяваныя разам з аперацыйная сістэмай).</p>
<p><strong>Прыклад 4</strong>. Выбраць усе коды доступу геномных зборак.</p>
<p class="samp-highl samp-vwide">seqkit seq -ni ribogrove_13.219_sequences.fasta.gz | cut -f1 -d':' | sort | uniq</p>
<p class="samp-comment">Такая команда спрацуе, толькі калі на кампʼютары ўсталяваныя ўтыліты <span class="samp">cut</span>, <span class="samp">sort</span> і <span class="samp">uniq</span> (на Linux і Mac OS гэтыя праграмы звычайна ўсталяваныя разам з аперацыйная сістэмай).</p>
<p><strong>Прыклад 5</strong>. Выбраць усе назвы тыпаў арганізмаў.</p>
<p class="samp-highl samp-vwide">seqkit seq -n ribogrove_13.219_sequences.fasta.gz | grep -Eo ';p__[^;]+' | sed -E 's/;|p__//g' | sort | uniq</p>
<p class="samp-comment">Такая команда спрацуе, толькі калі на кампʼютары ўсталяваныя ўтыліты <span class="samp">grep</span>, <span class="samp">sed</span>, <span class="samp">sort</span> і <span class="samp">uniq</span> (на Linux і Mac OS гэтыя праграмы звычайна ўсталяваныя разам з аперацыйная сістэмай).</p>
<hr>
<div id="contacts" class="pad-anchor"></div>
<h2>Кантакты</h2>
<p>Калі вы маеце пытанне пра RiboGrove, калі ласка, звяртайцеся да Максіма Сікаленкі на эл. адрас sikolenko<img class="sabaka" src="img/pes2.gif" alt="[ at ]" align="bottom">bio.bsu.by або maximdeynonih<img class="sabaka" src="img/pes2.gif" alt="[ at ]" align="bottom">gmail.com.</p>
<hr>
<div id="citing-ribogrove" class="pad-anchor"></div>
<h2>Цытаванне RiboGrove</h2>
<p>Калі база дадзеных RiboGrove прыдалася вам у вашых даследаваннях, спасылайцеся, калі ласка, на наступную публікацыю:</p>
<p>Maxim A. Sikolenko, Leonid N. Valentovich. “RiboGrove: a database of full-length prokaryotic 16S rRNA genes derived from completely assembled genomes” // Research in Microbiology, Volume 173, Issue 4, May 2022, 103936.<br> (<span style="cursor:help"title="Лічбавы ідэнтыфікатар абʼекта">ЛІА</span>: <a href="https://doi.org/10.1016/j.resmic.2022.103936">10.1016/j.resmic.2022.103936</a>).</p>
<hr>
<div id="faq" class="pad-anchor"></div>
<h2>Пытанні, якія ў людзей узнікаюць пра RiboGrove</h2>
<div><details>
  <summary><b>1. Як зрабіць файл з таксаноміяй, з якім маглі бы працаваць праграмы з пакета QIIME2?</b></summary>
  <p>Для гэтага можаце скарыстацца скрыптом <a href="https://github.com/masikol/ribogrove-tools/blob/main/data_transformation_scripts/make_qiime_taxonomy_file.py"><span class="samp">make_qiime_taxonomy_file.py</span></a>: ён трансфармуе файл <span class="samp">metadata/taxonomy.tsv</span> у файл, з якім змогуць працаваць праграмы QIIME2. Даведку-кіраўніцтва для гэтага скрыпта можаце знайсці ў адпаведным <a href="https://github.com/masikol/ribogrove-tools/blob/main/data_transformation_scripts/README.md">файле README</a>.</p>
</details></div><br>
<div><details>
  <summary><b>2. Як захаваць у файл дадзеныя, адабраныя з дапамогай Seqkit?</b></summary>
  <p>У адпаведнай дыскусіі на форуме ўжо далі некалькі карысных адказаў і парад: <a href="https://bioinformatics.stackexchange.com/questions/20915/how-do-i-save-selected-sequences-in-seqkit-to-a-file">https://bioinformatics.stackexchange.com/questions/20915/how-do-i-save-selected-sequences-in-seqkit-to-a-file</a>.</p>
</details></div><br>
<div><details>
  <summary><b>3. Як знайсці патрэбныя запісы у fasta-файле паводле паслядоўнасці з дапамогай Seqkit?</b></summary>
  <p>У адпаведнай дыскусіі на форуме ўжо далі некалькі карысных адказаў і парад: <a href="https://www.biostars.org/p/9561418">https://www.biostars.org/p/9561418</a>.</p>
</details></div>
<br>
<hr>
<p>RiboGrove, 20.07.2023</p>
</div>
</body>
</html>