<!DOCTYPE html>
<html lang="en">
<head>
<title>RiboGrove mirror</title>
<link rel="stylesheet" href="css/style.css">
<link rel="stylesheet" href="css/archive-style.css">
<link rel="icon" type="image/x-icon" href="favicon.ico">
<link rel="shortcut icon" type="image/x-icon" href="favicon.ico">
</head>
<body>
<div class="container">
<br>
<p><div class="warn-section">The <a href="https://mbio.bas-net.by/cager/en/ribogrove">main website</a> where RiboGrove is hosted may be unavailable outside Belarus due to technical troubles and the overall disaster.<br>Hence this mirror has been created, and RiboGrove files are available through Dropbox.<br><br>
Unfortunately, RiboGrove releases shall be published with some delay after release 13.219 until August 2024,<br>because the main curator of the database has been conscripted into the army.</div></p><br>
<img src="img/RiboGrove_logo.png"><br><br>
<p><a href="index.html">Home</a> / Release notes</p>
<h1 class="archive-title">RiboGrove release notes</h1>
<h2 style="margin-top: 1.0rem;">Release 17.223 (2024-03-21). Latest release.</h2>
<p><i style="font-size: 14px;">No significant differences from the previous release.</i><br /> <a href="index.html">Go to release 17.223.</a></p>
<h2 style="margin-top: .5rem;">Release 16.222 (2024-03-20).</h2>
<p><i style="font-size: 14px;">No significant differences from the previous release.</i><br /> <a href="archive/ribogrove_16.222_en.html">Go to release 16.222.</a></p>
<h2 style="margin-top: .5rem;">Release 15.221 (2024-03-20).</h2>
<p><i style="font-size: 14px;">No significant differences from the previous release.</i><br /> <a href="archive/ribogrove_15.221_en.html">Go to release 15.221.</a></p>
<h2 style="margin-top: .5rem;">Release 14.220 (2024-03-19).</h2>
<p><i style="font-size: 14px;">No significant differences from the previous release.</i><br /> <a href="archive/ribogrove_14.220_en.html">Go to release 14.220.</a></p>
<h2 style="margin-top: .5rem;">Release 13.219 (2023-07-19).</h2>
<p>Starting with RiboGrove 13.219, metadata contains file <span class="samp-highl">primer_pair_genomic_coverage.tsv</span>. In this file you can find coverage values for different V-region-targeting PCR primer pairs for each bacterial phylum, class, order, family, genus, and species.</p>
<p><a href="archive/ribogrove_13.219_en.html">Go to release 13.219.</a></p>
<h2 style="margin-top: .5rem;">Release 12.218 (2023-05-15).</h2>
<ul>
<li>Starting with the release 12.218 we use the “bscore” metric instead of “tscore” for pivotal gene detection (the ribotyper program calculates the mtrics).</li>
<li>Deletion length threshold was changed from 10 to 15 b.p. Now, deletions larger than 15 b.p. are considered long enough to discard a sequence.</li>
</ul>
<p><a href="archive/ribogrove_12.218_en.html">Go to release 12.218.</a></p>
<h2 style="margin-top: .5rem;">Release 11.217 (2023-03-21).</h2>
<p>Starting with release 11.217:
  <ul>
  <li>Header format has been changed again, and this is the last time it changes. Now, Assembly accession numbers are used instead of Assembly IDs with a <span class="samp-highl">G_</span> prefix (e.g. <span class="samp-highl">GCF_004799645.1:</span> instead of <span class="samp-highl">G_2580821:</span>).</li>
  <li>Redundant data has been removed from metadata files. Now, for example, instead of a taxonomy string for each gene, the <span class="samp-highl">taxonomy.tsv</span> file contains a taxonomy string for each genome, thus reducing the file size.</li>
  <li>File <span class="samp-highl">QIIME2-compatible-taxonomy.txt</span> has been removed from metadata archive. You can easily make it with the script <a href="https://github.com/masikol/ribogrove-tools/tree/main/data_transformation_scripts">make_qiime_taxonomy_file.py</a> from ribogrove-tools v11.217 and later.</li>
  <li>The process of sequence filtering has been modified:
    <ul>
      <li>we removed the step which searches the gene sequences for 16S rRNA conserved regions according to <a href="https://doi.org/10.7717%2Fpeerj.3036">Martinez-Porchas M et al.</a></li>
      <li>now, the pipeline uses the ribotyper program from the <a href="https://github.com/ncbi/ribovore">Ribovore</a> suite to detect pivotal genes, instead of cmscan.</li>
    </ul>
  </li>
  </ul>
</p>
<p><a href="archive/ribogrove_11.217_en.html">Go to release 11.217.</a></p>
<h2 style="margin-top: .5rem;">Release 10.216 (2023-01-18).</h2>
<p>Starting with RiboGrove 10.216 we use Rfam 14.9 for sequence filtering. In Rfam 14.9, the model RF00177 (Bacterial small subunit ribosomal RNA) is updated. Before RiboGrove 10.216 we used Rfam 14.6.</p>
<p><a href="archive/ribogrove_10.216_en.html">Go to release 10.216.</a></p>
<h2 style="margin-top: .5rem;">Release 9.215 (2022-11-30).</h2>
<p>Starting with release 9.215, we publish section “Coverage of primer pairs for different V-regions of bacterial 16S rRNA genes”.</p>
<p><a href="archive/ribogrove_9.215_en.html">Go to release 9.215.</a></p>
<h2 style="margin-top: .5rem;">Release 8.214 (2022-10-13).</h2>
<p>Starting with release 8.214:</p>
<ol>
<li>RiboGrove uses a new format of fasta headers to improve search specificity. Example:<br /> Old header:<br /><span style="font-family: monospace; font-size: 14px;">&gt;NZ_CP009686.1:8908-10459_plus Bacillus_cereus ;Bacteria;Firmicutes;Bacilli;Bacillales;Bacillaceae;Bacillus; category:1</span><br /> New header:<br /><span style="font-family: monospace; font-size: 14px;">&gt;G_324861:NZ_CP009686.1:8908-10459:plus ;d__Bacteria;p__Firmicutes;c__Bacilli;o__Bacillales;f__Bacillaceae;g__Bacillus;s__cereus; category:1</span><br /> Please, see the <a href="archive/ribogrove_8.214_en.html#header-format">8.214 release page</a> for details.</li>
<li>We don’t publish fasta file of “raw” gene sequences any more because its content is mostly the same as data stored in the fasta file of full-length sequences.</li>
<li>To replace the “raw” fasta file, metadata now contains file <span style="font-family: monospace;">discarded_sequences.fasta.gz</span>. This file contains 16S rRNA gene sequences which were present in source RefSeq genomes and were annotated as 16S rRNA genes but which have been discarded according to their incompleteness, internal repeats etc. and thus haven’t been included in RiboGrove.</li>
<li>Metadata now does not contain files <span style="font-family: monospace;">per_replicon_statistics.tsv</span> and <span style="font-family: monospace;">raw_per_replicon_statistics.tsv</span> because of their unnecessary redundancy. All data they contained can be easily obtained from files <span style="font-family: monospace;">gene_seqs_statistics.tsv</span> and <span style="font-family: monospace;">discarded_gene_seqs_statistics.tsv</span>.</li>
</ol>
<p><a href="archive/ribogrove_8.214_en.html">Go to release 8.214.</a></p>
<h2 style="margin-top: .5rem;">Release 7.213 (2022-07-20).</h2>
<p>Starting with release 7.213, metadata contains the file <span style="font-family: monospace;">QIIME2-compatible-taxonomy.txt</span>. This is a TSV file, which can be used to train a QIIME2 classifier (see the <a href="https://docs.qiime2.org">tutorial</a>).</p>
<p><a href="archive/ribogrove_7.213_en.html">Go to release 7.213.</a></p>
<h2 style="margin-top: .5rem;">Release 6.212 (2022-05-13).</h2>
<p><i style="font-size: 14px;">No significant differences from the previous release.</i><br /> <a href="archive/ribogrove_6.212_en.html">Go to release 6.212.</a></p>
<h2 style="margin-top: .5rem;">Release 5.211 (2022-03-14).</h2>
<p><i style="font-size: 14px;">No significant differences from the previous release.</i><br /> <a href="archive/ribogrove_5.211_en.html">Go to release 5.211.</a></p>
<h2 style="margin-top: .5rem;">Release 4.210 (2022-01-12).</h2>
<p><i style="font-size: 14px;">No significant differences from the previous release.</i><br /> <a href="archive/ribogrove_4.210_en.html">Go to release 4.210.</a></p>
<h2 style="margin-top: .5rem;">Release 3.209 (2021-11-25).</h2>
<p>The release date is correct: releases 2.208 and 3.209 were released on the same day.<br /> <a href="archive/ribogrove_3.209_en.html">Go to release 3.209.</a></p>
<h2 style="margin-top: .5rem;">Release 2.208 (2021-11-25)</h2>
<p>The release date is correct: releases 2.208 and 3.209 were released on the same day.<br /> <a href="archive/ribogrove_2.208_en.html">Go to release 2.208.</a></p>
<h2 style="margin-top: .5rem;">Release 1.207 (2021-12-08)</h2>
<p>You may wonder, why RiboGrove 1.207 was released on December 8<sup>th</sup>, 2021, and RiboGrove 2.208 on November 25<sup>th</sup>, 2021. The reason is that initially (on October 28<sup>th</sup>, 2021), RiboGrove 1.207 wasn’t, strictly speaking, based on RefSeq 207 — it contained sequences, which had been added to RefSeq after release 207 but before release 208. The corrected version of RiboGrove 1.207 (2021-12-08) is based on RefSeq 207, without “contaminations” from RefSeq 208.</p>
<p><a href="archive/ribogrove_1.207_en.html">Go to release 1.207.</a></p>

<hr>
<p>RiboGrove, 2024-03-20</p>
</div>
</body>
</html>
