<!DOCTYPE html>
<html lang="en">
<head>
<title>RiboGrove mirror</title>
<link rel="stylesheet" href="../css/style.css">
<link rel="stylesheet" href="../css/release-page.css">
</head>
<body>
<div class="container">
<br />
<p><div class="warn-section">The <a href="http://mbio.bas-net.by/cager/en/ribogrove">main website</a> where RiboGrove is hosted may be unavailable outside Belarus due to technical troubles and the overall disaster.<br />Hence this mirror has been created, and RiboGrove files are available through Dropbox (links are <a href="#downloads">below</a>).</div></p><br />
<p><img src="../img/RiboGrove_logo.png" /><br /><br /></p>
<p><a href="../index.html">Home</a> / <a href="ribogrove_archive_en.html">Release archive</a> / 2.208</p>
<h2>Contents</h2>
<ul>
<li><a href="#downloads">Downloads</a>
<ul>
<li><a href="#current-release">RiboGrove 2.208</li>
<li><a href="#release-archive">RiboGrove release archive</li>
</ul>
</li>
<li><a href="#summary">Statistical summary</a>
<ul>
<li><a href="#db-size">RiboGrove size</a></li>
<li><a href="#gene-lens">16S rRNA gene lengths</a></li>
<li><a href="#copy-number">16S rRNA gene copy number</a></li>
<li><a href="#top-longest">Top-10 longest 16S rRNA genes</a></li>
<li><a href="#top-shortest">Top-10 shortest 16S rRNA genes</a></li>
<li><a href="#top-copy-num">Top-10 genomes with largest 16S rRNA copy numbers</a></li>
<li><a href="#top-var">Top-10 genomes with highest intragenomic variability of 16S rRNA genes</a></li>
</ul>
</li>
</ul>
<hr />
<div id="downloads" class="pad-anchor"></div>
<h2>Downloads</h2>
<div id="current-release" class="pad-anchor"></div>
<h3>RiboGrove release 2.208 (2021-11-25)</h3>
<p>The release is based on RefSeq release 208.</p>
<ul>
<li>A fasta file of full-length 16S gene sequences. <a href="https://www.dropbox.com/s/3lra4bxsji00ykd/ribogrove_2.208_sequences.fasta.gz?dl=0">Download (gzipped fasta file, 4.95 MB)</a></li>
<li>A <q>raw</q> version of the fasta file above. This file contains partial sequences. <a href="https://www.dropbox.com/s/tu68q350jf0kzo8/raw_ribogrove_2.208_sequences.fasta.gz?dl=0">Download (gzipped fasta file, 5.18 MB)</a></li>
<li>Metadata <a href="https://www.dropbox.com/s/gfhffs71kwjeven/metadata_ribogrove_2.208.zip?dl=0">Download (zip archive 13.98 MB)</a><br>
  <details>
    <summary><small>What information exactly does the metadata contain?</small></summary>
    <small>
      <p>The metadata consists of the following files:</p>
      <ol>
      <li>
        <span class="samp-highl">source_RefSeq_genomes.tsv</span><br>
        This is a TSV files, which contains information about what genomes were used for the RiboGrove construction.
      </li>
      <li>
        <span class="samp-highl">gene_seqs_statistics.tsv</span>, <span class="samp-highl">raw_gene_seqs_statistics.tsv</span><br>
        These are TSV file, which contains nucleotide conposition, size, genomic and taxonomic affiliation of the gene sequences. The &ldquo;raw&rdquo; file also includes information about partial genes.
      </li>
      <li>
        <span class="samp-highl">per_replicon_statistics.tsv</span>, <span class="samp-highl">raw_per_replicon_statistics.tsv</span><br>
        These are TSV files, which contain information about number of 16S rRNA genes in each RefSeq genomic sequences, and about sizes of these genes. The &ldquo;raw&rdquo; file also includes information about partial genes.
      </li>
      <li>
        <span class="samp-highl">categories.tsv</span><br>
        This is a TSV file, which contains information about what genome categories were assigned to each genome and why. Moreover, it contains information about what sequencing technology was used to sequence each genome.
      </li>
      <li>
        <span class="samp-highl">taxonomy.tsv</span><br>
        This is a TSV file, which contains taxonomic affiliation of each genome and gene.
      </li>
      <li>
        <span class="samp-highl">intragenic_repeats.tsv</span><br>
        This is a TSV file, which contains information about intragenomic repeats found in gene sequences using <a href="https://github.com/deprekate/RepeatFinder">RepeatFinder</a>.
      </li>
      <li>
        <span class="samp-highl">cmscan_output_table.tblout</span><br>
        This is a TSV file, which contains complete output of <a href="http://eddylab.org/infernal/">cmscan</a> program outputted after processing all 16S rRNA sequences.
      </li>
      <li>
        <span class="samp-highl">entropy_summary.tsv</span><br>
        This is a TSV file, which contains summary of instragenomic variability of the 16S rRNA genes. Intragenomic variability are calculated only for the category&nbsp;1 genomes having more than one 16S rRNA gene. Intragenomic variability is evaluated using Shannon entropy. We align gene sequences from each genome using <a href="https://drive5.com/muscle5/">MUSCLE</a>, and then we calculate Shannon entropy for each multiple alignment column (i.e. base).
      </li>
    </ol></small>
  </details>
</li>
</ul>
<p>The fasta files are compressed with gzip, and the metadata file is a zip archive. To uncompress them, Linux and Mac OS users may use gzip and zip programs, they should be built-in. For Windows users, the free and open source (de)compression program <a href="https://www.7-zip.org/">7-Zip</a> is available.</p>
<div id="release-archive" class="pad-anchor"></div>
<h3>RiboGrove release archive</h3>
<p>You can find all releases at the <a href="ribogrove_archive_en.html">RiboGrove release archive</a>.</p>
<hr />
<div id="summary" class="pad-anchor"></div>
<h2>Statistical summary</h2>
<div id="db-size" class="pad-anchor"></div>
<table class="sum-table">
<caption>RiboGrove size</caption>
<tbody>
<tr><th></th><th class="numcol">Bacteria</th><th class="numcol">Archaea</th><th class="numcol">Total</th></tr>
<tr class="sumtab-row">
<td>Number of gene sequences</td>
<td class="numcol">119,848</td>
<td class="numcol">649</td>
<td class="numcol">120,497</td>
</tr>
<tr class="sumtab-row">
<td>Number of unique gene sequences</td>
<td class="numcol">34,419</td>
<td class="numcol">459</td>
<td class="numcol">34,878</td>
</tr>
<tr class="sumtab-row">
<td>Number of species</td>
<td class="numcol">6,389</td>
<td class="numcol">302</td>
<td class="numcol">6,691</td>
</tr>
<tr class="sumtab-row">
<td>Number of genomes</td>
<td class="numcol">23,029</td>
<td class="numcol">387</td>
<td class="numcol">23,416</td>
</tr>
<tr class="sumtab-row">
<td>Number of genomes of category 1</td>
<td class="numcol">14,967</td>
<td class="numcol">99</td>
<td class="numcol">15,066</td>
</tr>
<tr class="sumtab-row">
<td>Number of genomes of category 2</td>
<td class="numcol">7,938</td>
<td class="numcol">288</td>
<td class="numcol">8,226</td>
</tr>
<tr class="sumtab-row">
<td>Number of genomes of category 3</td>
<td class="numcol">124</td>
<td class="numcol">0</td>
<td class="numcol">124</td>
</tr>
</tbody>
</table>
<div id="gene-lens" class="pad-anchor"></div>
<table class="sum-table">
<caption>16S rRNA gene lengths</caption>
<tbody>
<tr><th></th><th class="numcol">Bacteria</th><th class="numcol">Archaea</th></tr>
<tr class="sumtab-row">
<td>Minimum (bp)</td>
<td class="numcol">1,448.00</td>
<td class="numcol">1,439.00</td>
</tr>
<tr class="sumtab-row">
<td>25th percentile (bp) <sup>*</sup></td>
<td class="numcol">1,517.25</td>
<td class="numcol">1,472.00</td>
</tr>
<tr class="sumtab-row">
<td>Median (bp) <sup>*</sup></td>
<td class="numcol">1,532.00</td>
<td class="numcol">1,475.00</td>
</tr>
<tr class="sumtab-row">
<td>75th percentile (bp) <sup>*</sup></td>
<td class="numcol">1,543.00</td>
<td class="numcol">1,492.00</td>
</tr>
<tr class="sumtab-row">
<td>Average (bp) <sup>*</sup></td>
<td class="numcol">1,528.44</td>
<td class="numcol">1,502.32</td>
</tr>
<tr class="sumtab-row">
<td>Mode (bp) <sup>*</sup></td>
<td class="numcol">1,537.00</td>
<td class="numcol">1,471.00<br />1,472.00</td>
</tr>
<tr class="sumtab-row">
<td>Maximum (bp)</td>
<td class="numcol">1,806.00</td>
<td class="numcol">3,604.00</td>
</tr>
<tr class="sumtab-row">
<td>Standard deviation (bp) <sup>*</sup></td>
<td class="numcol">25.90</td>
<td class="numcol">154.13</td>
</tr>
</tbody>
</table>
<p><sup>*</sup> Metrics marked with this sign were calculated with preliminary normalization, i.e. median within-species gene length were used for the summary.</p>
<div id="copy-number" class="pad-anchor"></div>
<table class="sum-table">
<caption>16S rRNA gene copy number<br><small>(Bacteria and Archaea)</small></caption>
<tbody>
<tr><th class="numcol">Copy<br>number<sup>&nbsp;*</sup></th><th class="numcol">Number of species</th><th class="numcol">Per cent of species (%)</th></tr>
<tr class="sumtab-row">
<td class="numcol">1</td>
<td class="numcol">935</td>
<td class="numcol">13.97</td>
</tr>
<tr class="sumtab-row">
<td class="numcol">2</td>
<td class="numcol">1,298</td>
<td class="numcol">19.40</td>
</tr>
<tr class="sumtab-row">
<td class="numcol">3</td>
<td class="numcol">1,033</td>
<td class="numcol">15.44</td>
</tr>
<tr class="sumtab-row">
<td class="numcol">4</td>
<td class="numcol">859</td>
<td class="numcol">12.84</td>
</tr>
<tr class="sumtab-row">
<td class="numcol">5</td>
<td class="numcol">515</td>
<td class="numcol">7.70</td>
</tr>
<tr class="sumtab-row">
<td class="numcol">6</td>
<td class="numcol">659</td>
<td class="numcol">9.85</td>
</tr>
<tr class="sumtab-row">
<td class="numcol">7</td>
<td class="numcol">528</td>
<td class="numcol">7.89</td>
</tr>
<tr class="sumtab-row">
<td class="numcol">8</td>
<td class="numcol">354</td>
<td class="numcol">5.29</td>
</tr>
<tr class="sumtab-row">
<td class="numcol">9</td>
<td class="numcol">164</td>
<td class="numcol">2.45</td>
</tr>
<tr class="sumtab-row">
<td class="numcol">10</td>
<td class="numcol">145</td>
<td class="numcol">2.17</td>
</tr>
<tr class="sumtab-row">
<td class="numcol">11</td>
<td class="numcol">64</td>
<td class="numcol">0.96</td>
</tr>
<tr class="sumtab-row">
<td class="numcol">12</td>
<td class="numcol">44</td>
<td class="numcol">0.66</td>
</tr>
<tr class="sumtab-row">
<td class="numcol">13</td>
<td class="numcol">31</td>
<td class="numcol">0.46</td>
</tr>
<tr class="sumtab-row">
<td class="numcol">14</td>
<td class="numcol">37</td>
<td class="numcol">0.55</td>
</tr>
<tr class="sumtab-row">
<td class="numcol">15</td>
<td class="numcol">7</td>
<td class="numcol">0.10</td>
</tr>
<tr class="sumtab-row">
<td class="numcol">16</td>
<td class="numcol">5</td>
<td class="numcol">0.07</td>
</tr>
<tr class="sumtab-row">
<td class="numcol">17</td>
<td class="numcol">4</td>
<td class="numcol">0.06</td>
</tr>
<tr class="sumtab-row">
<td class="numcol">18</td>
<td class="numcol">3</td>
<td class="numcol">0.04</td>
</tr>
<tr class="sumtab-row">
<td class="numcol">19</td>
<td class="numcol">1</td>
<td class="numcol">0.01</td>
</tr>
<tr class="sumtab-row">
<td class="numcol">20</td>
<td class="numcol">3</td>
<td class="numcol">0.04</td>
</tr>
<tr class="sumtab-row">
<td class="numcol">27</td>
<td class="numcol">1</td>
<td class="numcol">0.01</td>
</tr>
<tr class="sumtab-row">
<td class="numcol">37</td>
<td class="numcol">1</td>
<td class="numcol">0.01</td>
</tr>
</tbody>
</table>
<p><sup>*</sup> These are median within-species copy numbers.</p>
<div id="top-longest" class="pad-anchor"></div>
<table class="sum-table">
<caption>Top-10 longest 16S rRNA genes</caption>
<tbody>
<tr><th class="alnleft">Assembly ID</th><th class="numcol">Gene length<br>(bp)</th><th class="alnleft">RiboGrove Sequence ID (SeqID)</th><th class="alnleft">Organism</th></tr>
<tr>
<td colspan="4" class="subhead">Bacteria</td>
</tr>
<tr class="sumtab-row">
<td><a href="https://ncbi.nlm.nih.gov/assembly/8028891">8028891</a></td>
<td class="numcol">1,806</td>
<td class="seqid-td">NZ_AP022826.1:1258017-1259822_minus<br />NZ_AP022826.1:1532588-1534393_minus<br />NZ_AP022826.1:1939914-1941719_minus</td>
<td><i>Nitrosophilus labii</i> strain HRV44</td>
</tr>
<tr class="sumtab-row">
<td><a href="https://ncbi.nlm.nih.gov/assembly/10731991">10731991</a></td>
<td class="numcol">1,788</td>
<td class="seqid-td">NZ_AP024619.1:2016182-2017969_minus</td>
<td><i>Gelria</i> sp. Kuro-4</td>
</tr>
<tr class="sumtab-row">
<td><a href="https://ncbi.nlm.nih.gov/assembly/40148">40148</a></td>
<td class="numcol">1,781</td>
<td class="seqid-td">NC_010321.1:2265744-2267524_minus</td>
<td><i>Thermoanaerobacter pseudethanolicus</i> strain ATCC 33223</td>
</tr>
<tr class="sumtab-row">
<td><a href="https://ncbi.nlm.nih.gov/assembly/282748">282748</a></td>
<td class="numcol">1,781</td>
<td class="seqid-td">NC_014964.1:2252888-2254668_minus</td>
<td><i>Thermoanaerobacter brockii</i> strain Ako-1</td>
</tr>
<tr class="sumtab-row">
<td><a href="https://ncbi.nlm.nih.gov/assembly/1153941">1153941</a></td>
<td class="numcol">1,744</td>
<td class="seqid-td">NZ_CP019682.1:607981-609724_plus<br />NZ_CP019682.1:929565-931308_minus<br />NZ_CP019682.1:1501945-1503688_minus</td>
<td><i>Campylobacter sputorum</i> strain RM3237</td>
</tr>
<tr class="sumtab-row">
<td><a href="https://ncbi.nlm.nih.gov/assembly/4499991">4499991</a></td>
<td class="numcol">1,744</td>
<td class="seqid-td">NZ_CP043427.1:609141-610884_plus<br />NZ_CP043427.1:930699-932442_minus<br />NZ_CP043427.1:1503078-1504821_minus</td>
<td><i>Campylobacter sputorum</i> strain LMG 7795</td>
</tr>
<tr class="sumtab-row">
<td><a href="https://ncbi.nlm.nih.gov/assembly/1153911">1153911</a></td>
<td class="numcol">1,743</td>
<td class="seqid-td">NZ_CP019683.1:606847-608589_plus<br />NZ_CP019683.1:935163-936905_minus<br />NZ_CP019683.1:1558189-1559931_minus</td>
<td><i>Campylobacter sputorum</i> strain CCUG 20703</td>
</tr>
<tr class="sumtab-row">
<td><a href="https://ncbi.nlm.nih.gov/assembly/7294871">7294871</a></td>
<td class="numcol">1,742</td>
<td class="seqid-td">NZ_CP053828.1:357136-358877_plus<br />NZ_CP053828.1:1667816-1669557_minus</td>
<td><i>Campylobacter hyointestinalis</i> strain CHY5</td>
</tr>
<tr class="sumtab-row">
<td><a href="https://ncbi.nlm.nih.gov/assembly/1101781">1101781</a></td>
<td class="numcol">1,742</td>
<td class="seqid-td">NZ_CP018789.1:273370-275111_plus<br />NZ_CP018789.1:1545743-1547484_minus</td>
<td><i>Campylobacter</i> sp. RM6137</td>
</tr>
<tr class="sumtab-row">
<td><a href="https://ncbi.nlm.nih.gov/assembly/1153931">1153931</a></td>
<td class="numcol">1,742</td>
<td class="seqid-td">NZ_CP019685.1:577810-579551_plus<br />NZ_CP019685.1:891862-893603_minus<br />NZ_CP019685.1:1479764-1481505_minus</td>
<td><i>Campylobacter sputorum</i> strain RM8705</td>
</tr>
<tr>
<td colspan="4" class="subhead">Archaea</td>
</tr>
<tr class="sumtab-row">
<td><a href="https://ncbi.nlm.nih.gov/assembly/351728">351728</a></td>
<td class="numcol">3,604</td>
<td class="seqid-td">NC_016645.1:127214-130817_plus</td>
<td><i>Pyrobaculum ferrireducens</i> strain 1860</td>
</tr>
<tr class="sumtab-row">
<td><a href="https://ncbi.nlm.nih.gov/assembly/28808">28808</a></td>
<td class="numcol">2,213</td>
<td class="seqid-td">NC_003364.1:1089640-1091852_plus</td>
<td><i>Pyrobaculum aerophilum</i> strain IM2</td>
</tr>
<tr class="sumtab-row">
<td><a href="https://ncbi.nlm.nih.gov/assembly/37488">37488</a></td>
<td class="numcol">2,212</td>
<td class="seqid-td">NC_009376.1:623323-625534_minus</td>
<td><i>Pyrobaculum arsenaticum</i> strain DSM 13514</td>
</tr>
<tr class="sumtab-row">
<td><a href="https://ncbi.nlm.nih.gov/assembly/32288">32288</a></td>
<td class="numcol">2,202</td>
<td class="seqid-td">NC_000854.2:1218712-1220913_minus</td>
<td><i>Aeropyrum pernix</i> strain K1</td>
</tr>
<tr class="sumtab-row">
<td><a href="https://ncbi.nlm.nih.gov/assembly/40848">40848</a></td>
<td class="numcol">2,197</td>
<td class="seqid-td">NC_010525.1:690419-692615_plus</td>
<td><i>Pyrobaculum neutrophilum</i> strain V24Sta</td>
</tr>
<tr class="sumtab-row">
<td><a href="https://ncbi.nlm.nih.gov/assembly/1145431">1145431</a></td>
<td class="numcol">2,008</td>
<td class="seqid-td">NZ_CP019964.1:751297-753304_minus</td>
<td><i>Candidatus Mancarchaeum acidiphilum</i> strain Mia14</td>
</tr>
<tr class="sumtab-row">
<td><a href="https://ncbi.nlm.nih.gov/assembly/9220081">9220081</a></td>
<td class="numcol">2,003</td>
<td class="seqid-td">NZ_CP060530.1:203642-205644_minus</td>
<td><i>Candidatus Micrarchaeum</i> sp. A_DKE</td>
</tr>
<tr class="sumtab-row">
<td><a href="https://ncbi.nlm.nih.gov/assembly/39388">39388</a></td>
<td class="numcol">1,679</td>
<td class="seqid-td">NC_009954.1:129150-130828_minus</td>
<td><i>Caldivirga maquilingensis</i> strain IC-167</td>
</tr>
<tr class="sumtab-row">
<td><a href="https://ncbi.nlm.nih.gov/assembly/127981">127981</a></td>
<td class="numcol">1,650</td>
<td class="seqid-td">NC_022521.1:1165168-1166817_minus</td>
<td><i>Aeropyrum camini</i> strain SY1</td>
</tr>
<tr class="sumtab-row">
<td><a href="https://ncbi.nlm.nih.gov/assembly/304318">304318</a></td>
<td class="numcol">1,576</td>
<td class="seqid-td">NC_015931.1:84671-86246_minus</td>
<td><i>Pyrolobus fumarii</i> strain 1A</td>
</tr>
</tbody>
</table>
<div id="top-shortest" class="pad-anchor"></div>
<table class="sum-table">
<caption>Top-10 shortest 16S rRNA genes</caption>
<tbody>
<tr><th class="alnleft">Assembly ID</th><th class="numcol">Gene length<br>(bp)</th><th class="alnleft">RiboGrove Sequence ID (SeqID)</th><th class="alnleft">Organism</th></tr>
<tr>
<td colspan="4" class="subhead">Bacteria</td>
</tr>
<tr class="sumtab-row">
<td><a href="https://ncbi.nlm.nih.gov/assembly/44428">44428</a></td>
<td class="numcol">1,448</td>
<td class="seqid-td">NC_012982.1:2336679-2338126_minus</td>
<td><i>Hirschia baltica</i> strain ATCC 49814</td>
</tr>
<tr class="sumtab-row">
<td><a href="https://ncbi.nlm.nih.gov/assembly/1460951">1460951</a></td>
<td class="numcol">1,449</td>
<td class="seqid-td">NZ_CP021913.1:2386837-2388285_plus<br />NZ_CP021913.1:3597920-3599368_plus</td>
<td><i>Sagittula</i> sp. P11</td>
</tr>
<tr class="sumtab-row">
<td><a href="https://ncbi.nlm.nih.gov/assembly/860061">860061</a></td>
<td class="numcol">1,451</td>
<td class="seqid-td">NZ_CP017718.1:2304269-2305719_minus</td>
<td><i>Hyphomonas</i> sp. Mor2</td>
</tr>
<tr class="sumtab-row">
<td><a href="https://ncbi.nlm.nih.gov/assembly/1163161">1163161</a></td>
<td class="numcol">1,453</td>
<td class="seqid-td">NZ_CP022540.1:1369380-1370832_plus<br />NZ_CP022540.1:2482480-2483932_plus</td>
<td><i>Antarctobacter heliothermus</i> strain SMS3</td>
</tr>
<tr class="sumtab-row">
<td><a href="https://ncbi.nlm.nih.gov/assembly/6279751">6279751</a></td>
<td class="numcol">1,454</td>
<td class="seqid-td">NZ_AP022337.1:267139-268592_plus<br />NZ_AP022337.1:1420942-1422395_plus<br />NZ_AP022337.1:3191208-3192661_minus</td>
<td><i>Mameliella alba</i> strain KU6B</td>
</tr>
<tr class="sumtab-row">
<td><a href="https://ncbi.nlm.nih.gov/assembly/9503471">9503471</a></td>
<td class="numcol">1,455</td>
<td class="seqid-td">NZ_CP022271.1:2407999-2409453_minus</td>
<td><i>Hyphomonas</i> sp. KY3</td>
</tr>
<tr class="sumtab-row">
<td><a href="https://ncbi.nlm.nih.gov/assembly/34128">34128</a></td>
<td class="numcol">1,455</td>
<td class="seqid-td">NC_008358.1:2818466-2819920_minus</td>
<td><i>Hyphomonas neptunium</i> strain ATCC 15444</td>
</tr>
<tr class="sumtab-row">
<td><a href="https://ncbi.nlm.nih.gov/assembly/8694041">8694041</a></td>
<td class="numcol">1,458</td>
<td class="seqid-td">NZ_CP060436.1:2482207-2483664_minus</td>
<td><i>Pseudooceanicola algae</i> strain Lw-13e</td>
</tr>
<tr class="sumtab-row">
<td><a href="https://ncbi.nlm.nih.gov/assembly/8738751">8738751</a></td>
<td class="numcol">1,459</td>
<td class="seqid-td">NZ_CP065429.1:477373-478831_plus</td>
<td><i>Sulfitobacter</i> sp. B30-2</td>
</tr>
<tr class="sumtab-row">
<td><a href="https://ncbi.nlm.nih.gov/assembly/8872011">8872011</a></td>
<td class="numcol">1,459</td>
<td class="seqid-td">NZ_CP065915.1:2729819-2731277_minus<br />NZ_CP065915.1:3593071-3594529_minus</td>
<td><i>Pelagovum pacificum</i> strain SM1903</td>
</tr>
<tr class="sumtab-row">
<td><a href="https://ncbi.nlm.nih.gov/assembly/1078381">1078381</a></td>
<td class="numcol">1,459</td>
<td class="seqid-td">NZ_CP020612.1:582021-583479_minus<br />NZ_CP020612.1:1166317-1167775_minus</td>
<td><i>Paracoccus contaminans</i> strain RKI 16-01929</td>
</tr>
<tr class="sumtab-row">
<td><a href="https://ncbi.nlm.nih.gov/assembly/9217271">9217271</a></td>
<td class="numcol">1,459</td>
<td class="seqid-td">NZ_CP069004.1:3093411-3094869_plus</td>
<td><i>Sulfitobacter mediterraneus</i> strain SC1-11</td>
</tr>
<tr>
<td colspan="4" class="subhead">Archaea</td>
</tr>
<tr class="sumtab-row">
<td><a href="https://ncbi.nlm.nih.gov/assembly/39048">39048</a></td>
<td class="numcol">1,439</td>
<td class="seqid-td">NC_009776.1:728362-729800_plus</td>
<td><i>Ignicoccus hospitalis</i> strain KIN4/I</td>
</tr>
<tr class="sumtab-row">
<td><a href="https://ncbi.nlm.nih.gov/assembly/10131521">10131521</a></td>
<td class="numcol">1,457</td>
<td class="seqid-td">NZ_LR792632.1:542755-544211_plus</td>
<td><i>Methanocaldococcus</i> sp. SG7</td>
</tr>
<tr class="sumtab-row">
<td><a href="https://ncbi.nlm.nih.gov/assembly/2220501">2220501</a></td>
<td class="numcol">1,463</td>
<td class="seqid-td">NZ_CP034692.1:397753-399215_minus</td>
<td><i>Halorubrum</i> sp. BOL3-1</td>
</tr>
<tr class="sumtab-row">
<td><a href="https://ncbi.nlm.nih.gov/assembly/4352521">4352521</a></td>
<td class="numcol">1,466</td>
<td class="seqid-td">NZ_LR699000.1:283607-285072_plus</td>
<td><i>Candidatus Methanomethylophilus alvus</i> strain MGYG-HGUT-02456</td>
</tr>
<tr class="sumtab-row">
<td><a href="https://ncbi.nlm.nih.gov/assembly/34548">34548</a></td>
<td class="numcol">1,466</td>
<td class="seqid-td">NC_007796.1:39814-41279_plus<br />NC_007796.1:1301079-1302544_minus<br />NC_007796.1:3501525-3502990_minus<br />NC_007796.1:3507609-3509074_minus</td>
<td><i>Methanospirillum hungatei</i> strain JF-1</td>
</tr>
<tr class="sumtab-row">
<td><a href="https://ncbi.nlm.nih.gov/assembly/10519241">10519241</a></td>
<td class="numcol">1,466</td>
<td class="seqid-td">NZ_CP077107.1:4649-6114_plus<br />NZ_CP077107.1:1359562-1361027_minus<br />NZ_CP077107.1:1365502-1366967_minus<br />NZ_CP077107.1:1986020-1987485_minus</td>
<td><i>Methanospirillum hungatei</i> strain GP1</td>
</tr>
<tr class="sumtab-row">
<td><a href="https://ncbi.nlm.nih.gov/assembly/2068141">2068141</a></td>
<td class="numcol">1,466</td>
<td class="seqid-td">NZ_CP017686.1:283608-285073_plus</td>
<td><i>Candidatus Methanomethylophilus alvus</i> strain Mx-05</td>
</tr>
<tr class="sumtab-row">
<td><a href="https://ncbi.nlm.nih.gov/assembly/7330651">7330651</a></td>
<td class="numcol">1,466</td>
<td class="seqid-td">NZ_CP058334.1:1530622-1532087_minus</td>
<td><i>Natronomonas halophila</i> strain C90</td>
</tr>
<tr class="sumtab-row">
<td><a href="https://ncbi.nlm.nih.gov/assembly/10123301">10123301</a></td>
<td class="numcol">1,466</td>
<td class="seqid-td">NZ_CP075546.1:133354-134819_plus<br />NZ_CP075546.1:825954-827419_plus<br />NZ_CP075546.1:872641-874106_plus<br />NZ_CP075546.1:1727419-1728884_plus</td>
<td><i>Methanospirillum</i> sp. J.3.6.1-F.2.7.3</td>
</tr>
<tr class="sumtab-row">
<td><a href="https://ncbi.nlm.nih.gov/assembly/599268">599268</a></td>
<td class="numcol">1,466</td>
<td class="seqid-td">NC_020913.1:283607-285072_plus</td>
<td><i>Candidatus Methanomethylophilus alvus</i> strain Mx1201</td>
</tr>
</tbody>
</table>
<div id="top-copy-num" class="pad-anchor"></div>
<table class="sum-table">
<caption>Top-10 genomes with largest 16S rRNA copy numbers</caption>
<tbody>
<tr><th class="alnleft">Assembly ID</th><th class="numcol">Copy<br>number</th><th class="alnleft">Organism</th></tr>
<tr>
<td colspan="4" class="subhead">Bacteria</td>
</tr>
<tr class="sumtab-row">
<td><a href="https://ncbi.nlm.nih.gov/assembly/1115491">1115491</a></td>
<td class="numcol">37</td>
<td><i>Tumebacillus avium</i> stain AR23208</td>
</tr>
<tr class="sumtab-row">
<td><a href="https://ncbi.nlm.nih.gov/assembly/1166771">1166771</a></td>
<td class="numcol">27</td>
<td><i>Tumebacillus algifaecis</i> strain THMBR28</td>
</tr>
<tr class="sumtab-row">
<td><a href="https://ncbi.nlm.nih.gov/assembly/6720751">6720751</a></td>
<td class="numcol">21</td>
<td><i>Priestia megaterium</i> strain S2</td>
</tr>
<tr class="sumtab-row">
<td><a href="https://ncbi.nlm.nih.gov/assembly/9972241">9972241</a></td>
<td class="numcol">20</td>
<td><i>Moritella</i> sp. 36</td>
</tr>
<tr class="sumtab-row">
<td><a href="https://ncbi.nlm.nih.gov/assembly/9972251">9972251</a></td>
<td class="numcol">20</td>
<td><i>Moritella</i> sp. 28</td>
</tr>
<tr class="sumtab-row">
<td><a href="https://ncbi.nlm.nih.gov/assembly/9972261">9972261</a></td>
<td class="numcol">20</td>
<td><i>Moritella</i> sp. 5</td>
</tr>
<tr class="sumtab-row">
<td><a href="https://ncbi.nlm.nih.gov/assembly/2023811">2023811</a></td>
<td class="numcol">19</td>
<td><i>Metabacillus litoralis</i> strain Bac94</td>
</tr>
<tr class="sumtab-row">
<td><a href="https://ncbi.nlm.nih.gov/assembly/9513491">9513491</a></td>
<td class="numcol">18</td>
<td><i>Photobacterium ganghwense</i> strain C2.2</td>
</tr>
<tr class="sumtab-row">
<td><a href="https://ncbi.nlm.nih.gov/assembly/9972131">9972131</a></td>
<td class="numcol">18</td>
<td><i>Moritella</i> sp. 24</td>
</tr>
<tr class="sumtab-row">
<td><a href="https://ncbi.nlm.nih.gov/assembly/2201351">2201351</a></td>
<td class="numcol">18</td>
<td><i>Peribacillus asahii</i> strain OM18</td>
</tr>
<tr>
<td colspan="4" class="subhead">Archaea</td>
</tr>
<tr class="sumtab-row">
<td><a href="https://ncbi.nlm.nih.gov/assembly/5073821">5073821</a></td>
<td class="numcol">5</td>
<td><i>Natronorubrum aibiense</i> strain 7-3</td>
</tr>
<tr class="sumtab-row">
<td><a href="https://ncbi.nlm.nih.gov/assembly/10842511">10842511</a></td>
<td class="numcol">5</td>
<td><i>Natrinema</i> sp. SYSU A 869</td>
</tr>
<tr class="sumtab-row">
<td><a href="https://ncbi.nlm.nih.gov/assembly/2580821">2580821</a></td>
<td class="numcol">5</td>
<td><i>Natronorubrum bangense</i> strain JCM 10635</td>
</tr>
<tr class="sumtab-row">
<td><a href="https://ncbi.nlm.nih.gov/assembly/521038">521038</a></td>
<td class="numcol">4</td>
<td><i>Natronococcus occultus</i> strain SP4</td>
</tr>
<tr class="sumtab-row">
<td><a href="https://ncbi.nlm.nih.gov/assembly/9298621">9298621</a></td>
<td class="numcol">4</td>
<td><i>Haloterrigena salifodinae</i> strain BOL5-1</td>
</tr>
<tr class="sumtab-row">
<td><a href="https://ncbi.nlm.nih.gov/assembly/9962651">9962651</a></td>
<td class="numcol">4</td>
<td><i>Haloarcula sinaiiensis</i> strain ATCC 33800</td>
</tr>
<tr class="sumtab-row">
<td><a href="https://ncbi.nlm.nih.gov/assembly/10123301">10123301</a></td>
<td class="numcol">4</td>
<td><i>Methanospirillum</i> sp. J.3.6.1-F.2.7.3</td>
</tr>
<tr class="sumtab-row">
<td><a href="https://ncbi.nlm.nih.gov/assembly/4349641">4349641</a></td>
<td class="numcol">4</td>
<td><i>Methanosphaera stadtmanae</i> strain MGYG-HGUT-02164</td>
</tr>
<tr class="sumtab-row">
<td><a href="https://ncbi.nlm.nih.gov/assembly/34548">34548</a></td>
<td class="numcol">4</td>
<td><i>Methanospirillum hungatei</i> strain JF-1</td>
</tr>
<tr class="sumtab-row">
<td><a href="https://ncbi.nlm.nih.gov/assembly/33648">33648</a></td>
<td class="numcol">4</td>
<td><i>Methanosphaera stadtmanae</i> strain DSM 3091</td>
</tr>
<tr class="sumtab-row">
<td><a href="https://ncbi.nlm.nih.gov/assembly/10519241">10519241</a></td>
<td class="numcol">4</td>
<td><i>Methanospirillum hungatei</i> strain GP1</td>
</tr>
<tr class="sumtab-row">
<td><a href="https://ncbi.nlm.nih.gov/assembly/38268">38268</a></td>
<td class="numcol">4</td>
<td><i>Methanococcus vannielii</i> strain SB</td>
</tr>
</tbody>
</table>
<div id="top-var" class="pad-anchor"></div>
<table class="sum-table">
<caption>Top-10 genomes with highest intragenomic variability of 16S rRNA genes</caption>
<tbody>
<tr><th class="alnleft">Assembly ID</th><th class="numcol">Sum of entropy<sup>&nbsp;*</sup><br>(bits)</th><th class="numcol">Mean entropy<sup>&nbsp;*</sup><br>(bits)</th><th class="numcol">Number<br>of variable<br>positions</th><th class="numcol">Gene<br>copy<br>number</th><th class="alnleft">Organism</th></tr>
<tr>
<td colspan="6" class="subhead">Bacteria</td>
</tr>
<tr class="sumtab-row">
<td><a href="https://www.ncbi.nlm.nih.gov/assembly/4155511">4155511</a></td>
<td class="numcol">226.25</td>
<td class="numcol">0.13</td>
<td class="numcol">247</td>
<td class="numcol">12</td>
<td><i>Sporomusa termitida</i> strain DSM 4440</td>
</tr>
<tr class="sumtab-row">
<td><a href="https://www.ncbi.nlm.nih.gov/assembly/7294871">7294871</a></td>
<td class="numcol">217.64</td>
<td class="numcol">0.12</td>
<td class="numcol">237</td>
<td class="numcol">3</td>
<td><i>Campylobacter hyointestinalis</i> strain CHY5</td>
</tr>
<tr class="sumtab-row">
<td><a href="https://www.ncbi.nlm.nih.gov/assembly/1101781">1101781</a></td>
<td class="numcol">211.21</td>
<td class="numcol">0.12</td>
<td class="numcol">230</td>
<td class="numcol">3</td>
<td><i>Campylobacter</i> sp. RM6137</td>
</tr>
<tr class="sumtab-row">
<td><a href="https://www.ncbi.nlm.nih.gov/assembly/9010851">9010851</a></td>
<td class="numcol">184.58</td>
<td class="numcol">0.12</td>
<td class="numcol">201</td>
<td class="numcol">3</td>
<td><i>Sinorhizobium meliloti</i> strain AK76</td>
</tr>
<tr class="sumtab-row">
<td><a href="https://www.ncbi.nlm.nih.gov/assembly/8227731">8227731</a></td>
<td class="numcol">167.21</td>
<td class="numcol">0.10</td>
<td class="numcol">216</td>
<td class="numcol">5</td>
<td><i>Klebsiella pneumoniae</i> strain GZ-1</td>
</tr>
<tr class="sumtab-row">
<td><a href="https://www.ncbi.nlm.nih.gov/assembly/1469691">1469691</a></td>
<td class="numcol">145.25</td>
<td class="numcol">0.10</td>
<td class="numcol">156</td>
<td class="numcol">3</td>
<td><i>Olleya</i> sp. Bg11-27</td>
</tr>
<tr class="sumtab-row">
<td><a href="https://www.ncbi.nlm.nih.gov/assembly/7292581">7292581</a></td>
<td class="numcol">136.25</td>
<td class="numcol">0.09</td>
<td class="numcol">145</td>
<td class="numcol">4</td>
<td><i>Microbulbifer</i> sp. YPW1</td>
</tr>
<tr class="sumtab-row">
<td><a href="https://www.ncbi.nlm.nih.gov/assembly/638441">638441</a></td>
<td class="numcol">135.84</td>
<td class="numcol">0.08</td>
<td class="numcol">138</td>
<td class="numcol">4</td>
<td><i>Selenomonas</i> sp. F0591</td>
</tr>
<tr class="sumtab-row">
<td><a href="https://www.ncbi.nlm.nih.gov/assembly/9428571">9428571</a></td>
<td class="numcol">122.06</td>
<td class="numcol">0.08</td>
<td class="numcol">123</td>
<td class="numcol">4</td>
<td><i>Thermobispora bispora</i> strain ZC4</td>
</tr>
<tr class="sumtab-row">
<td><a href="https://www.ncbi.nlm.nih.gov/assembly/9307231">9307231</a></td>
<td class="numcol">117.00</td>
<td class="numcol">0.08</td>
<td class="numcol">117</td>
<td class="numcol">2</td>
<td><i>Thermoleptolyngbya</i> sp. PKUAC-SCTB121</td>
</tr>
<tr>
<td colspan="6" class="subhead">Archaea</td>
</tr>
<tr class="sumtab-row">
<td><a href="https://www.ncbi.nlm.nih.gov/assembly/4982121">4982121</a></td>
<td class="numcol">137.00</td>
<td class="numcol">0.09</td>
<td class="numcol">137</td>
<td class="numcol">2</td>
<td><i>Halomicrobium</i> sp. ZPS1 <sup>**</sup></td>
</tr>
<tr class="sumtab-row">
<td><a href="https://www.ncbi.nlm.nih.gov/assembly/2582391">2582391</a></td>
<td class="numcol">125.81</td>
<td class="numcol">0.09</td>
<td class="numcol">137</td>
<td class="numcol">3</td>
<td><i>Halomicrobium mukohataei</i> strain JP60</td>
</tr>
<tr class="sumtab-row">
<td><a href="https://www.ncbi.nlm.nih.gov/assembly/999901">999901</a></td>
<td class="numcol">98.00</td>
<td class="numcol">0.07</td>
<td class="numcol">98</td>
<td class="numcol">2</td>
<td><i>Halorientalis</i> sp. IM1011</td>
</tr>
<tr class="sumtab-row">
<td><a href="https://www.ncbi.nlm.nih.gov/assembly/8834941">8834941</a></td>
<td class="numcol">97.00</td>
<td class="numcol">0.07</td>
<td class="numcol">97</td>
<td class="numcol">2</td>
<td><i>Halosimplex litoreum</i> strain YGH94</td>
</tr>
<tr class="sumtab-row">
<td><a href="https://www.ncbi.nlm.nih.gov/assembly/5205301">5205301</a></td>
<td class="numcol">88.16</td>
<td class="numcol">0.06</td>
<td class="numcol">96</td>
<td class="numcol">3</td>
<td><i>Halomicrobium</i> sp. LC1Hm</td>
</tr>
<tr class="sumtab-row">
<td><a href="https://www.ncbi.nlm.nih.gov/assembly/2835121">2835121</a></td>
<td class="numcol">78.06</td>
<td class="numcol">0.05</td>
<td class="numcol">85</td>
<td class="numcol">3</td>
<td><i>Haloarcula marismortui</i> strain ATCC 43049</td>
</tr>
<tr class="sumtab-row">
<td><a href="https://www.ncbi.nlm.nih.gov/assembly/247521">247521</a></td>
<td class="numcol">72.55</td>
<td class="numcol">0.05</td>
<td class="numcol">79</td>
<td class="numcol">3</td>
<td><i>Haloarcula</i> sp. CBA1115</td>
</tr>
<tr class="sumtab-row">
<td><a href="https://www.ncbi.nlm.nih.gov/assembly/9962651">9962651</a></td>
<td class="numcol">67.78</td>
<td class="numcol">0.05</td>
<td class="numcol">82</td>
<td class="numcol">4</td>
<td><i>Haloarcula sinaiiensis</i> strain ATCC 33800</td>
</tr>
<tr class="sumtab-row">
<td><a href="https://www.ncbi.nlm.nih.gov/assembly/1492601">1492601</a></td>
<td class="numcol">66.00</td>
<td class="numcol">0.04</td>
<td class="numcol">66</td>
<td class="numcol">2</td>
<td><i>Haloarcula taiwanensis</i> strain <i>Taiwanensis</i></td>
</tr>
<tr class="sumtab-row">
<td><a href="https://www.ncbi.nlm.nih.gov/assembly/10843081">10843081</a></td>
<td class="numcol">58.00</td>
<td class="numcol">0.04</td>
<td class="numcol">58</td>
<td class="numcol">2</td>
<td><i>Halobaculum roseum</i> strain CGMCC 1.15501</td>
</tr>
</tbody>
</table>
<p><sup>*</sup> Entropy is <a href="https://en.wikipedia.org/wiki/Entropy_(information_theory)">Shannon entropy</a> calculated for each column of the multiple sequence alignment (MSA) of all full-length 16S rRNA genes of a genome. Entropy is then summed up (column "Sum of entropy") and averaged (column "Mean entropy").</p>
<p><sup>**</sup> <i>Halomicrobium</i> sp. ZPS1 is a quite remarkable case. This genome harbours two 16S rRNA genes, therefore entropy is equal to the number of mismatching nucleotides between sequences of the genes. Respectively, per cent of identity between these two gene sequences is 90.70%! This is remarkable, because the usual (however arbitrary) <em>genus</em> demarcation threshold of per cent of identity is 95%.</p>
<hr />
<p>RiboGrove, 2022-08-05</p>
</div>
</body>
</html>
