<!DOCTYPE html>
<html lang="en">
<head>
<title>RiboGrove mirror</title>
<link rel="stylesheet" href="../css/style.css">
<link rel="stylesheet" href="../css/release-page.css">
</head>
<body>
<div class="container">
<br />
<p><div class="warn-section">The <a href="http://mbio.bas-net.by/cager/en/ribogrove">main website</a> where RiboGrove is hosted may be unavailable outside Belarus due to technical troubles and the overall disaster.<br />Hence this mirror has been created, and RiboGrove files are available through Dropbox (links are <a href="#downloads">below</a>).</div></p><br />
<p><img src="../img/RiboGrove_logo.png" /><br /><br /></p>
<h2>Contents</h2>
<ul>
<li><a href="#overview">Overview</a></li>
<li><a href="#downloads">Downloads</a></li>
<li><a href="#summary">Summary</a>
<ul style="list-style-type: none;">
<li><a href="#db-size">RiboGrove size</a></li>
<li><a href="#gene-lens">16S rRNA gene lengths</a></li>
<li><a href="#copy-number">16S rRNA gene copy number</a></li>
<li><a href="#top-longest">Top-10 longest 16S rRNA genes</a></li>
<li><a href="#top-shortest">Top-10 shortest 16S rRNA genes</a></li>
<li><a href="#top-copy-num">Top-10 genomes with largest 16S rRNA copy numbers</a></li>
<li><a href="#top-var">Top-10 genomes with highest intragenomic variability of 16S rRNA genes</a></li>
</ul>
</li>
<li><a href="#searching-data">Searching data in RiboGrove</a>
<ul style="list-style-type: none;">
<li><a href="#subset-by-header">Subsetting sequences by header</a></li>
<li><a href="#subset-by-len">Subsetting sequences by length</a></li>
<li><a href="#select-head">Selecting header data</a></li>
</ul>
</li>
<li><a href="#citing-ribogrove">Citing RiboGrove</a></li>
</ul>
<hr />
<h2>Overview</h2>
<div id="overview" class="pad-anchor"></div>
<p>RiboGrove is a database of 16S rRNA gene sequences of bacteria and archaea. RiboGrove contains only full-length sequences of 16S rRNA genes since amplicon data is not used for its construction. In other words, unlike similar rRNA resources like <a href="https://www.arb-silva.de/">Silva</a> and <a href="http://rdp.cme.msu.edu/">RDP</a>, RiboGrove does not contain partial gene sequences.</p>
<p>Completely assembled prokaryotic genomes deposited in <a href="https://www.ncbi.nlm.nih.gov/refseq/">RefSeq</a> database were used for the RiboGrove construction.</p>
<p>The software used for the RiboGrove construction can be found in the following GitHub repository: <a href="https://github.com/masikol/ribogrove-tools">ribogrove-tools</a>.</p>
<hr />
<h2>Downloads</h2>
<div id="downloads" class="pad-anchor"></div>
<h3>Release &mdash; 4.210 (2022-01-12)</h3>
<p>The release is based on RefSeq release 210.</p>
<ul>
<li>A fasta file of full-length 16S gene sequences. <a href="https://www.dropbox.com/s/erfpgop4n95nnrj/ribogrove_4.210_sequences.fasta.gz?dl=0">Download (gzipped fasta file, 5.25 MB)</a></li>
<li>A <q>raw</q> version of the fasta file above. This file contains partial sequences. <a href="https://www.dropbox.com/s/bnmmudtxxrh4fbe/raw_ribogrove_4.210_sequences.fasta.gz?dl=0">Download (gzipped fasta file, 5.50 MB)</a></li>
<li>Metadata <a href="https://www.dropbox.com/s/835cu999ejtztic/metadata_ribogrove_4.210.zip?dl=0">Download (zip archive 15.50 MB)</a></li>
</ul>
<p>The fasta files are compressed with gzip, and the metadata file is a zip archive. To uncompress them, Linux and Mac OS users may use gzip and zip programs, they should be built-in. For Windows users, the free and open source (de)compression program <a href="https://www.7-zip.org/">7-Zip</a> is available.</p>
<p>You can find all releases at the <a href="ribogrove_archive_en.html">RiboGrove release archive</a>.</p>
<hr />
<h2>Summary</h2>
<div id="summary" class="pad-anchor"></div>
<table class="sum-table"><caption>RiboGrove size
<div id="db-size" class="pad-anchor"></div>
</caption>
<tbody>
<tr><th></th><th class="numcol">Bacteria</th><th class="numcol">Archaea</th><th class="numcol">Total</th></tr>
<tr class="sumtab-row">
<td>Number of gene sequences</td>
<td class="numcol">128,152</td>
<td class="numcol">699</td>
<td class="numcol">128,851</td>
</tr>
<tr class="sumtab-row">
<td>Number of unique gene sequences</td>
<td class="numcol">36,349</td>
<td class="numcol">504</td>
<td class="numcol">36,853</td>
</tr>
<tr class="sumtab-row">
<td>Number of species</td>
<td class="numcol">6,743</td>
<td class="numcol">321</td>
<td class="numcol">7,064</td>
</tr>
<tr class="sumtab-row">
<td>Number of genomes</td>
<td class="numcol">24,505</td>
<td class="numcol">410</td>
<td class="numcol">24,915</td>
</tr>
<tr class="sumtab-row">
<td>Number of genomes of category 1</td>
<td class="numcol">16,008</td>
<td class="numcol">122</td>
<td class="numcol">16,130</td>
</tr>
<tr class="sumtab-row">
<td>Number of genomes of category 2</td>
<td class="numcol">8,369</td>
<td class="numcol">288</td>
<td class="numcol">8,657</td>
</tr>
<tr class="sumtab-row">
<td>Number of genomes of category 3</td>
<td class="numcol">128</td>
<td class="numcol">0</td>
<td class="numcol">128</td>
</tr>
</tbody>
</table>
<table class="sum-table"><caption>16S rRNA gene lengths
<div id="gene-lens" class="pad-anchor"></div>
</caption>
<tbody>
<tr><th></th><th class="numcol">Bacteria</th><th class="numcol">Archaea</th></tr>
<tr class="sumtab-row">
<td>Minimum (bp)</td>
<td class="numcol">1,448.00</td>
<td class="numcol">1,439.00</td>
</tr>
<tr class="sumtab-row">
<td>25th percentile (bp) <sup>*</sup></td>
<td class="numcol">1,517.25</td>
<td class="numcol">1,472.00</td>
</tr>
<tr class="sumtab-row">
<td>Median (bp) <sup>*</sup></td>
<td class="numcol">1,532.00</td>
<td class="numcol">1,475.00</td>
</tr>
<tr class="sumtab-row">
<td>75th percentile (bp) <sup>*</sup></td>
<td class="numcol">1,543.00</td>
<td class="numcol">1,489.00</td>
</tr>
<tr class="sumtab-row">
<td>Average (bp) <sup>*</sup></td>
<td class="numcol">1,528.38</td>
<td class="numcol">1,500.67</td>
</tr>
<tr class="sumtab-row">
<td>Mode (bp) <sup>*</sup></td>
<td class="numcol">1,537.00</td>
<td class="numcol">1,473.00</td>
</tr>
<tr class="sumtab-row">
<td>Maximum (bp)</td>
<td class="numcol">2,438.00</td>
<td class="numcol">3,604.00</td>
</tr>
<tr class="sumtab-row">
<td>Standard deviation (bp) <sup>*</sup></td>
<td class="numcol">26.04</td>
<td class="numcol">149.64</td>
</tr>
</tbody>
</table>
<p><sup>*</sup> &mdash; metrics marked with this sign were calculated with preliminary normalization, i.e. median within-species gene length were used for the summary.</p>
<table class="sum-table"><caption>16S rRNA gene copy number
<div id="copy-number" class="pad-anchor"></div>
</caption>
<tbody>
<tr><th class="numcol">Copy number <sup>*</sup></th><th class="numcol">Number of species</th><th class="numcol">Per cent of species (%)</th></tr>
<tr class="sumtab-row">
<td class="numcol">1</td>
<td class="numcol">982</td>
<td class="numcol">13.90</td>
</tr>
<tr class="sumtab-row">
<td class="numcol">2</td>
<td class="numcol">1,354</td>
<td class="numcol">19.17</td>
</tr>
<tr class="sumtab-row">
<td class="numcol">3</td>
<td class="numcol">1,088</td>
<td class="numcol">15.40</td>
</tr>
<tr class="sumtab-row">
<td class="numcol">4</td>
<td class="numcol">897</td>
<td class="numcol">12.70</td>
</tr>
<tr class="sumtab-row">
<td class="numcol">5</td>
<td class="numcol">560</td>
<td class="numcol">7.93</td>
</tr>
<tr class="sumtab-row">
<td class="numcol">6</td>
<td class="numcol">698</td>
<td class="numcol">9.88</td>
</tr>
<tr class="sumtab-row">
<td class="numcol">7</td>
<td class="numcol">565</td>
<td class="numcol">8.00</td>
</tr>
<tr class="sumtab-row">
<td class="numcol">8</td>
<td class="numcol">370</td>
<td class="numcol">5.24</td>
</tr>
<tr class="sumtab-row">
<td class="numcol">9</td>
<td class="numcol">178</td>
<td class="numcol">2.52</td>
</tr>
<tr class="sumtab-row">
<td class="numcol">10</td>
<td class="numcol">152</td>
<td class="numcol">2.15</td>
</tr>
<tr class="sumtab-row">
<td class="numcol">11</td>
<td class="numcol">71</td>
<td class="numcol">1.01</td>
</tr>
<tr class="sumtab-row">
<td class="numcol">12</td>
<td class="numcol">49</td>
<td class="numcol">0.69</td>
</tr>
<tr class="sumtab-row">
<td class="numcol">13</td>
<td class="numcol">34</td>
<td class="numcol">0.48</td>
</tr>
<tr class="sumtab-row">
<td class="numcol">14</td>
<td class="numcol">40</td>
<td class="numcol">0.57</td>
</tr>
<tr class="sumtab-row">
<td class="numcol">15</td>
<td class="numcol">7</td>
<td class="numcol">0.10</td>
</tr>
<tr class="sumtab-row">
<td class="numcol">16</td>
<td class="numcol">6</td>
<td class="numcol">0.08</td>
</tr>
<tr class="sumtab-row">
<td class="numcol">17</td>
<td class="numcol">4</td>
<td class="numcol">0.06</td>
</tr>
<tr class="sumtab-row">
<td class="numcol">18</td>
<td class="numcol">4</td>
<td class="numcol">0.06</td>
</tr>
<tr class="sumtab-row">
<td class="numcol">20</td>
<td class="numcol">3</td>
<td class="numcol">0.04</td>
</tr>
<tr class="sumtab-row">
<td class="numcol">27</td>
<td class="numcol">1</td>
<td class="numcol">0.01</td>
</tr>
<tr class="sumtab-row">
<td class="numcol">37</td>
<td class="numcol">1</td>
<td class="numcol">0.01</td>
</tr>
</tbody>
</table>
<p><sup>*</sup> &mdash; these are median within-species copy numbers.</p>
<table class="sum-table"><caption>Top-10 longest 16S rRNA genes
<div id="top-longest" class="pad-anchor"></div>
</caption>
<tbody>
<tr><th class="alnleft">Assembly ID</th><th class="numcol">Gene length (bp)</th><th class="alnleft">RiboGrove Sequence ID(s)</th><th class="alnleft">Organism</th></tr>
<tr>
<td colspan="4" class="subhead">Bacteria</td>
</tr>
<tr class="sumtab-row">
<td><a href="https://ncbi.nlm.nih.gov/assembly/10898951">10898951</a></td>
<td class="numcol">2,438</td>
<td class="seqid-td">NZ_AP024929.1:249100-251537_minus</td>
<td><i>Thermus thermophilus</i> strain AA2-2</td>
</tr>
<tr class="sumtab-row">
<td><a href="https://ncbi.nlm.nih.gov/assembly/11277031">11277031</a></td>
<td class="numcol">1,887</td>
<td class="seqid-td">NZ_CP045876.1:290071-291957_minus</td>
<td><i>Candidatus Annandia pinicola</i> isolate Ad13-065</td>
</tr>
<tr class="sumtab-row">
<td><a href="https://ncbi.nlm.nih.gov/assembly/8028891">8028891</a></td>
<td class="numcol">1,806</td>
<td class="seqid-td">NZ_AP022826.1:1258017-1259822_minus<br />NZ_AP022826.1:1532588-1534393_minus<br />NZ_AP022826.1:1939914-1941719_minus</td>
<td><i>Nitrosophilus labii</i> strain HRV44</td>
</tr>
<tr class="sumtab-row">
<td><a href="https://ncbi.nlm.nih.gov/assembly/10731991">10731991</a></td>
<td class="numcol">1,788</td>
<td class="seqid-td">NZ_AP024619.1:2016182-2017969_minus</td>
<td><i>Gelria</i> sp. Kuro-4</td>
</tr>
<tr class="sumtab-row">
<td><a href="https://ncbi.nlm.nih.gov/assembly/282748">282748</a></td>
<td class="numcol">1,781</td>
<td class="seqid-td">NC_014964.1:2252888-2254668_minus</td>
<td><i>Thermoanaerobacter brockii</i> strain Ako-1</td>
</tr>
<tr class="sumtab-row">
<td><a href="https://ncbi.nlm.nih.gov/assembly/40148">40148</a></td>
<td class="numcol">1,781</td>
<td class="seqid-td">NC_010321.1:2265744-2267524_minus</td>
<td><i>Thermoanaerobacter pseudethanolicus</i> strain ATCC 33223</td>
</tr>
<tr class="sumtab-row">
<td><a href="https://ncbi.nlm.nih.gov/assembly/4499991">4499991</a></td>
<td class="numcol">1,744</td>
<td class="seqid-td">NZ_CP043427.1:609141-610884_plus<br />NZ_CP043427.1:930699-932442_minus<br />NZ_CP043427.1:1503078-1504821_minus</td>
<td><i>Campylobacter sputorum</i> strain LMG 7795</td>
</tr>
<tr class="sumtab-row">
<td><a href="https://ncbi.nlm.nih.gov/assembly/1153941">1153941</a></td>
<td class="numcol">1,744</td>
<td class="seqid-td">NZ_CP019682.1:607981-609724_plus<br />NZ_CP019682.1:929565-931308_minus<br />NZ_CP019682.1:1501945-1503688_minus</td>
<td><i>Campylobacter sputorum</i> strain RM3237</td>
</tr>
<tr class="sumtab-row">
<td><a href="https://ncbi.nlm.nih.gov/assembly/1153911">1153911</a></td>
<td class="numcol">1,743</td>
<td class="seqid-td">NZ_CP019683.1:606847-608589_plus<br />NZ_CP019683.1:935163-936905_minus<br />NZ_CP019683.1:1558189-1559931_minus</td>
<td><i>Campylobacter sputorum</i> CCUG 20703</td>
</tr>
<tr class="sumtab-row">
<td><a href="https://ncbi.nlm.nih.gov/assembly/1101781">1101781</a></td>
<td class="numcol">1,742</td>
<td class="seqid-td">NZ_CP018789.1:273370-275111_plus<br />NZ_CP018789.1:1545743-1547484_minus</td>
<td><i>Campylobacter</i> sp. RM6137</td>
</tr>
<tr class="sumtab-row">
<td><a href="https://ncbi.nlm.nih.gov/assembly/1153931">1153931</a></td>
<td class="numcol">1,742</td>
<td class="seqid-td">NZ_CP019685.1:577810-579551_plus<br />NZ_CP019685.1:891862-893603_minus<br />NZ_CP019685.1:1479764-1481505_minus</td>
<td><i>Campylobacter sputorum</i> strain RM8705</td>
</tr>
<tr class="sumtab-row">
<td><a href="https://ncbi.nlm.nih.gov/assembly/7294871">7294871</a></td>
<td class="numcol">1,742</td>
<td class="seqid-td">NZ_CP053828.1:357136-358877_plus<br />NZ_CP053828.1:1667816-1669557_minus</td>
<td><i>Campylobacter hyointestinalis</i> strain CHY5</td>
</tr>
<tr>
<td colspan="4" class="subhead">Archaea</td>
</tr>
<tr class="sumtab-row">
<td><a href="https://ncbi.nlm.nih.gov/assembly/351728">351728</a></td>
<td class="numcol">3,604</td>
<td class="seqid-td">NC_016645.1:127214-130817_plus</td>
<td><i>Pyrobaculum ferrireducens</i> strain 1860</td>
</tr>
<tr class="sumtab-row">
<td><a href="https://ncbi.nlm.nih.gov/assembly/28808">28808</a></td>
<td class="numcol">2,213</td>
<td class="seqid-td">NC_003364.1:1089640-1091852_plus</td>
<td><i>Pyrobaculum aerophilum</i> strain IM2</td>
</tr>
<tr class="sumtab-row">
<td><a href="https://ncbi.nlm.nih.gov/assembly/37488">37488</a></td>
<td class="numcol">2,212</td>
<td class="seqid-td">NC_009376.1:623323-625534_minus</td>
<td><i>Pyrobaculum arsenaticum</i> strain DSM 13514</td>
</tr>
<tr class="sumtab-row">
<td><a href="https://ncbi.nlm.nih.gov/assembly/32288">32288</a></td>
<td class="numcol">2,202</td>
<td class="seqid-td">NC_000854.2:1218712-1220913_minus</td>
<td><i>Aeropyrum pernix</i> strain K1</td>
</tr>
<tr class="sumtab-row">
<td><a href="https://ncbi.nlm.nih.gov/assembly/40848">40848</a></td>
<td class="numcol">2,197</td>
<td class="seqid-td">NC_010525.1:690419-692615_plus</td>
<td><i>Pyrobaculum neutrophilum</i> strain V24Sta</td>
</tr>
<tr class="sumtab-row">
<td><a href="https://ncbi.nlm.nih.gov/assembly/1145431">1145431</a></td>
<td class="numcol">2,008</td>
<td class="seqid-td">NZ_CP019964.1:751297-753304_minus</td>
<td><i>Candidatus Mancarchaeum acidiphilum</i> strain Mia14</td>
</tr>
<tr class="sumtab-row">
<td><a href="https://ncbi.nlm.nih.gov/assembly/9220081">9220081</a></td>
<td class="numcol">2,003</td>
<td class="seqid-td">NZ_CP060530.1:203642-205644_minus</td>
<td><i>Candidatus Micrarchaeum</i> sp. A_DKE</td>
</tr>
<tr class="sumtab-row">
<td><a href="https://ncbi.nlm.nih.gov/assembly/39388">39388</a></td>
<td class="numcol">1,679</td>
<td class="seqid-td">NC_009954.1:129150-130828_minus</td>
<td><i>Caldivirga maquilingensis</i> strain IC-167</td>
</tr>
<tr class="sumtab-row">
<td><a href="https://ncbi.nlm.nih.gov/assembly/127981">127981</a></td>
<td class="numcol">1,650</td>
<td class="seqid-td">NC_022521.1:1165168-1166817_minus</td>
<td><i>Aeropyrum camini</i> strain SY1</td>
</tr>
<tr class="sumtab-row">
<td><a href="https://ncbi.nlm.nih.gov/assembly/304318">304318</a></td>
<td class="numcol">1,576</td>
<td class="seqid-td">NC_015931.1:84671-86246_minus</td>
<td><i>Pyrolobus fumarii</i> strain 1A</td>
</tr>
</tbody>
</table>
<table class="sum-table"><caption>Top-10 shortest 16S rRNA genes
<div id="top-shortest" class="pad-anchor"></div>
</caption>
<tbody>
<tr><th class="alnleft">Assembly ID</th><th class="numcol">Gene length (bp)</th><th class="alnleft">RiboGrove Sequence ID(s)</th><th class="alnleft">Organism</th></tr>
<tr>
<td colspan="4" class="subhead">Bacteria</td>
</tr>
<tr class="sumtab-row">
<td><a href="https://ncbi.nlm.nih.gov/assembly/44428">44428</a></td>
<td class="numcol">1,448</td>
<td class="seqid-td">NC_012982.1:2336679-2338126_minus</td>
<td><i>Hirschia baltica</i> strain ATCC 49814</td>
</tr>
<tr class="sumtab-row">
<td><a href="https://ncbi.nlm.nih.gov/assembly/1460951">1460951</a></td>
<td class="numcol">1,449</td>
<td class="seqid-td">NZ_CP021913.1:2386837-2388285_plus<br />NZ_CP021913.1:3597920-3599368_plus</td>
<td><i>Sagittula</i> sp. P11</td>
</tr>
<tr class="sumtab-row">
<td><a href="https://ncbi.nlm.nih.gov/assembly/860061">860061</a></td>
<td class="numcol">1,451</td>
<td class="seqid-td">NZ_CP017718.1:2304269-2305719_minus</td>
<td><i>Hyphomonas</i> sp. Mor2</td>
</tr>
<tr class="sumtab-row">
<td><a href="https://ncbi.nlm.nih.gov/assembly/1163161">1163161</a></td>
<td class="numcol">1,453</td>
<td class="seqid-td">NZ_CP022540.1:1369380-1370832_plus<br />NZ_CP022540.1:2482480-2483932_plus</td>
<td><i>Antarctobacter heliothermus</i> strain SMS3</td>
</tr>
<tr class="sumtab-row">
<td><a href="https://ncbi.nlm.nih.gov/assembly/6279751">6279751</a></td>
<td class="numcol">1,454</td>
<td class="seqid-td">NZ_AP022337.1:267139-268592_plus<br />NZ_AP022337.1:1420942-1422395_plus<br />NZ_AP022337.1:3191208-3192661_minus</td>
<td><i>Mameliella alba</i> strain KU6B</td>
</tr>
<tr class="sumtab-row">
<td><a href="https://ncbi.nlm.nih.gov/assembly/9503471">9503471</a></td>
<td class="numcol">1,455</td>
<td class="seqid-td">NZ_CP022271.1:2407999-2409453_minus</td>
<td><i>Hyphomonas</i> sp. KY3</td>
</tr>
<tr class="sumtab-row">
<td><a href="https://ncbi.nlm.nih.gov/assembly/34128">34128</a></td>
<td class="numcol">1,455</td>
<td class="seqid-td">NC_008358.1:2818466-2819920_minus</td>
<td><i>Hyphomonas neptunium</i> strain ATCC 15444</td>
</tr>
<tr class="sumtab-row">
<td><a href="https://ncbi.nlm.nih.gov/assembly/10854641">10854641</a></td>
<td class="numcol">1,458</td>
<td class="seqid-td">NZ_CP065359.1:3157837-3159294_minus</td>
<td><i>Ruegeria</i> sp. SCSIO 43209</td>
</tr>
<tr class="sumtab-row">
<td><a href="https://ncbi.nlm.nih.gov/assembly/8694041">8694041</a></td>
<td class="numcol">1,458</td>
<td class="seqid-td">NZ_CP060436.1:2482207-2483664_minus</td>
<td><i>Pseudooceanicola algae</i> strain Lw-13e</td>
</tr>
<tr class="sumtab-row">
<td><a href="https://ncbi.nlm.nih.gov/assembly/1078381">1078381</a></td>
<td class="numcol">1,459</td>
<td class="seqid-td">NZ_CP020612.1:582021-583479_minus<br />NZ_CP020612.1:1166317-1167775_minus</td>
<td><i>Paracoccus contaminans</i> strain RKI 16-01929</td>
</tr>
<tr class="sumtab-row">
<td><a href="https://ncbi.nlm.nih.gov/assembly/9217271">9217271</a></td>
<td class="numcol">1,459</td>
<td class="seqid-td">NZ_CP069004.1:3093411-3094869_plus</td>
<td><i>Sulfitobacter mediterraneus</i> strain SC1-11</td>
</tr>
<tr class="sumtab-row">
<td><a href="https://ncbi.nlm.nih.gov/assembly/8738751">8738751</a></td>
<td class="numcol">1,459</td>
<td class="seqid-td">NZ_CP065429.1:477373-478831_plus</td>
<td><i>Sulfitobacter</i> sp. B30-2</td>
</tr>
<tr class="sumtab-row">
<td><a href="https://ncbi.nlm.nih.gov/assembly/8872011">8872011</a></td>
<td class="numcol">1,459</td>
<td class="seqid-td">NZ_CP065915.1:2729819-2731277_minus<br />NZ_CP065915.1:3593071-3594529_minus</td>
<td><i>Pelagovum pacificum</i> strain SM1903</td>
</tr>
<tr>
<td colspan="4" class="subhead">Archaea</td>
</tr>
<tr class="sumtab-row">
<td><a href="https://ncbi.nlm.nih.gov/assembly/39048">39048</a></td>
<td class="numcol">1,439</td>
<td class="seqid-td">NC_009776.1:728362-729800_plus</td>
<td><i>Ignicoccus hospitalis</i> strain KIN4/I</td>
</tr>
<tr class="sumtab-row">
<td><a href="https://ncbi.nlm.nih.gov/assembly/10131521">10131521</a></td>
<td class="numcol">1,457</td>
<td class="seqid-td">NZ_LR792632.1:542755-544211_plus</td>
<td><i>Methanocaldococcus</i> sp. SG7</td>
</tr>
<tr class="sumtab-row">
<td><a href="https://ncbi.nlm.nih.gov/assembly/2220501">2220501</a></td>
<td class="numcol">1,463</td>
<td class="seqid-td">NZ_CP034692.1:397753-399215_minus</td>
<td><i>Halorubrum</i> sp. BOL3-1</td>
</tr>
<tr class="sumtab-row">
<td><a href="https://ncbi.nlm.nih.gov/assembly/7330651">7330651</a></td>
<td class="numcol">1,466</td>
<td class="seqid-td">NZ_CP058334.1:1530622-1532087_minus</td>
<td><i>Natronomonas halophila</i> strain C90</td>
</tr>
<tr class="sumtab-row">
<td><a href="https://ncbi.nlm.nih.gov/assembly/34548">34548</a></td>
<td class="numcol">1,466</td>
<td class="seqid-td">NC_007796.1:39814-41279_plus<br />NC_007796.1:1301079-1302544_minus<br />NC_007796.1:3501525-3502990_minus<br />NC_007796.1:3507609-3509074_minus</td>
<td><i>Methanospirillum hungatei</i> strain JF-1</td>
</tr>
<tr class="sumtab-row">
<td><a href="https://ncbi.nlm.nih.gov/assembly/599268">599268</a></td>
<td class="numcol">1,466</td>
<td class="seqid-td">NC_020913.1:283607-285072_plus</td>
<td><i>Candidatus Methanomethylophilus alvus</i> strain Mx1201</td>
</tr>
<tr class="sumtab-row">
<td><a href="https://ncbi.nlm.nih.gov/assembly/10123301">10123301</a></td>
<td class="numcol">1,466</td>
<td class="seqid-td">NZ_CP075546.1:133354-134819_plus<br />NZ_CP075546.1:825954-827419_plus<br />NZ_CP075546.1:872641-874106_plus<br />NZ_CP075546.1:1727419-1728884_plus</td>
<td><i>Methanospirillum</i> sp. J.3.6.1-F.2.7.3</td>
</tr>
<tr class="sumtab-row">
<td><a href="https://ncbi.nlm.nih.gov/assembly/10519241">10519241</a></td>
<td class="numcol">1,466</td>
<td class="seqid-td">NZ_CP077107.1:4649-6114_plus<br />NZ_CP077107.1:1359562-1361027_minus<br />NZ_CP077107.1:1365502-1366967_minus<br />NZ_CP077107.1:1986020-1987485_minus</td>
<td><i>Methanospirillum hungatei</i> strain GP1</td>
</tr>
<tr class="sumtab-row">
<td><a href="https://ncbi.nlm.nih.gov/assembly/4352521">4352521</a></td>
<td class="numcol">1,466</td>
<td class="seqid-td">NZ_LR699000.1:283607-285072_plus</td>
<td><i>Candidatus Methanomethylophilus alvus</i> strain MGYG-HGUT-02456</td>
</tr>
<tr class="sumtab-row">
<td><a href="https://ncbi.nlm.nih.gov/assembly/2068141">2068141</a></td>
<td class="numcol">1,466</td>
<td class="seqid-td">NZ_CP017686.1:283608-285073_plus</td>
<td><i>Candidatus Methanomethylophilus alvus</i> strain Mx-05</td>
</tr>
</tbody>
</table>
<table class="sum-table"><caption>Top-10 genomes with largest 16S rRNA copy numbers
<div id="top-copy-num" class="pad-anchor"></div>
</caption>
<tbody>
<tr><th class="alnleft">Assembly ID</th><th class="numcol">Copy number</th><th class="alnleft">Organism</th></tr>
<tr>
<td colspan="4" class="subhead">Bacteria</td>
</tr>
<tr class="sumtab-row">
<td><a href="https://ncbi.nlm.nih.gov/assembly/1115491">1115491</a></td>
<td class="numcol">37</td>
<td><i>Tumebacillus avium</i> stain AR23208</td>
</tr>
<tr class="sumtab-row">
<td><a href="https://ncbi.nlm.nih.gov/assembly/1166771">1166771</a></td>
<td class="numcol">27</td>
<td><i>Tumebacillus algifaecis</i> strain THMBR28</td>
</tr>
<tr class="sumtab-row">
<td><a href="https://ncbi.nlm.nih.gov/assembly/6720751">6720751</a></td>
<td class="numcol">21</td>
<td><i>Priestia megaterium</i> strain S2</td>
</tr>
<tr class="sumtab-row">
<td><a href="https://ncbi.nlm.nih.gov/assembly/9972241">9972241</a></td>
<td class="numcol">20</td>
<td><i>Moritella</i> sp. 36</td>
</tr>
<tr class="sumtab-row">
<td><a href="https://ncbi.nlm.nih.gov/assembly/9972251">9972251</a></td>
<td class="numcol">20</td>
<td><i>Moritella</i> sp. 28</td>
</tr>
<tr class="sumtab-row">
<td><a href="https://ncbi.nlm.nih.gov/assembly/9972261">9972261</a></td>
<td class="numcol">20</td>
<td><i>Moritella</i> sp. 5</td>
</tr>
<tr class="sumtab-row">
<td><a href="https://ncbi.nlm.nih.gov/assembly/2023811">2023811</a></td>
<td class="numcol">19</td>
<td><i>Metabacillus litoralis</i> strain Bac94</td>
</tr>
<tr class="sumtab-row">
<td><a href="https://ncbi.nlm.nih.gov/assembly/9513491">9513491</a></td>
<td class="numcol">18</td>
<td><i>Photobacterium ganghwense</i> strain C2.2</td>
</tr>
<tr class="sumtab-row">
<td><a href="https://ncbi.nlm.nih.gov/assembly/9972131">9972131</a></td>
<td class="numcol">18</td>
<td><i>Moritella</i> sp. 24</td>
</tr>
<tr class="sumtab-row">
<td><a href="https://ncbi.nlm.nih.gov/assembly/2201351">2201351</a></td>
<td class="numcol">18</td>
<td><i>Peribacillus asahii</i> strain OM18</td>
</tr>
<tr>
<td colspan="4" class="subhead">Archaea</td>
</tr>
<tr class="sumtab-row">
<td><a href="https://ncbi.nlm.nih.gov/assembly/5073821">5073821</a></td>
<td class="numcol">5</td>
<td><i>Natronorubrum aibiense</i> strain 7-3</td>
</tr>
<tr class="sumtab-row">
<td><a href="https://ncbi.nlm.nih.gov/assembly/2580821">2580821</a></td>
<td class="numcol">5</td>
<td><i>Natronorubrum bangense</i> strain JCM 10635</td>
</tr>
<tr class="sumtab-row">
<td><a href="https://ncbi.nlm.nih.gov/assembly/11622961">11622961</a></td>
<td class="numcol">5</td>
<td><i>Methanococcoides</i> sp. LMO-1</td>
</tr>
<tr class="sumtab-row">
<td><a href="https://ncbi.nlm.nih.gov/assembly/10842511">10842511</a></td>
<td class="numcol">5</td>
<td><i>Natrinema</i> sp. SYSU A 869</td>
</tr>
<tr class="sumtab-row">
<td><a href="https://ncbi.nlm.nih.gov/assembly/521038">521038</a></td>
<td class="numcol">4</td>
<td><i>Natronococcus occultus</i> strain SP4</td>
</tr>
<tr class="sumtab-row">
<td><a href="https://ncbi.nlm.nih.gov/assembly/11057071">11057071</a></td>
<td class="numcol">4</td>
<td><i>Halosiccatus urmianus</i> strain IBRC-M: 10911</td>
</tr>
<tr class="sumtab-row">
<td><a href="https://ncbi.nlm.nih.gov/assembly/4349641">4349641</a></td>
<td class="numcol">4</td>
<td><i>Methanosphaera stadtmanae</i> strain MGYG-HGUT-02164</td>
</tr>
<tr class="sumtab-row">
<td><a href="https://ncbi.nlm.nih.gov/assembly/33648">33648</a></td>
<td class="numcol">4</td>
<td><i>Methanosphaera stadtmanae</i> strain DSM 3091</td>
</tr>
<tr class="sumtab-row">
<td><a href="https://ncbi.nlm.nih.gov/assembly/11151391">11151391</a></td>
<td class="numcol">4</td>
<td><i>Halosiccatus</i> sp. LT50</td>
</tr>
<tr class="sumtab-row">
<td><a href="https://ncbi.nlm.nih.gov/assembly/34548">34548</a></td>
<td class="numcol">4</td>
<td><i>Methanospirillum hungatei</i> strain JF-1</td>
</tr>
<tr class="sumtab-row">
<td><a href="https://ncbi.nlm.nih.gov/assembly/11151361">11151361</a></td>
<td class="numcol">4</td>
<td><i>Halosiccatus</i> sp. LT50</td>
</tr>
<tr class="sumtab-row">
<td><a href="https://ncbi.nlm.nih.gov/assembly/38268">38268</a></td>
<td class="numcol">4</td>
<td><i>Methanococcus vannielii</i> strain SB</td>
</tr>
<tr class="sumtab-row">
<td><a href="https://ncbi.nlm.nih.gov/assembly/10519241">10519241</a></td>
<td class="numcol">4</td>
<td><i>Methanospirillum hungatei</i> strain GP1</td>
</tr>
<tr class="sumtab-row">
<td><a href="https://ncbi.nlm.nih.gov/assembly/9298621">9298621</a></td>
<td class="numcol">4</td>
<td><i>Haloterrigena salifodinae</i> strain BOL5-1</td>
</tr>
<tr class="sumtab-row">
<td><a href="https://ncbi.nlm.nih.gov/assembly/9962651">9962651</a></td>
<td class="numcol">4</td>
<td><i>Haloarcula sinaiiensis</i> strain ATCC 33800</td>
</tr>
<tr class="sumtab-row">
<td><a href="https://ncbi.nlm.nih.gov/assembly/10123301">10123301</a></td>
<td class="numcol">4</td>
<td><i>Methanospirillum</i> sp. J.3.6.1-F.2.7.3</td>
</tr>
</tbody>
</table>
<table class="sum-table"><caption>Top-10 genomes with highest intragenomic variability of 16S rRNA genes
<div id="top-var" class="pad-anchor"></div>
</caption>
<tbody>
<tr><th class="alnleft">Assembly ID</th><th class="numcol">Sum of entropy<sup>*</sup> (bits)</th><th class="numcol">Mean entropy<sup>*</sup> (bits)</th><th class="numcol">Number of variable positions</th><th class="numcol">Copy number</th><th class="alnleft">Organism</th></tr>
<tr>
<td colspan="6" class="subhead">Bacteria</td>
</tr>
<tr class="sumtab-row">
<td><a href="https://www.ncbi.nlm.nih.gov/assembly/4155511">4155511</a></td>
<td class="numcol">226.25</td>
<td class="numcol">0.13</td>
<td class="numcol">247</td>
<td class="numcol">12</td>
<td><i>Sporomusa termitida</i> strain DSM 4440</td>
</tr>
<tr class="sumtab-row">
<td><a href="https://www.ncbi.nlm.nih.gov/assembly/7294871">7294871</a></td>
<td class="numcol">217.64</td>
<td class="numcol">0.12</td>
<td class="numcol">237</td>
<td class="numcol">3</td>
<td><i>Campylobacter hyointestinalis</i> strain CHY5</td>
</tr>
<tr class="sumtab-row">
<td><a href="https://www.ncbi.nlm.nih.gov/assembly/1101781">1101781</a></td>
<td class="numcol">211.21</td>
<td class="numcol">0.12</td>
<td class="numcol">230</td>
<td class="numcol">3</td>
<td><i>Campylobacter</i> sp. RM6137</td>
</tr>
<tr class="sumtab-row">
<td><a href="https://www.ncbi.nlm.nih.gov/assembly/9010851">9010851</a></td>
<td class="numcol">184.58</td>
<td class="numcol">0.12</td>
<td class="numcol">201</td>
<td class="numcol">3</td>
<td><i>Sinorhizobium meliloti</i> strain AK76</td>
</tr>
<tr class="sumtab-row">
<td><a href="https://www.ncbi.nlm.nih.gov/assembly/8227731">8227731</a></td>
<td class="numcol">167.21</td>
<td class="numcol">0.10</td>
<td class="numcol">216</td>
<td class="numcol">5</td>
<td><i>Klebsiella pneumoniae</i> strain GZ-1</td>
</tr>
<tr class="sumtab-row">
<td><a href="https://www.ncbi.nlm.nih.gov/assembly/1469691">1469691</a></td>
<td class="numcol">145.25</td>
<td class="numcol">0.10</td>
<td class="numcol">156</td>
<td class="numcol">3</td>
<td><i>Olleya</i> sp. Bg11-27</td>
</tr>
<tr class="sumtab-row">
<td><a href="https://www.ncbi.nlm.nih.gov/assembly/7292581">7292581</a></td>
<td class="numcol">136.25</td>
<td class="numcol">0.09</td>
<td class="numcol">145</td>
<td class="numcol">4</td>
<td><i>Microbulbifer</i> sp. YPW1</td>
</tr>
<tr class="sumtab-row">
<td><a href="https://www.ncbi.nlm.nih.gov/assembly/638441">638441</a></td>
<td class="numcol">135.84</td>
<td class="numcol">0.08</td>
<td class="numcol">138</td>
<td class="numcol">4</td>
<td><i>Selenomonas</i> sp. F0591</td>
</tr>
<tr class="sumtab-row">
<td><a href="https://www.ncbi.nlm.nih.gov/assembly/9428571">9428571</a></td>
<td class="numcol">122.06</td>
<td class="numcol">0.08</td>
<td class="numcol">123</td>
<td class="numcol">4</td>
<td><i>Thermobispora bispora</i> strain ZC4</td>
</tr>
<tr class="sumtab-row">
<td><a href="https://www.ncbi.nlm.nih.gov/assembly/9307231">9307231</a></td>
<td class="numcol">117.00</td>
<td class="numcol">0.08</td>
<td class="numcol">117</td>
<td class="numcol">2</td>
<td><i>Thermoleptolyngbya</i> sp. PKUAC-SCTB121</td>
</tr>
<tr>
<td colspan="6" class="subhead">Archaea</td>
</tr>
<tr class="sumtab-row">
<td><a href="https://www.ncbi.nlm.nih.gov/assembly/4982121">4982121</a></td>
<td class="numcol">137.00</td>
<td class="numcol">0.09</td>
<td class="numcol">137</td>
<td class="numcol">2</td>
<td><i>Halomicrobium</i> sp. ZPS1</td>
</tr>
<tr class="sumtab-row">
<td><a href="https://www.ncbi.nlm.nih.gov/assembly/11057071">11057071</a></td>
<td class="numcol">131.55</td>
<td class="numcol">0.09</td>
<td class="numcol">146</td>
<td class="numcol">4</td>
<td><i>Halosiccatus urmianus</i> strain IBRC-M: 10911</td>
</tr>
<tr class="sumtab-row">
<td><a href="https://www.ncbi.nlm.nih.gov/assembly/9390741">9390741</a></td>
<td class="numcol">128.00</td>
<td class="numcol">0.09</td>
<td class="numcol">128</td>
<td class="numcol">2</td>
<td><i>Halapricum desulfuricans</i> strain HSR12-2</td>
</tr>
<tr class="sumtab-row">
<td><a href="https://www.ncbi.nlm.nih.gov/assembly/11151391">11151391</a></td>
<td class="numcol">127.74</td>
<td class="numcol">0.09</td>
<td class="numcol">145</td>
<td class="numcol">4</td>
<td><i>Halosiccatus</i> sp. LT50</td>
</tr>
<tr class="sumtab-row">
<td><a href="https://www.ncbi.nlm.nih.gov/assembly/9390521">9390521</a></td>
<td class="numcol">127.00</td>
<td class="numcol">0.09</td>
<td class="numcol">127</td>
<td class="numcol">2</td>
<td><i>Halapricum desulfuricans</i> strain HSR-Bgl</td>
</tr>
<tr class="sumtab-row">
<td><a href="https://www.ncbi.nlm.nih.gov/assembly/2582391">2582391</a></td>
<td class="numcol">125.81</td>
<td class="numcol">0.09</td>
<td class="numcol">137</td>
<td class="numcol">3</td>
<td><i>Halomicrobium mukohataei</i> strain JP60</td>
</tr>
<tr class="sumtab-row">
<td><a href="https://www.ncbi.nlm.nih.gov/assembly/11151361">11151361</a></td>
<td class="numcol">123.31</td>
<td class="numcol">0.08</td>
<td class="numcol">140</td>
<td class="numcol">4</td>
<td><i>Halosiccatus</i> sp. LT50</td>
</tr>
<tr class="sumtab-row">
<td><a href="https://www.ncbi.nlm.nih.gov/assembly/9390681">9390681</a></td>
<td class="numcol">111.00</td>
<td class="numcol">0.08</td>
<td class="numcol">111</td>
<td class="numcol">2</td>
<td><i>Halapricum desulfuricans</i> strain HSR-Est</td>
</tr>
<tr class="sumtab-row">
<td><a href="https://www.ncbi.nlm.nih.gov/assembly/9390731">9390731</a></td>
<td class="numcol">109.00</td>
<td class="numcol">0.07</td>
<td class="numcol">109</td>
<td class="numcol">2</td>
<td><i>Halapricum desulfuricans</i> strain HSR12-1</td>
</tr>
<tr class="sumtab-row">
<td><a href="https://www.ncbi.nlm.nih.gov/assembly/999901">999901</a></td>
<td class="numcol">98.00</td>
<td class="numcol">0.07</td>
<td class="numcol">98</td>
<td class="numcol">2</td>
<td><i>Halorientalis</i> sp. IM1011</td>
</tr>
</tbody>
</table>
<p><sup>*</sup> &mdash; entropy is <a href="https://en.wikipedia.org/wiki/Entropy_(information_theory)">Shannon entropy</a> calculated for each column of the multiple sequence alignment (MSA) of all full-length 16S rRNA genes of a genome. Entropy is then summed up (column "Sum of entropy") and averaged (column "Mean entropy").</p>
<hr />
<h2>Searching data in RiboGrove</h2>
<div id="searching-data" class="pad-anchor"></div>
<p>RiboGrove is a very minimalistic database &mdash; it comprises a collection of plain fasta files with metadata. Thus, extended search instruments are not available for it. We admit this problem and provide a list of suggestions below. The suggestions would help you to explore and subset RiboGrove data.</p>
<h3>Header format</h3>
<p>RiboGrove fasta data has the following format of header:</p>
<p class="samp-highl samp-vwide">&gt;NZ_CP079719.1:86193-87742_plus Bacillus_velezensis ;Bacteria;Firmicutes;Bacilli;Bacillales;Bacillaceae;Bacillus; category:2</p>
<p>Major blocks of a header are separated by spaces. A header consists of four such blocks:</p>
<ol>
<li>Sequence ID (seqID): <span class="samp-highl">NZ_CP079719.1:86193-87742_plus</span>. SeqID, in turn, consists of three parts:<ol>
<li>the accession number of the RefSeq sequence, from which the gene originates: <span class="samp-highl">NZ_CP079719.1</span>;</li>
<li>coordinates of the gene within this RefSeq genomic sequence: <span class="samp-highl">86193-87742</span> (coordinates are 1-based, left-closed and right-closed);</li>
<li>strand of the RefSeq genomic sequence, where the gene is located: <span class="samp-highl">plus</span>.</li>
</ol></li>
<li>The complete name of the organism, according to the <a href="https://www.ncbi.nlm.nih.gov/taxonomy">NCBI Taxonomy database</a>: <span class="samp-highl">Bacillus_velezensis</span>.</li>
<li>Taxonomy string, comprising domain (<span class="samp-highl">Bacteria</span>), phylum (<span class="samp-highl">Firmicutes</span>), class (<span class="samp-highl">Bacilli</span>), order (<span class="samp-highl">Bacillales</span>), family (<span class="samp-highl">Bacillaceae</span>), and genus (<span class="samp-highl">Bacillus</span>) names. The names are separated and flanked by semicolons (<span class="samp-highl">;</span>).</li>
<li>The category of the genome, from which the gene sequence originates: (<span class="samp-highl">category:2</span>). Category 1 genomes are of the highest reliability, and category 3 genomes are least reliable.</li>
</ol>
<h3>Subsetting sequences</h3>
<p>You can subset specific sequences from fasta files using the <strong>Seqkit</strong> program (<a href="https://github.com/shenwei356/seqkit">GitHub repo</a>, <a href="https://bioinf.shenwei.me/seqkit/">documentation</a>). It is free, cross-platform, multifunctional and pretty fast and can process both gzipped and uncompressed fasta files. To be specific, the <span class="samp">seqkit grep</span> is the program that should be used for subsetting.</p>
<h4>Subsetting sequences by header</h4>
<div id="subset-by-header" class="pad-anchor"></div>
<p>Given the downloaded fasta file <span class="samp">ribogrove_4.210_sequences.fasta.gz</span>, consider the following examples of sequence subsetting using <span class="samp">seqkit grep</span>:</p>
<p><strong>Example 1</strong>. Select a single sequence by SeqID</p>
<p class="samp-highl samp-vwide">seqkit grep -p "NZ_CP079719.1:86193-87742_plus" ribogrove_4.210_sequences.fasta.gz</p>
<p class="samp-comment">The <span>-p</span> option sets a pattern to search in fasta headers (only in sequence IDs, actually).</p>
<p><strong>Example 2</strong>. Select all gene sequences of a single RefSeq genomic sequence by accession number</p>
<p class="samp-highl samp-vwide">seqkit grep -nrp "NZ_CP079719.1" ribogrove_4.210_sequences.fasta.gz</p>
<p class="samp-comment">Here, two more options are required: <span class="samp">-n</span> and <span class="samp">-r</span>. The former tells the program to match the whole headers instead of IDs only. The latter tells the program not to exclude partial matches from output, i.e. if the pattern is a substring of a header, the header will be printed to output.</p>
<p><strong>Example 3</strong>. Select all gene sequences of a single genome (Assembly ID <a href="https://www.ncbi.nlm.nih.gov/assembly/10577151">10577151</a>), which has two replicons: <a href="https://www.ncbi.nlm.nih.gov/nuccore/NZ_CP079110.1">NZ_CP079110.1</a> and <a href="https://www.ncbi.nlm.nih.gov/nuccore/NZ_CP079111.1">NZ_CP079111.1</a></p>
<p class="samp-highl samp-vwide">seqkit grep -nr -p "NZ_CP079110.1" -p "NZ_CP079111.1" ribogrove_4.210_sequences.fasta.gz</p>
<p><strong>Example 4</strong>. Select all actinobacterial sequences</p>
<p class="samp-highl samp-vwide">seqkit grep -nrp ";Actinobacteria;" ribogrove_4.210_sequences.fasta.gz</p>
<p class="samp-comment">Just in case, surround the taxonomy name with semicolons (<span class="samp-highl">;</span>).</p>
<p><strong>Example 5</strong>. Select all sequences originating from category 1 genomes</p>
<p class="samp-highl samp-vwide">seqkit grep -nrp "category:1" ribogrove_4.210_sequences.fasta.gz</p>
<p><strong>Example 6</strong>. Select all sequences except for those belonging to <i>Firmicutes</i></p>
<p class="samp-highl samp-vwide">seqkit grep -nvrp ";Firmicutes;" ribogrove_4.210_sequences.fasta.gz</p>
<p class="samp-comment">Recognize the <span class="samp">-v</span> option within option sequence <span class="samp">-nvrp</span>. This option inverts match, i.e. without it the search would result in sequences belonging to <i>Firmicutes</i> only.</p>
<h4>Subsetting sequences by length</h4>
<div id="subset-by-len" class="pad-anchor"></div>
<p>Given the downloaded fasta file <span class="samp">ribogrove_4.210_sequences.fasta.gz</span>, consider the following examples of sequence subsetting using <span class="samp">seqkit seq</span>:</p>
<p><strong>Example 1</strong>. Select all sequences longer than 1600 bp</p>
<p class="samp-highl samp-vwide">seqkit seq -m 1601 ribogrove_4.210_sequences.fasta.gz</p>
<p class="samp-comment">The <span class="samp">-m</span> option sets the minimum length of a sequence to be printed to output.</p>
<p><strong>Example 2</strong>. Select all sequences shorter than 1500 bp</p>
<p class="samp-highl samp-vwide">seqkit seq -M 1499 ribogrove_4.210_sequences.fasta.gz</p>
<p class="samp-comment">The <span class="samp">-M</span> option sets the maximum length of a sequence to be printed to output.</p>
<p><strong>Example 3</strong>. Select all sequences having length in range [1500, 1600] bp</p>
<p class="samp-highl samp-vwide">seqkit seq -m 1500 -M 1600 ribogrove_4.210_sequences.fasta.gz</p>
<h4>Selecting header data</h4>
<div id="select-head" class="pad-anchor"></div>
<p>It is sometimes useful to retrieve only header information from a fasta file. So, please consider the following examples of data selection using <span class="samp">seqkit seq</span>:</p>
<p><strong>Example 1</strong>. Select all headers</p>
<p class="samp-highl samp-vwide">seqkit seq -n ribogrove_4.210_sequences.fasta.gz</p>
<p class="samp-comment">The <span class="samp">-n</span> option tells the program to output only headers.</p>
<p><strong>Example 2</strong>. Select all SeqIDs (header parts before the first space)</p>
<p class="samp-highl samp-vwide">seqkit seq -ni ribogrove_4.210_sequences.fasta.gz</p>
<p class="samp-comment">The <span class="samp">-i</span> option tells the program to output only sequence IDs.</p>
<p><strong>Example 3</strong>. Select all accession numbers</p>
<p class="samp-highl samp-vwide">seqkit seq -ni ribogrove_4.210_sequences.fasta.gz | sort | uniq</p>
<p class="samp-comment">This might be done only if you have <span class="samp">sort</span> and <span class="samp">uniq</span> utilities installed (Linux and Mac OS systems should have them built-in).</p>
<h2>Citing RiboGrove</h2>
<div id="citing-ribogrove" class="pad-anchor"></div>
<p>If you find RiboGrove useful for your research please cite:</p>
<p>Maxim A. Sikolenko, Leonid N. Valentovich. <q>RiboGrove: a database of full-length prokaryotic 16S rRNA genes derived from completely assembled genomes</q> // Research in Microbiology, Volume 173, Issue 4, May 2022, 103936.<br /> (DOI: <a href="https://doi.org/10.1016/j.resmic.2022.103936">10.1016/j.resmic.2022.103936</a>.</p>
<br />
<hr />
<p>RiboGrove, 2022-05-19</p>
</div>
</body>
</html>
